{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "124f91e7-79a9-4a33-a5e8-f51023dfdcae",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "载入需要的程序包：ChIPseeker\n",
      "\n",
      "\n",
      "\n",
      "ChIPseeker v1.42.1 Learn more at https://yulab-smu.top/contribution-knowledge-mining/\n",
      "\n",
      "Please cite:\n",
      "\n",
      "Guangchuang Yu, Li-Gen Wang, and Qing-Yu He. ChIPseeker: an\n",
      "R/Bioconductor package for ChIP peak annotation, comparison and\n",
      "visualization. Bioinformatics. 2015, 31(14):2382-2383\n",
      "\n",
      "载入需要的程序包：GenomicRanges\n",
      "\n",
      "载入需要的程序包：stats4\n",
      "\n",
      "载入需要的程序包：BiocGenerics\n",
      "\n",
      "\n",
      "载入程序包：‘BiocGenerics’\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:stats’:\n",
      "\n",
      "    IQR, mad, sd, var, xtabs\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:base’:\n",
      "\n",
      "    anyDuplicated, aperm, append, as.data.frame, basename, cbind,\n",
      "    colnames, dirname, do.call, duplicated, eval, evalq, Filter, Find,\n",
      "    get, grep, grepl, intersect, is.unsorted, lapply, Map, mapply,\n",
      "    match, mget, order, paste, pmax, pmax.int, pmin, pmin.int,\n",
      "    Position, rank, rbind, Reduce, rownames, sapply, saveRDS, setdiff,\n",
      "    table, tapply, union, unique, unsplit, which.max, which.min\n",
      "\n",
      "\n",
      "载入需要的程序包：S4Vectors\n",
      "\n",
      "\n",
      "载入程序包：‘S4Vectors’\n",
      "\n",
      "\n",
      "The following object is masked from ‘package:utils’:\n",
      "\n",
      "    findMatches\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:base’:\n",
      "\n",
      "    expand.grid, I, unname\n",
      "\n",
      "\n",
      "载入需要的程序包：IRanges\n",
      "\n",
      "载入需要的程序包：GenomeInfoDb\n",
      "\n",
      "\n",
      "\n",
      "========================================\n",
      "rGREAT version 2.8.0\n",
      "Bioconductor page: http://bioconductor.org/packages/rGREAT/\n",
      "Github page: https://github.com/jokergoo/rGREAT\n",
      "\n",
      "If you use it in published research, please cite:\n",
      "Gu, Z. rGREAT: an R/Bioconductor package for functional enrichment \n",
      "  on genomic regions. Bioinformatics 2023.\n",
      "\n",
      "This message can be suppressed by:\n",
      "  suppressPackageStartupMessages(library(rGREAT))\n",
      "\n",
      "Note: On Aug 19 2019 GREAT released version 4 where it supports `hg38`\n",
      "genome and removes some ontologies such pathways. `submitGreatJob()`\n",
      "still takes `hg19` as default. `hg38` can be specified by the `genome\n",
      "= 'hg38'` argument. To use the older versions such as 3.0.0, specify as\n",
      "`submitGreatJob(..., version = '3.0.0')`.\n",
      "\n",
      "From rGREAT version 1.99.0, it also implements the GREAT algorithm and\n",
      "it allows to integrate GREAT analysis with the Bioconductor annotation\n",
      "ecosystem. By default it supports more than 500 organisms. Check the\n",
      "new function `great()` and the new vignette.\n",
      "========================================\n",
      "\n",
      "\n",
      "载入需要的程序包：TxDb.Hsapiens.UCSC.hg38.knownGene\n",
      "\n",
      "载入需要的程序包：GenomicFeatures\n",
      "\n",
      "载入需要的程序包：AnnotationDbi\n",
      "\n",
      "载入需要的程序包：Biobase\n",
      "\n",
      "Welcome to Bioconductor\n",
      "\n",
      "    Vignettes contain introductory material; view with\n",
      "    'browseVignettes()'. To cite Bioconductor, see\n",
      "    'citation(\"Biobase\")', and for packages 'citation(\"pkgname\")'.\n",
      "\n",
      "\n",
      "\n",
      "载入程序包：‘dplyr’\n",
      "\n",
      "\n",
      "The following object is masked from ‘package:AnnotationDbi’:\n",
      "\n",
      "    select\n",
      "\n",
      "\n",
      "The following object is masked from ‘package:Biobase’:\n",
      "\n",
      "    combine\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:GenomicRanges’:\n",
      "\n",
      "    intersect, setdiff, union\n",
      "\n",
      "\n",
      "The following object is masked from ‘package:GenomeInfoDb’:\n",
      "\n",
      "    intersect\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:IRanges’:\n",
      "\n",
      "    collapse, desc, intersect, setdiff, slice, union\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:S4Vectors’:\n",
      "\n",
      "    first, intersect, rename, setdiff, setequal, union\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:BiocGenerics’:\n",
      "\n",
      "    combine, intersect, setdiff, union\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:stats’:\n",
      "\n",
      "    filter, lag\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:base’:\n",
      "\n",
      "    intersect, setdiff, setequal, union\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "require(ChIPseeker)\n",
    "library(rGREAT)\n",
    "require(TxDb.Hsapiens.UCSC.hg38.knownGene)\n",
    "library(ggplot2)\n",
    "library(dplyr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "f0e74b5e-20d1-4e2b-9273-b2893179fe38",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "txdb <- TxDb.Hsapiens.UCSC.hg38.knownGene"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "6d59c8fd-a005-4715-8884-f8ebcc2ba568",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">> preparing features information...\t\t 2026-01-04 14时11分57秒 \n",
      ">> identifying nearest features...\t\t 2026-01-04 14时11分59秒 \n",
      ">> calculating distance from peak to TSS...\t 2026-01-04 14时12分01秒 \n",
      ">> assigning genomic annotation...\t\t 2026-01-04 14时12分01秒 \n",
      ">> adding gene annotation...\t\t\t 2026-01-04 14时12分37秒 \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "载入需要的程序包：org.Hs.eg.db\n",
      "\n",
      "'select()' returned 1:1 mapping between keys and columns\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">> assigning chromosome lengths\t\t\t 2026-01-04 14时12分37秒 \n",
      ">> done...\t\t\t\t\t 2026-01-04 14时12分37秒 \n"
     ]
    }
   ],
   "source": [
    "#peaks <- read.table(\"/media/AnalysisDisk1/xuzekai/20250715_BMI/ATAC/DAR_enrichment/DAR_unique_peaks.bed\", header = FALSE, col.names = c(\"chr\", \"start\", \"end\"))\n",
    "peaks <- data.frame(\n",
    "  chr   = c(\"chr16\", \"chr2\"),\n",
    "  start = c(3065087, 136132091),\n",
    "  end   = c(3065588, 136132592),\n",
    "  stringsAsFactors = FALSE\n",
    ")\n",
    "gr <- makeGRangesFromDataFrame(peaks, ignore.strand = TRUE)\n",
    "peakAnno <- annotatePeak(gr, tssRegion=c(-20000, 20000), \n",
    "                         TxDb=txdb, annoDb=\"org.Hs.eg.db\")\n",
    "annotation <- peakAnno@annoStat"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "1966de5e-580a-465e-b069-f21373fd88fb",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A data.frame: 2 × 2</caption>\n",
       "<thead>\n",
       "\t<tr><th></th><th scope=col>Feature</th><th scope=col>Frequency</th></tr>\n",
       "\t<tr><th></th><th scope=col>&lt;fct&gt;</th><th scope=col>&lt;dbl&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><th scope=row>1</th><td><span style=white-space:pre-wrap>Promoter (&lt;=1kb)  </span></td><td>50</td></tr>\n",
       "\t<tr><th scope=row>2</th><td>Promoter (12-13kb)</td><td>50</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 2 × 2\n",
       "\\begin{tabular}{r|ll}\n",
       "  & Feature & Frequency\\\\\n",
       "  & <fct> & <dbl>\\\\\n",
       "\\hline\n",
       "\t1 & Promoter (<=1kb)   & 50\\\\\n",
       "\t2 & Promoter (12-13kb) & 50\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 2 × 2\n",
       "\n",
       "| <!--/--> | Feature &lt;fct&gt; | Frequency &lt;dbl&gt; |\n",
       "|---|---|---|\n",
       "| 1 | Promoter (&lt;=1kb)   | 50 |\n",
       "| 2 | Promoter (12-13kb) | 50 |\n",
       "\n"
      ],
      "text/plain": [
       "  Feature            Frequency\n",
       "1 Promoter (<=1kb)   50       \n",
       "2 Promoter (12-13kb) 50       "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "annotation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "8680505e-3b09-4455-bb80-77a6ead6a29f",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Processing: Atypical_Bm_ITGAX\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">> preparing features information...\t\t 2026-01-04 14时12分38秒 \n",
      ">> identifying nearest features...\t\t 2026-01-04 14时12分38秒 \n",
      ">> calculating distance from peak to TSS...\t 2026-01-04 14时12分38秒 \n",
      ">> assigning genomic annotation...\t\t 2026-01-04 14时12分38秒 \n",
      ">> adding gene annotation...\t\t\t 2026-01-04 14时12分42秒 \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "'select()' returned 1:many mapping between keys and columns\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">> assigning chromosome lengths\t\t\t 2026-01-04 14时12分42秒 \n",
      ">> done...\t\t\t\t\t 2026-01-04 14时12分42秒 \n"
     ]
    },
    {
     "ename": "ERROR",
     "evalue": "Error: 找不到对象'all_results'\n",
     "output_type": "error",
     "traceback": [
      "Error: 找不到对象'all_results'\nTraceback:\n",
      "1. mutate(., Feature = factor(Feature, levels = rev(c(\"Promoter (<=1kb)\", \n .     \"Promoter (1-2kb)\", \"Promoter (2-3kb)\", \"5' UTR\", \"3' UTR\", \n .     \"1st Exon\", \"Other Exon\", \"1st Intron\", \"Other Intron\", \"Downstream (<=300)\", \n .     \"Distal Intergenic\"))))",
      "2. .handleSimpleError(function (cnd) \n . {\n .     watcher$capture_plot_and_output()\n .     cnd <- sanitize_call(cnd)\n .     watcher$push(cnd)\n .     switch(on_error, continue = invokeRestart(\"eval_continue\"), \n .         stop = invokeRestart(\"eval_stop\"), error = NULL)\n . }, \"找不到对象'all_results'\", base::quote(eval(expr, envir)))"
     ]
    }
   ],
   "source": [
    "bed_dir <- \"/media/AnalysisDisk1/xuzekai/20250715_BMI/ATAC/DAR_bed/\"\n",
    "output_csv <- \"/media/AnalysisDisk1/xuzekai/20250715_BMI/ATAC/DAR_enrichment/all_celltypes_annotation_summary.csv\"\n",
    "\n",
    "# 加载数据库（根据你的物种和参考基因组）\n",
    "txdb <- TxDb.Hsapiens.UCSC.hg38.knownGene  # 或 TxDb.Hsapiens.UCSC.hg19.knownGene 等\n",
    "# 如果报错找不到，可以用：txdb <- loadDb(\"your_txdb.sqlite\")\n",
    "\n",
    "# 获取所有 .bed 文件\n",
    "bed_files <- list.files(bed_dir, pattern = \"*.bed$\", full.names = TRUE)\n",
    "names(bed_files) <- gsub(\"\\\\.bed$\", \"\", basename(bed_files))  # 去掉 .bed 作为 celltype 名称\n",
    "\n",
    "# 存储结果的列表\n",
    "results_list <- list()\n",
    "\n",
    "# 循环处理每个细胞类型\n",
    "for (file in bed_files) {\n",
    "  celltype <- gsub(\"\\\\.bed$\", \"\", basename(file))\n",
    "  message(\"Processing: \", celltype)\n",
    "  \n",
    "  # 读取 peaks\n",
    "  peaks <- read.table(file, header = FALSE, col.names = c(\"chr\", \"start\", \"end\"))\n",
    "  \n",
    "  # 转为 GRanges\n",
    "  gr <- makeGRangesFromDataFrame(peaks, ignore.strand = TRUE)\n",
    "  \n",
    "  # 注释 peaks\n",
    "  peakAnno <- annotatePeak(gr, \n",
    "                           tssRegion = c(-3000, 3000), \n",
    "                           TxDb = txdb, \n",
    "                           annoDb = \"org.Hs.eg.db\")\n",
    "  \n",
    "  # 提取 annoStat（统计表）\n",
    "  annotation_df <- peakAnno@annoStat\n",
    "  \n",
    "  # 添加 celltype 列\n",
    "  annotation_df$celltype <- celltype\n",
    "  library(ggplot2)\n",
    "library(dplyr)\n",
    "\n",
    "# 确保 Feature 是因子，并设置你喜欢的顺序\n",
    "all_results <- all_results %>%\n",
    "  mutate(Feature = factor(Feature, \n",
    "                          levels = rev(c(\"Promoter (<=1kb)\", \"Promoter (1-2kb)\", \"Promoter (2-3kb)\",\n",
    "                                         \"5' UTR\", \"3' UTR\", \"1st Exon\", \"Other Exon\",\n",
    "                                         \"1st Intron\", \"Other Intron\",\n",
    "                                         \"Downstream (<=300)\", \"Distal Intergenic\"))))\n",
    "\n",
    "# 绘制堆叠柱状图：每个细胞类型一个柱子\n",
    "p <- ggplot(all_results, aes(x = celltype, y = Frequency, fill = Feature)) +\n",
    "  geom_col() +  # 堆叠柱状图\n",
    "  scale_y_continuous(expand = expansion(mult = c(0, 0.05))) +\n",
    "  scale_fill_brewer(type = \"qual\", palette = \"Set3\", direction = -1) +  # reverse colors\n",
    "  labs(\n",
    "    title = \"Genomic Feature Distribution by Cell Type\",\n",
    "    x = \"Cell Type\",\n",
    "    y = \"Percentage (%)\"\n",
    "  ) +\n",
    "  theme_bw() +\n",
    "  theme(\n",
    "    axis.text.x = element_text(angle = 45, hjust = 1, size = 10),\n",
    "    legend.title = element_text(size = 11),\n",
    "    legend.text = element_text(size = 9),\n",
    "    legend.position = \"right\"\n",
    "  )\n",
    "\n",
    "# 显示图形\n",
    "print(p)\n",
    "  # 存入列表\n",
    "  results_list[[celltype]] <- annotation_df\n",
    "}\n",
    "\n",
    "# 合并所有结果\n",
    "all_results <- do.call(rbind, results_list)\n",
    "\n",
    "# 重置行名\n",
    "rownames(all_results) <- NULL\n",
    "\n",
    "# 查看前几行\n",
    "print(head(all_results))\n",
    "\n",
    "# 保存为 CSV\n",
    "write.csv(all_results, output_csv, row.names = FALSE)\n",
    "\n",
    "message(\"✅ 所有细胞类型的注释汇总已保存至: \", output_csv)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4f37dfe4-6db4-452c-8c04-9faece8531e8",
   "metadata": {},
   "outputs": [],
   "source": [
    "epeak_number_final_cs_ch<-read.csv('/media/AnalysisDisk1/xuzekai/20250115_CIMA/2025_0212_CIMA_eGene_caPeak_Statistics/eGene_cellspecial_number_20250117.csv')\n",
    "celltype_mapping <- tibble::tribble(\n",
    "  ~celltype,     ~celltype.1,\n",
    "    \"AS_DC\",       \"Mono\",\n",
    "    \"CD4_Tem_CCR5\",       \"CD4\",\n",
    "    \"CD4_Tr1_IL10\",       \"CD4\",\n",
    "    \"DC1_CLEC9A\",       \"Mono\",\n",
    "    \"DC2_CD1C\",       \"Mono\",\n",
    "    \"DC_CSF2RA\",       \"Mono\",\n",
    "    \"Immature_T_CABP4\",       \"CD8\",\n",
    "    \"Proliferative_NK_MKI67\",       \"NK\",\n",
    "    \"Proliferative_T_MKI67\",       \"CD8\",\n",
    "    \"Transitional_B_MME\",       \"B\",\n",
    "    \"pre-Switched_Bm_IFIT3\",       \"B\",\n",
    "  # 可以在这里继续添加其他映射\n",
    ")\n",
    "\n",
    "# 如果你还想保留 epeak_number_final_cs_ch 中的其他映射，可以合并\n",
    "# 假设原表 epeak_number_final_cs_ch 有部分映射，但不完整\n",
    "full_mapping <- bind_rows(\n",
    "  celltype_mapping,\n",
    "  epeak_number_final_cs_ch %>% select(celltype, celltype.1)\n",
    ") %>% distinct(celltype, .keep_all = TRUE)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6a23d1f6-9469-438c-bfac-6bcc76102446",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "all_results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "78e8c7b9-4fc5-4753-b3b3-89bcbb9ae5dc",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "result_wide_mapped <- all_results %>%\n",
    "  left_join(full_mapping, by = \"celltype\")\n",
    "# 查看结果\n",
    "#result_wide_mapped= result_wide_mapped%>%select(celltype,down,up,celltype.1)\n",
    "result_wide_mapped"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f9fee09d-dadd-43e6-84c1-00d2e1b26c1d",
   "metadata": {},
   "outputs": [],
   "source": [
    "super_group_order <- c(\"B\", \"CD4\", \"CD8\", \"NK\", \"Mono\")\n",
    "\n",
    "# 创建排序键\n",
    "ordering_table <- result_wide_mapped  %>%\n",
    "  mutate(\n",
    "   celltype.1 = factor(celltype.1, levels = super_group_order),\n",
    "    celltype = factor(celltype, levels = unique(celltype))  # 可自定义顺序\n",
    "  ) %>%\n",
    "  arrange(celltype.1, celltype)  # 按大类 + 名称排序"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "58f24913-e07f-4a4d-a4c0-f59a5ddaf8df",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "ordering_table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d013ebf8-98f1-43b5-bb35-c244fb25593f",
   "metadata": {},
   "outputs": [],
   "source": [
    "unique_celltype_order <- unique(ordering_table$celltype)\n",
    "\n",
    "# 2. 将 all_results_with_group 中的 celltype 设为有序因子\n",
    "ordering_table$celltype <- factor(\n",
    "  ordering_table$celltype,\n",
    "  levels = unique_celltype_order,    # ← 这里传入唯一值的向量\n",
    "  ordered = TRUE\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d2df4c4a-6f59-4567-aed7-8d1fe7bf9083",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "# 确保 Feature 是因子，并设置你喜欢的顺序\n",
    "ordering_table <- ordering_table %>%\n",
    "  mutate(Feature = factor(Feature, \n",
    "                          levels = rev(c(\"Promoter (<=1kb)\", \"Promoter (1-2kb)\", \"Promoter (2-3kb)\",\n",
    "                                         \"5' UTR\", \"3' UTR\", \"1st Exon\", \"Other Exon\",\n",
    "                                         \"1st Intron\", \"Other Intron\",\n",
    "                                         \"Downstream (<=300)\", \"Distal Intergenic\"))))\n",
    "\n",
    "# 绘制堆叠柱状图：每个细胞类型一个柱子\n",
    "p <- ggplot(ordering_table, aes(x = celltype, y = Frequency, fill = Feature)) +\n",
    "  geom_col() +  # 堆叠柱状图\n",
    "  scale_y_continuous(expand = expansion(mult = 0)) +\n",
    "  scale_fill_brewer(type = \"qual\", palette = \"Set3\", direction = -1) +  # reverse colors\n",
    "  labs(\n",
    "    title = \"Genomic Feature Distribution by Cell Type\",\n",
    "    x = \"Cell Type\",\n",
    "    y = \"Percentage (%)\"\n",
    "  ) +\n",
    "  theme_bw() +\n",
    "  theme(\n",
    "    axis.text.x = element_text(angle = 90, hjust = 1, size = 10),\n",
    "    legend.title = element_text(size = 11),\n",
    "    legend.text = element_text(size = 9),\n",
    "    legend.position = \"right\"\n",
    "  )\n",
    "\n",
    "# 显示图形\n",
    "print(p)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "c4a213c4-716d-4b14-810c-b80c4a3dcb34",
   "metadata": {},
   "outputs": [],
   "source": [
    "ggsave(\"feature_distribution_by_celltype.pdf\", p, width = 10, height = 6)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3c5abacd-20fd-43b0-bc4b-bf880e486cb2",
   "metadata": {},
   "outputs": [],
   "source": [
    "plotAnnoBar.data.frame <- function(anno.df,\n",
    "                                   xlab=\"\",\n",
    "                                   ylab=\"Percentage(%)\",\n",
    "                                   title=\"Feature Distribution\",\n",
    "                                   categoryColumn) {\n",
    "\n",
    "\n",
    "    anno.df$Feature <- factor(anno.df$Feature, levels = rev(levels(anno.df$Feature)))\n",
    "\n",
    "    p <- ggplot(anno.df, aes_string(x = categoryColumn,\n",
    "                                    fill = \"Feature\",\n",
    "                                    y = \"Frequency\"))\n",
    "\n",
    "    p <- p + geom_bar(stat=\"identity\") + coord_flip() + theme_bw()\n",
    "    p <- p + ylab(ylab) + xlab(xlab) + ggtitle(title)\n",
    "\n",
    "    if (categoryColumn == 1) {\n",
    "        p <- p + scale_x_continuous(breaks=NULL)\n",
    "        p <- p+scale_fill_manual(values=rev(getCols(nrow(anno.df))), guide=guide_legend(reverse=TRUE))\n",
    "    } else {\n",
    "        p <- p+scale_fill_manual(values=rev(getCols(length(unique(anno.df$Feature)))), guide=guide_legend(reverse=TRUE))\n",
    "    }\n",
    "\n",
    "    return(p)\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1903684a-818f-4962-bdd9-0cdd71dda3d9",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7f8370d5-90c0-472a-b764-afe991d0cafb",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "5537c533-3eb6-4ad5-8b35-949ae1f13108",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d457613a-0af6-42c7-92a0-148e732d7b7e",
   "metadata": {},
   "outputs": [],
   "source": [
    "pdf(\"peak_annotation_pie.pdf\", width = 8, height = 6)\n",
    "plotAnnoPie(peakAnno)\n",
    "dev.off()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1a2c19e4-4338-41d0-b1cc-9afcfa0374cf",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "plotAnnoBar(peakAnno)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "73784c68-e9c0-4490-864d-f97231047158",
   "metadata": {},
   "outputs": [],
   "source": [
    "peaks <- GRanges(\n",
    "  seqnames = \"chr15\",\n",
    "  ranges = IRanges(start = 39580700, end = 39581201),\n",
    "  name = \"Peak_1\"  # 可选：给 peak 起个名字\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "09ba2902-3ac3-476e-9349-c84822288305",
   "metadata": {},
   "outputs": [],
   "source": [
    "peakAnno <- annotatePeak(peaks, tssRegion=c(-3000, 3000), \n",
    "                         TxDb=txdb, annoDb=\"org.Hs.eg.db\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8d96dcbb-b5c2-418a-8fcd-7569626358c4",
   "metadata": {},
   "outputs": [],
   "source": [
    "peakAnno"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1a9102e1-430e-4d47-8697-926af79c42fb",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "R (r-signac)",
   "language": "R",
   "name": "r-signac"
  },
  "language_info": {
   "codemirror_mode": "r",
   "file_extension": ".r",
   "mimetype": "text/x-r-source",
   "name": "R",
   "pygments_lexer": "r",
   "version": "4.4.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
