{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "0f0bed17-8cf0-4713-9027-941bf0edc40e",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "载入需要的程序包：GenomicRanges\n",
      "\n",
      "载入需要的程序包：stats4\n",
      "\n",
      "载入需要的程序包：BiocGenerics\n",
      "\n",
      "\n",
      "载入程序包：‘BiocGenerics’\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:stats’:\n",
      "\n",
      "    IQR, mad, sd, var, xtabs\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:base’:\n",
      "\n",
      "    anyDuplicated, aperm, append, as.data.frame, basename, cbind,\n",
      "    colnames, dirname, do.call, duplicated, eval, evalq, Filter, Find,\n",
      "    get, grep, grepl, intersect, is.unsorted, lapply, Map, mapply,\n",
      "    match, mget, order, paste, pmax, pmax.int, pmin, pmin.int,\n",
      "    Position, rank, rbind, Reduce, rownames, sapply, saveRDS, setdiff,\n",
      "    table, tapply, union, unique, unsplit, which.max, which.min\n",
      "\n",
      "\n",
      "载入需要的程序包：S4Vectors\n",
      "\n",
      "\n",
      "载入程序包：‘S4Vectors’\n",
      "\n",
      "\n",
      "The following object is masked from ‘package:utils’:\n",
      "\n",
      "    findMatches\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:base’:\n",
      "\n",
      "    expand.grid, I, unname\n",
      "\n",
      "\n",
      "载入需要的程序包：IRanges\n",
      "\n",
      "载入需要的程序包：GenomeInfoDb\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "========================================\n",
      "rGREAT version 2.8.0\n",
      "Bioconductor page: http://bioconductor.org/packages/rGREAT/\n",
      "Github page: https://github.com/jokergoo/rGREAT\n",
      "\n",
      "If you use it in published research, please cite:\n",
      "Gu, Z. rGREAT: an R/Bioconductor package for functional enrichment \n",
      "  on genomic regions. Bioinformatics 2023.\n",
      "\n",
      "This message can be suppressed by:\n",
      "  suppressPackageStartupMessages(library(rGREAT))\n",
      "\n",
      "Note: On Aug 19 2019 GREAT released version 4 where it supports `hg38`\n",
      "genome and removes some ontologies such pathways. `submitGreatJob()`\n",
      "still takes `hg19` as default. `hg38` can be specified by the `genome\n",
      "= 'hg38'` argument. To use the older versions such as 3.0.0, specify as\n",
      "`submitGreatJob(..., version = '3.0.0')`.\n",
      "\n",
      "From rGREAT version 1.99.0, it also implements the GREAT algorithm and\n",
      "it allows to integrate GREAT analysis with the Bioconductor annotation\n",
      "ecosystem. By default it supports more than 500 organisms. Check the\n",
      "new function `great()` and the new vignette.\n",
      "========================================\n",
      "\n",
      "\n",
      "\n",
      "载入程序包：‘dplyr’\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:GenomicRanges’:\n",
      "\n",
      "    intersect, setdiff, union\n",
      "\n",
      "\n",
      "The following object is masked from ‘package:GenomeInfoDb’:\n",
      "\n",
      "    intersect\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:IRanges’:\n",
      "\n",
      "    collapse, desc, intersect, setdiff, slice, union\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:S4Vectors’:\n",
      "\n",
      "    first, intersect, rename, setdiff, setequal, union\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:BiocGenerics’:\n",
      "\n",
      "    combine, intersect, setdiff, union\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:stats’:\n",
      "\n",
      "    filter, lag\n",
      "\n",
      "\n",
      "The following objects are masked from ‘package:base’:\n",
      "\n",
      "    intersect, setdiff, setequal, union\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "library(rGREAT)\n",
    "library(dplyr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "d5228ed1-15f6-411d-8906-99eef335fb19",
   "metadata": {},
   "outputs": [],
   "source": [
    "peaks <- read.table(\"total_DAR_rgreat_input.bed\", header = FALSE)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "1f9b8703-b846-4d8e-8cb7-a48d3e993cb0",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A data.frame: 33529 × 13</caption>\n",
       "<thead>\n",
       "\t<tr><th scope=col>V1</th><th scope=col>V2</th><th scope=col>V3</th><th scope=col>V4</th><th scope=col>V5</th><th scope=col>V6</th><th scope=col>V7</th><th scope=col>V8</th><th scope=col>V9</th><th scope=col>V10</th><th scope=col>V11</th><th scope=col>V12</th><th scope=col>V13</th></tr>\n",
       "\t<tr><th scope=col>&lt;int&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;int&gt;</th><th scope=col>&lt;int&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><td> 0</td><td>1.799742e-15</td><td>-4.3480747</td><td>6.083777e-10</td><td>Underweight_vs_Normal    </td><td>chrY:12904575-12905076  </td><td>CD4_Treg_FCRL3   </td><td>Underweight</td><td>Normal    </td><td>chrY </td><td> 12904575</td><td> 12905076</td><td>down</td></tr>\n",
       "\t<tr><td> 1</td><td>4.509594e-11</td><td>-2.6346098</td><td>1.524405e-05</td><td>Underweight_vs_Normal    </td><td>chrY:20575413-20575914  </td><td>CD4_Treg_FCRL3   </td><td>Underweight</td><td>Normal    </td><td>chrY </td><td> 20575413</td><td> 20575914</td><td>down</td></tr>\n",
       "\t<tr><td> 2</td><td>4.035983e-09</td><td>-2.3952563</td><td>1.364308e-03</td><td>Underweight_vs_Normal    </td><td>chrY:7273539-7274040    </td><td>CD4_Treg_FCRL3   </td><td>Underweight</td><td>Normal    </td><td>chrY </td><td>  7273539</td><td>  7274040</td><td>down</td></tr>\n",
       "\t<tr><td> 3</td><td>4.173088e-09</td><td>-3.3596235</td><td>1.410654e-03</td><td>Underweight_vs_Overweight</td><td>chrY:20575413-20575914  </td><td>CD4_Treg_FCRL3   </td><td>Underweight</td><td>Overweight</td><td>chrY </td><td> 20575413</td><td> 20575914</td><td>down</td></tr>\n",
       "\t<tr><td> 4</td><td>9.946644e-08</td><td>-4.4064829</td><td>3.362324e-02</td><td>Underweight_vs_Overweight</td><td>chrY:12904575-12905076  </td><td>CD4_Treg_FCRL3   </td><td>Underweight</td><td>Overweight</td><td>chrY </td><td> 12904575</td><td> 12905076</td><td>down</td></tr>\n",
       "\t<tr><td> 5</td><td>1.316137e-26</td><td>-1.8496298</td><td>4.449016e-21</td><td>Underweight_vs_Normal    </td><td>chrY:12904575-12905076  </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chrY </td><td> 12904575</td><td> 12905076</td><td>down</td></tr>\n",
       "\t<tr><td> 6</td><td>5.311663e-25</td><td>-2.7756344</td><td>1.795533e-19</td><td>Underweight_vs_Normal    </td><td>chrY:19567010-19567511  </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chrY </td><td> 19567010</td><td> 19567511</td><td>down</td></tr>\n",
       "\t<tr><td> 7</td><td>8.764596e-20</td><td>-1.7067677</td><td>2.962749e-14</td><td>Underweight_vs_Normal    </td><td>chrY:20575413-20575914  </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chrY </td><td> 20575413</td><td> 20575914</td><td>down</td></tr>\n",
       "\t<tr><td> 8</td><td>3.747125e-19</td><td>-2.7628132</td><td>1.266663e-13</td><td>Underweight_vs_Normal    </td><td>chrY:12662009-12662510  </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chrY </td><td> 12662009</td><td> 12662510</td><td>down</td></tr>\n",
       "\t<tr><td> 9</td><td>8.780786e-17</td><td>-2.3446464</td><td>2.968222e-11</td><td>Underweight_vs_Normal    </td><td>chrY:7273539-7274040    </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chrY </td><td>  7273539</td><td>  7274040</td><td>down</td></tr>\n",
       "\t<tr><td>10</td><td>4.452406e-16</td><td>-2.2751276</td><td>1.505073e-10</td><td>Underweight_vs_Normal    </td><td>chrY:21240194-21240695  </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chrY </td><td> 21240194</td><td> 21240695</td><td>down</td></tr>\n",
       "\t<tr><td>11</td><td>2.697797e-15</td><td>-2.8051588</td><td>9.119526e-10</td><td>Underweight_vs_Normal    </td><td>chrY:8535695-8536196    </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chrY </td><td>  8535695</td><td>  8536196</td><td>down</td></tr>\n",
       "\t<tr><td>12</td><td>2.884100e-14</td><td>-2.3366449</td><td>9.749297e-09</td><td>Underweight_vs_Normal    </td><td>chrY:13479855-13480356  </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chrY </td><td> 13479855</td><td> 13480356</td><td>down</td></tr>\n",
       "\t<tr><td>13</td><td>1.085742e-13</td><td>-2.1524231</td><td>3.670199e-08</td><td>Underweight_vs_Normal    </td><td>chrY:19567551-19568052  </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chrY </td><td> 19567551</td><td> 19568052</td><td>down</td></tr>\n",
       "\t<tr><td>14</td><td>1.294381e-13</td><td>-2.2614920</td><td>4.375474e-08</td><td>Underweight_vs_Normal    </td><td>chrY:2935801-2936302    </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chrY </td><td>  2935801</td><td>  2936302</td><td>down</td></tr>\n",
       "\t<tr><td>15</td><td>1.425224e-12</td><td> 0.9795291</td><td>4.817772e-07</td><td>Underweight_vs_Normal    </td><td>chr8:2133762-2134263    </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chr8 </td><td>  2133762</td><td>  2134263</td><td>up  </td></tr>\n",
       "\t<tr><td>16</td><td>4.264866e-12</td><td>-2.3683660</td><td>1.441678e-06</td><td>Underweight_vs_Normal    </td><td>chrY:12905426-12905927  </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chrY </td><td> 12905426</td><td> 12905927</td><td>down</td></tr>\n",
       "\t<tr><td>17</td><td>1.101998e-11</td><td>-2.6900289</td><td>3.725149e-06</td><td>Underweight_vs_Normal    </td><td>chrY:2934881-2935382    </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chrY </td><td>  2934881</td><td>  2935382</td><td>down</td></tr>\n",
       "\t<tr><td>18</td><td>1.455245e-11</td><td>-3.2319445</td><td>4.919254e-06</td><td>Underweight_vs_Normal    </td><td>chrY:21249476-21249977  </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chrY </td><td> 21249476</td><td> 21249977</td><td>down</td></tr>\n",
       "\t<tr><td>19</td><td>5.140688e-11</td><td> 0.6653898</td><td>1.737738e-05</td><td>Underweight_vs_Normal    </td><td>chrX:73851023-73851524  </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chrX </td><td> 73851023</td><td> 73851524</td><td>up  </td></tr>\n",
       "\t<tr><td>20</td><td>3.149590e-10</td><td>-2.7421408</td><td>1.064675e-04</td><td>Underweight_vs_Normal    </td><td>chrY:13703557-13704058  </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chrY </td><td> 13703557</td><td> 13704058</td><td>down</td></tr>\n",
       "\t<tr><td>21</td><td>5.443223e-10</td><td>-2.1728118</td><td>1.840005e-04</td><td>Underweight_vs_Normal    </td><td>chrY:21256797-21257298  </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chrY </td><td> 21256797</td><td> 21257298</td><td>down</td></tr>\n",
       "\t<tr><td>22</td><td>6.260554e-10</td><td> 1.0074009</td><td>2.116292e-04</td><td>Underweight_vs_Normal    </td><td>chr17:83083463-83083964 </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chr17</td><td> 83083463</td><td> 83083964</td><td>up  </td></tr>\n",
       "\t<tr><td>23</td><td>1.527570e-09</td><td>-2.2332651</td><td>5.163738e-04</td><td>Underweight_vs_Normal    </td><td>chrY:20798332-20798833  </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chrY </td><td> 20798332</td><td> 20798833</td><td>down</td></tr>\n",
       "\t<tr><td>24</td><td>5.358021e-09</td><td> 0.5625515</td><td>1.811204e-03</td><td>Underweight_vs_Normal    </td><td>chr17:1607284-1607785   </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chr17</td><td>  1607284</td><td>  1607785</td><td>up  </td></tr>\n",
       "\t<tr><td>25</td><td>8.781393e-09</td><td> 0.9031823</td><td>2.968427e-03</td><td>Underweight_vs_Normal    </td><td>chr10:22505988-22506489 </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chr10</td><td> 22505988</td><td> 22506489</td><td>up  </td></tr>\n",
       "\t<tr><td>26</td><td>1.625563e-08</td><td> 0.8910524</td><td>5.494988e-03</td><td>Underweight_vs_Normal    </td><td>chr15:69881734-69882235 </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chr15</td><td> 69881734</td><td> 69882235</td><td>up  </td></tr>\n",
       "\t<tr><td>27</td><td>1.841356e-08</td><td> 1.2060734</td><td>6.224447e-03</td><td>Underweight_vs_Normal    </td><td>chr3:69984627-69985128  </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chr3 </td><td> 69984627</td><td> 69985128</td><td>up  </td></tr>\n",
       "\t<tr><td>28</td><td>1.908690e-08</td><td>-2.4654723</td><td>6.452060e-03</td><td>Underweight_vs_Normal    </td><td>chr15:27865591-27866092 </td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chr15</td><td> 27865591</td><td> 27866092</td><td>down</td></tr>\n",
       "\t<tr><td>29</td><td>1.988223e-08</td><td> 1.0965137</td><td>6.720908e-03</td><td>Underweight_vs_Normal    </td><td>chr9:130352049-130352550</td><td>Atypical_Bm_ITGAX</td><td>Underweight</td><td>Normal    </td><td>chr9 </td><td>130352049</td><td>130352550</td><td>up  </td></tr>\n",
       "\t<tr><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td></tr>\n",
       "\t<tr><td>43508</td><td>1.919081e-07</td><td>10.285568</td><td>0.06487184</td><td>Underweight_vs_Overweight</td><td>chr4:57529425-57529926   </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr4 </td><td> 57529425</td><td> 57529926</td><td>up  </td></tr>\n",
       "\t<tr><td>43509</td><td>1.920635e-07</td><td>10.300419</td><td>0.06492436</td><td>Underweight_vs_Overweight</td><td>chr1:184427558-184428059 </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr1 </td><td>184427558</td><td>184428059</td><td>up  </td></tr>\n",
       "\t<tr><td>43510</td><td>1.925364e-07</td><td> 2.854993</td><td>0.06508422</td><td>Underweight_vs_Overweight</td><td>chr2:164739926-164740427 </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr2 </td><td>164739926</td><td>164740427</td><td>up  </td></tr>\n",
       "\t<tr><td>43511</td><td>1.930047e-07</td><td>10.312927</td><td>0.06524254</td><td>Underweight_vs_Overweight</td><td>chr1:76868036-76868537   </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr1 </td><td> 76868036</td><td> 76868537</td><td>up  </td></tr>\n",
       "\t<tr><td>43512</td><td>1.939351e-07</td><td>10.346625</td><td>0.06555704</td><td>Underweight_vs_Overweight</td><td>chr2:63125560-63126061   </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr2 </td><td> 63125560</td><td> 63126061</td><td>up  </td></tr>\n",
       "\t<tr><td>43513</td><td>1.943840e-07</td><td>10.340085</td><td>0.06570880</td><td>Underweight_vs_Overweight</td><td>chr6:89244161-89244662   </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr6 </td><td> 89244161</td><td> 89244662</td><td>up  </td></tr>\n",
       "\t<tr><td>43514</td><td>1.944907e-07</td><td> 4.163310</td><td>0.06574487</td><td>Underweight_vs_Overweight</td><td>chr6:90402163-90402664   </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr6 </td><td> 90402163</td><td> 90402664</td><td>up  </td></tr>\n",
       "\t<tr><td>43515</td><td>1.965275e-07</td><td>10.333976</td><td>0.06643335</td><td>Underweight_vs_Overweight</td><td>chr15:55777044-55777545  </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr15</td><td> 55777044</td><td> 55777545</td><td>up  </td></tr>\n",
       "\t<tr><td>43516</td><td>1.967347e-07</td><td>10.299023</td><td>0.06650342</td><td>Underweight_vs_Overweight</td><td>chr1:60792940-60793441   </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr1 </td><td> 60792940</td><td> 60793441</td><td>up  </td></tr>\n",
       "\t<tr><td>43517</td><td>1.970261e-07</td><td>10.287704</td><td>0.06660192</td><td>Underweight_vs_Overweight</td><td>chr4:11610564-11611065   </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr4 </td><td> 11610564</td><td> 11611065</td><td>up  </td></tr>\n",
       "\t<tr><td>43518</td><td>1.977493e-07</td><td>10.307856</td><td>0.06684640</td><td>Underweight_vs_Overweight</td><td>chr4:120575210-120575711 </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr4 </td><td>120575210</td><td>120575711</td><td>up  </td></tr>\n",
       "\t<tr><td>43519</td><td>1.977906e-07</td><td>10.356051</td><td>0.06686034</td><td>Underweight_vs_Overweight</td><td>chr8:125660798-125661299 </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr8 </td><td>125660798</td><td>125661299</td><td>up  </td></tr>\n",
       "\t<tr><td>43520</td><td>1.989527e-07</td><td>10.366777</td><td>0.06725319</td><td>Underweight_vs_Overweight</td><td>chr11:94928559-94929060  </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr11</td><td> 94928559</td><td> 94929060</td><td>up  </td></tr>\n",
       "\t<tr><td>43521</td><td>1.991894e-07</td><td>10.329678</td><td>0.06733317</td><td>Underweight_vs_Overweight</td><td>chr6:76917140-76917641   </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr6 </td><td> 76917140</td><td> 76917641</td><td>up  </td></tr>\n",
       "\t<tr><td>43522</td><td>1.997892e-07</td><td>10.343486</td><td>0.06753595</td><td>Underweight_vs_Overweight</td><td>chr12:92137156-92137657  </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr12</td><td> 92137156</td><td> 92137657</td><td>up  </td></tr>\n",
       "\t<tr><td>43523</td><td>2.007245e-07</td><td>10.336654</td><td>0.06785211</td><td>Underweight_vs_Overweight</td><td>chr3:111841047-111841548 </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr3 </td><td>111841047</td><td>111841548</td><td>up  </td></tr>\n",
       "\t<tr><td>43524</td><td>2.013605e-07</td><td>10.371312</td><td>0.06806711</td><td>Underweight_vs_Overweight</td><td>chr2:136422487-136422988 </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr2 </td><td>136422487</td><td>136422988</td><td>up  </td></tr>\n",
       "\t<tr><td>43525</td><td>2.015351e-07</td><td>10.379426</td><td>0.06812612</td><td>Underweight_vs_Overweight</td><td>chr4:146155292-146155793 </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr4 </td><td>146155292</td><td>146155793</td><td>up  </td></tr>\n",
       "\t<tr><td>43526</td><td>2.026111e-07</td><td>10.361621</td><td>0.06848984</td><td>Underweight_vs_Overweight</td><td>chr14:97921242-97921743  </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr14</td><td> 97921242</td><td> 97921743</td><td>up  </td></tr>\n",
       "\t<tr><td>43527</td><td>2.026805e-07</td><td>10.323577</td><td>0.06851329</td><td>Underweight_vs_Overweight</td><td>chr10:45456745-45457246  </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr10</td><td> 45456745</td><td> 45457246</td><td>up  </td></tr>\n",
       "\t<tr><td>43528</td><td>2.032085e-07</td><td>10.337345</td><td>0.06869179</td><td>Underweight_vs_Overweight</td><td>chr4:40111989-40112490   </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr4 </td><td> 40111989</td><td> 40112490</td><td>up  </td></tr>\n",
       "\t<tr><td>43529</td><td>2.034039e-07</td><td>10.379938</td><td>0.06875785</td><td>Underweight_vs_Overweight</td><td>chr13:109984802-109985303</td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr13</td><td>109984802</td><td>109985303</td><td>up  </td></tr>\n",
       "\t<tr><td>43530</td><td>2.041574e-07</td><td>10.342093</td><td>0.06901255</td><td>Underweight_vs_Overweight</td><td>chrX:43350827-43351328   </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chrX </td><td> 43350827</td><td> 43351328</td><td>up  </td></tr>\n",
       "\t<tr><td>43531</td><td>2.054669e-07</td><td>10.381156</td><td>0.06945522</td><td>Underweight_vs_Overweight</td><td>chr13:97413574-97414075  </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr13</td><td> 97413574</td><td> 97414075</td><td>up  </td></tr>\n",
       "\t<tr><td>43532</td><td>2.058333e-07</td><td>10.380163</td><td>0.06957907</td><td>Underweight_vs_Overweight</td><td>chr9:33271029-33271530   </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr9 </td><td> 33271029</td><td> 33271530</td><td>up  </td></tr>\n",
       "\t<tr><td>43533</td><td>2.092217e-07</td><td>10.388479</td><td>0.07072446</td><td>Underweight_vs_Overweight</td><td>chr16:81578042-81578543  </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr16</td><td> 81578042</td><td> 81578543</td><td>up  </td></tr>\n",
       "\t<tr><td>43534</td><td>2.447439e-07</td><td> 4.547295</td><td>0.08273224</td><td>Underweight_vs_Overweight</td><td>chr7:44631362-44631863   </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr7 </td><td> 44631362</td><td> 44631863</td><td>up  </td></tr>\n",
       "\t<tr><td>43535</td><td>2.463935e-07</td><td>-2.429886</td><td>0.08328988</td><td>Underweight_vs_Overweight</td><td>chr5:74839170-74839671   </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr5 </td><td> 74839170</td><td> 74839671</td><td>down</td></tr>\n",
       "\t<tr><td>43536</td><td>2.574729e-07</td><td> 3.842864</td><td>0.08703511</td><td>Underweight_vs_Overweight</td><td>chr20:45405444-45405945  </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr20</td><td> 45405444</td><td> 45405945</td><td>up  </td></tr>\n",
       "\t<tr><td>43537</td><td>2.666070e-07</td><td> 4.368078</td><td>0.09012276</td><td>Underweight_vs_Overweight</td><td>chr20:49750515-49751016  </td><td>Unswitched_Bm_CD1C</td><td>Underweight</td><td>Overweight</td><td>chr20</td><td> 49750515</td><td> 49751016</td><td>up  </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 33529 × 13\n",
       "\\begin{tabular}{lllllllllllll}\n",
       " V1 & V2 & V3 & V4 & V5 & V6 & V7 & V8 & V9 & V10 & V11 & V12 & V13\\\\\n",
       " <int> & <dbl> & <dbl> & <dbl> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <int> & <int> & <chr>\\\\\n",
       "\\hline\n",
       "\t  0 & 1.799742e-15 & -4.3480747 & 6.083777e-10 & Underweight\\_vs\\_Normal     & chrY:12904575-12905076   & CD4\\_Treg\\_FCRL3    & Underweight & Normal     & chrY  &  12904575 &  12905076 & down\\\\\n",
       "\t  1 & 4.509594e-11 & -2.6346098 & 1.524405e-05 & Underweight\\_vs\\_Normal     & chrY:20575413-20575914   & CD4\\_Treg\\_FCRL3    & Underweight & Normal     & chrY  &  20575413 &  20575914 & down\\\\\n",
       "\t  2 & 4.035983e-09 & -2.3952563 & 1.364308e-03 & Underweight\\_vs\\_Normal     & chrY:7273539-7274040     & CD4\\_Treg\\_FCRL3    & Underweight & Normal     & chrY  &   7273539 &   7274040 & down\\\\\n",
       "\t  3 & 4.173088e-09 & -3.3596235 & 1.410654e-03 & Underweight\\_vs\\_Overweight & chrY:20575413-20575914   & CD4\\_Treg\\_FCRL3    & Underweight & Overweight & chrY  &  20575413 &  20575914 & down\\\\\n",
       "\t  4 & 9.946644e-08 & -4.4064829 & 3.362324e-02 & Underweight\\_vs\\_Overweight & chrY:12904575-12905076   & CD4\\_Treg\\_FCRL3    & Underweight & Overweight & chrY  &  12904575 &  12905076 & down\\\\\n",
       "\t  5 & 1.316137e-26 & -1.8496298 & 4.449016e-21 & Underweight\\_vs\\_Normal     & chrY:12904575-12905076   & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chrY  &  12904575 &  12905076 & down\\\\\n",
       "\t  6 & 5.311663e-25 & -2.7756344 & 1.795533e-19 & Underweight\\_vs\\_Normal     & chrY:19567010-19567511   & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chrY  &  19567010 &  19567511 & down\\\\\n",
       "\t  7 & 8.764596e-20 & -1.7067677 & 2.962749e-14 & Underweight\\_vs\\_Normal     & chrY:20575413-20575914   & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chrY  &  20575413 &  20575914 & down\\\\\n",
       "\t  8 & 3.747125e-19 & -2.7628132 & 1.266663e-13 & Underweight\\_vs\\_Normal     & chrY:12662009-12662510   & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chrY  &  12662009 &  12662510 & down\\\\\n",
       "\t  9 & 8.780786e-17 & -2.3446464 & 2.968222e-11 & Underweight\\_vs\\_Normal     & chrY:7273539-7274040     & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chrY  &   7273539 &   7274040 & down\\\\\n",
       "\t 10 & 4.452406e-16 & -2.2751276 & 1.505073e-10 & Underweight\\_vs\\_Normal     & chrY:21240194-21240695   & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chrY  &  21240194 &  21240695 & down\\\\\n",
       "\t 11 & 2.697797e-15 & -2.8051588 & 9.119526e-10 & Underweight\\_vs\\_Normal     & chrY:8535695-8536196     & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chrY  &   8535695 &   8536196 & down\\\\\n",
       "\t 12 & 2.884100e-14 & -2.3366449 & 9.749297e-09 & Underweight\\_vs\\_Normal     & chrY:13479855-13480356   & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chrY  &  13479855 &  13480356 & down\\\\\n",
       "\t 13 & 1.085742e-13 & -2.1524231 & 3.670199e-08 & Underweight\\_vs\\_Normal     & chrY:19567551-19568052   & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chrY  &  19567551 &  19568052 & down\\\\\n",
       "\t 14 & 1.294381e-13 & -2.2614920 & 4.375474e-08 & Underweight\\_vs\\_Normal     & chrY:2935801-2936302     & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chrY  &   2935801 &   2936302 & down\\\\\n",
       "\t 15 & 1.425224e-12 &  0.9795291 & 4.817772e-07 & Underweight\\_vs\\_Normal     & chr8:2133762-2134263     & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chr8  &   2133762 &   2134263 & up  \\\\\n",
       "\t 16 & 4.264866e-12 & -2.3683660 & 1.441678e-06 & Underweight\\_vs\\_Normal     & chrY:12905426-12905927   & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chrY  &  12905426 &  12905927 & down\\\\\n",
       "\t 17 & 1.101998e-11 & -2.6900289 & 3.725149e-06 & Underweight\\_vs\\_Normal     & chrY:2934881-2935382     & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chrY  &   2934881 &   2935382 & down\\\\\n",
       "\t 18 & 1.455245e-11 & -3.2319445 & 4.919254e-06 & Underweight\\_vs\\_Normal     & chrY:21249476-21249977   & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chrY  &  21249476 &  21249977 & down\\\\\n",
       "\t 19 & 5.140688e-11 &  0.6653898 & 1.737738e-05 & Underweight\\_vs\\_Normal     & chrX:73851023-73851524   & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chrX  &  73851023 &  73851524 & up  \\\\\n",
       "\t 20 & 3.149590e-10 & -2.7421408 & 1.064675e-04 & Underweight\\_vs\\_Normal     & chrY:13703557-13704058   & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chrY  &  13703557 &  13704058 & down\\\\\n",
       "\t 21 & 5.443223e-10 & -2.1728118 & 1.840005e-04 & Underweight\\_vs\\_Normal     & chrY:21256797-21257298   & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chrY  &  21256797 &  21257298 & down\\\\\n",
       "\t 22 & 6.260554e-10 &  1.0074009 & 2.116292e-04 & Underweight\\_vs\\_Normal     & chr17:83083463-83083964  & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chr17 &  83083463 &  83083964 & up  \\\\\n",
       "\t 23 & 1.527570e-09 & -2.2332651 & 5.163738e-04 & Underweight\\_vs\\_Normal     & chrY:20798332-20798833   & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chrY  &  20798332 &  20798833 & down\\\\\n",
       "\t 24 & 5.358021e-09 &  0.5625515 & 1.811204e-03 & Underweight\\_vs\\_Normal     & chr17:1607284-1607785    & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chr17 &   1607284 &   1607785 & up  \\\\\n",
       "\t 25 & 8.781393e-09 &  0.9031823 & 2.968427e-03 & Underweight\\_vs\\_Normal     & chr10:22505988-22506489  & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chr10 &  22505988 &  22506489 & up  \\\\\n",
       "\t 26 & 1.625563e-08 &  0.8910524 & 5.494988e-03 & Underweight\\_vs\\_Normal     & chr15:69881734-69882235  & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chr15 &  69881734 &  69882235 & up  \\\\\n",
       "\t 27 & 1.841356e-08 &  1.2060734 & 6.224447e-03 & Underweight\\_vs\\_Normal     & chr3:69984627-69985128   & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chr3  &  69984627 &  69985128 & up  \\\\\n",
       "\t 28 & 1.908690e-08 & -2.4654723 & 6.452060e-03 & Underweight\\_vs\\_Normal     & chr15:27865591-27866092  & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chr15 &  27865591 &  27866092 & down\\\\\n",
       "\t 29 & 1.988223e-08 &  1.0965137 & 6.720908e-03 & Underweight\\_vs\\_Normal     & chr9:130352049-130352550 & Atypical\\_Bm\\_ITGAX & Underweight & Normal     & chr9  & 130352049 & 130352550 & up  \\\\\n",
       "\t ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮\\\\\n",
       "\t 43508 & 1.919081e-07 & 10.285568 & 0.06487184 & Underweight\\_vs\\_Overweight & chr4:57529425-57529926    & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr4  &  57529425 &  57529926 & up  \\\\\n",
       "\t 43509 & 1.920635e-07 & 10.300419 & 0.06492436 & Underweight\\_vs\\_Overweight & chr1:184427558-184428059  & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr1  & 184427558 & 184428059 & up  \\\\\n",
       "\t 43510 & 1.925364e-07 &  2.854993 & 0.06508422 & Underweight\\_vs\\_Overweight & chr2:164739926-164740427  & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr2  & 164739926 & 164740427 & up  \\\\\n",
       "\t 43511 & 1.930047e-07 & 10.312927 & 0.06524254 & Underweight\\_vs\\_Overweight & chr1:76868036-76868537    & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr1  &  76868036 &  76868537 & up  \\\\\n",
       "\t 43512 & 1.939351e-07 & 10.346625 & 0.06555704 & Underweight\\_vs\\_Overweight & chr2:63125560-63126061    & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr2  &  63125560 &  63126061 & up  \\\\\n",
       "\t 43513 & 1.943840e-07 & 10.340085 & 0.06570880 & Underweight\\_vs\\_Overweight & chr6:89244161-89244662    & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr6  &  89244161 &  89244662 & up  \\\\\n",
       "\t 43514 & 1.944907e-07 &  4.163310 & 0.06574487 & Underweight\\_vs\\_Overweight & chr6:90402163-90402664    & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr6  &  90402163 &  90402664 & up  \\\\\n",
       "\t 43515 & 1.965275e-07 & 10.333976 & 0.06643335 & Underweight\\_vs\\_Overweight & chr15:55777044-55777545   & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr15 &  55777044 &  55777545 & up  \\\\\n",
       "\t 43516 & 1.967347e-07 & 10.299023 & 0.06650342 & Underweight\\_vs\\_Overweight & chr1:60792940-60793441    & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr1  &  60792940 &  60793441 & up  \\\\\n",
       "\t 43517 & 1.970261e-07 & 10.287704 & 0.06660192 & Underweight\\_vs\\_Overweight & chr4:11610564-11611065    & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr4  &  11610564 &  11611065 & up  \\\\\n",
       "\t 43518 & 1.977493e-07 & 10.307856 & 0.06684640 & Underweight\\_vs\\_Overweight & chr4:120575210-120575711  & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr4  & 120575210 & 120575711 & up  \\\\\n",
       "\t 43519 & 1.977906e-07 & 10.356051 & 0.06686034 & Underweight\\_vs\\_Overweight & chr8:125660798-125661299  & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr8  & 125660798 & 125661299 & up  \\\\\n",
       "\t 43520 & 1.989527e-07 & 10.366777 & 0.06725319 & Underweight\\_vs\\_Overweight & chr11:94928559-94929060   & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr11 &  94928559 &  94929060 & up  \\\\\n",
       "\t 43521 & 1.991894e-07 & 10.329678 & 0.06733317 & Underweight\\_vs\\_Overweight & chr6:76917140-76917641    & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr6  &  76917140 &  76917641 & up  \\\\\n",
       "\t 43522 & 1.997892e-07 & 10.343486 & 0.06753595 & Underweight\\_vs\\_Overweight & chr12:92137156-92137657   & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr12 &  92137156 &  92137657 & up  \\\\\n",
       "\t 43523 & 2.007245e-07 & 10.336654 & 0.06785211 & Underweight\\_vs\\_Overweight & chr3:111841047-111841548  & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr3  & 111841047 & 111841548 & up  \\\\\n",
       "\t 43524 & 2.013605e-07 & 10.371312 & 0.06806711 & Underweight\\_vs\\_Overweight & chr2:136422487-136422988  & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr2  & 136422487 & 136422988 & up  \\\\\n",
       "\t 43525 & 2.015351e-07 & 10.379426 & 0.06812612 & Underweight\\_vs\\_Overweight & chr4:146155292-146155793  & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr4  & 146155292 & 146155793 & up  \\\\\n",
       "\t 43526 & 2.026111e-07 & 10.361621 & 0.06848984 & Underweight\\_vs\\_Overweight & chr14:97921242-97921743   & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr14 &  97921242 &  97921743 & up  \\\\\n",
       "\t 43527 & 2.026805e-07 & 10.323577 & 0.06851329 & Underweight\\_vs\\_Overweight & chr10:45456745-45457246   & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr10 &  45456745 &  45457246 & up  \\\\\n",
       "\t 43528 & 2.032085e-07 & 10.337345 & 0.06869179 & Underweight\\_vs\\_Overweight & chr4:40111989-40112490    & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr4  &  40111989 &  40112490 & up  \\\\\n",
       "\t 43529 & 2.034039e-07 & 10.379938 & 0.06875785 & Underweight\\_vs\\_Overweight & chr13:109984802-109985303 & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr13 & 109984802 & 109985303 & up  \\\\\n",
       "\t 43530 & 2.041574e-07 & 10.342093 & 0.06901255 & Underweight\\_vs\\_Overweight & chrX:43350827-43351328    & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chrX  &  43350827 &  43351328 & up  \\\\\n",
       "\t 43531 & 2.054669e-07 & 10.381156 & 0.06945522 & Underweight\\_vs\\_Overweight & chr13:97413574-97414075   & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr13 &  97413574 &  97414075 & up  \\\\\n",
       "\t 43532 & 2.058333e-07 & 10.380163 & 0.06957907 & Underweight\\_vs\\_Overweight & chr9:33271029-33271530    & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr9  &  33271029 &  33271530 & up  \\\\\n",
       "\t 43533 & 2.092217e-07 & 10.388479 & 0.07072446 & Underweight\\_vs\\_Overweight & chr16:81578042-81578543   & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr16 &  81578042 &  81578543 & up  \\\\\n",
       "\t 43534 & 2.447439e-07 &  4.547295 & 0.08273224 & Underweight\\_vs\\_Overweight & chr7:44631362-44631863    & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr7  &  44631362 &  44631863 & up  \\\\\n",
       "\t 43535 & 2.463935e-07 & -2.429886 & 0.08328988 & Underweight\\_vs\\_Overweight & chr5:74839170-74839671    & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr5  &  74839170 &  74839671 & down\\\\\n",
       "\t 43536 & 2.574729e-07 &  3.842864 & 0.08703511 & Underweight\\_vs\\_Overweight & chr20:45405444-45405945   & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr20 &  45405444 &  45405945 & up  \\\\\n",
       "\t 43537 & 2.666070e-07 &  4.368078 & 0.09012276 & Underweight\\_vs\\_Overweight & chr20:49750515-49751016   & Unswitched\\_Bm\\_CD1C & Underweight & Overweight & chr20 &  49750515 &  49751016 & up  \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 33529 × 13\n",
       "\n",
       "| V1 &lt;int&gt; | V2 &lt;dbl&gt; | V3 &lt;dbl&gt; | V4 &lt;dbl&gt; | V5 &lt;chr&gt; | V6 &lt;chr&gt; | V7 &lt;chr&gt; | V8 &lt;chr&gt; | V9 &lt;chr&gt; | V10 &lt;chr&gt; | V11 &lt;int&gt; | V12 &lt;int&gt; | V13 &lt;chr&gt; |\n",
       "|---|---|---|---|---|---|---|---|---|---|---|---|---|\n",
       "|  0 | 1.799742e-15 | -4.3480747 | 6.083777e-10 | Underweight_vs_Normal     | chrY:12904575-12905076   | CD4_Treg_FCRL3    | Underweight | Normal     | chrY  |  12904575 |  12905076 | down |\n",
       "|  1 | 4.509594e-11 | -2.6346098 | 1.524405e-05 | Underweight_vs_Normal     | chrY:20575413-20575914   | CD4_Treg_FCRL3    | Underweight | Normal     | chrY  |  20575413 |  20575914 | down |\n",
       "|  2 | 4.035983e-09 | -2.3952563 | 1.364308e-03 | Underweight_vs_Normal     | chrY:7273539-7274040     | CD4_Treg_FCRL3    | Underweight | Normal     | chrY  |   7273539 |   7274040 | down |\n",
       "|  3 | 4.173088e-09 | -3.3596235 | 1.410654e-03 | Underweight_vs_Overweight | chrY:20575413-20575914   | CD4_Treg_FCRL3    | Underweight | Overweight | chrY  |  20575413 |  20575914 | down |\n",
       "|  4 | 9.946644e-08 | -4.4064829 | 3.362324e-02 | Underweight_vs_Overweight | chrY:12904575-12905076   | CD4_Treg_FCRL3    | Underweight | Overweight | chrY  |  12904575 |  12905076 | down |\n",
       "|  5 | 1.316137e-26 | -1.8496298 | 4.449016e-21 | Underweight_vs_Normal     | chrY:12904575-12905076   | Atypical_Bm_ITGAX | Underweight | Normal     | chrY  |  12904575 |  12905076 | down |\n",
       "|  6 | 5.311663e-25 | -2.7756344 | 1.795533e-19 | Underweight_vs_Normal     | chrY:19567010-19567511   | Atypical_Bm_ITGAX | Underweight | Normal     | chrY  |  19567010 |  19567511 | down |\n",
       "|  7 | 8.764596e-20 | -1.7067677 | 2.962749e-14 | Underweight_vs_Normal     | chrY:20575413-20575914   | Atypical_Bm_ITGAX | Underweight | Normal     | chrY  |  20575413 |  20575914 | down |\n",
       "|  8 | 3.747125e-19 | -2.7628132 | 1.266663e-13 | Underweight_vs_Normal     | chrY:12662009-12662510   | Atypical_Bm_ITGAX | Underweight | Normal     | chrY  |  12662009 |  12662510 | down |\n",
       "|  9 | 8.780786e-17 | -2.3446464 | 2.968222e-11 | Underweight_vs_Normal     | chrY:7273539-7274040     | Atypical_Bm_ITGAX | Underweight | Normal     | chrY  |   7273539 |   7274040 | down |\n",
       "| 10 | 4.452406e-16 | -2.2751276 | 1.505073e-10 | Underweight_vs_Normal     | chrY:21240194-21240695   | Atypical_Bm_ITGAX | Underweight | Normal     | chrY  |  21240194 |  21240695 | down |\n",
       "| 11 | 2.697797e-15 | -2.8051588 | 9.119526e-10 | Underweight_vs_Normal     | chrY:8535695-8536196     | Atypical_Bm_ITGAX | Underweight | Normal     | chrY  |   8535695 |   8536196 | down |\n",
       "| 12 | 2.884100e-14 | -2.3366449 | 9.749297e-09 | Underweight_vs_Normal     | chrY:13479855-13480356   | Atypical_Bm_ITGAX | Underweight | Normal     | chrY  |  13479855 |  13480356 | down |\n",
       "| 13 | 1.085742e-13 | -2.1524231 | 3.670199e-08 | Underweight_vs_Normal     | chrY:19567551-19568052   | Atypical_Bm_ITGAX | Underweight | Normal     | chrY  |  19567551 |  19568052 | down |\n",
       "| 14 | 1.294381e-13 | -2.2614920 | 4.375474e-08 | Underweight_vs_Normal     | chrY:2935801-2936302     | Atypical_Bm_ITGAX | Underweight | Normal     | chrY  |   2935801 |   2936302 | down |\n",
       "| 15 | 1.425224e-12 |  0.9795291 | 4.817772e-07 | Underweight_vs_Normal     | chr8:2133762-2134263     | Atypical_Bm_ITGAX | Underweight | Normal     | chr8  |   2133762 |   2134263 | up   |\n",
       "| 16 | 4.264866e-12 | -2.3683660 | 1.441678e-06 | Underweight_vs_Normal     | chrY:12905426-12905927   | Atypical_Bm_ITGAX | Underweight | Normal     | chrY  |  12905426 |  12905927 | down |\n",
       "| 17 | 1.101998e-11 | -2.6900289 | 3.725149e-06 | Underweight_vs_Normal     | chrY:2934881-2935382     | Atypical_Bm_ITGAX | Underweight | Normal     | chrY  |   2934881 |   2935382 | down |\n",
       "| 18 | 1.455245e-11 | -3.2319445 | 4.919254e-06 | Underweight_vs_Normal     | chrY:21249476-21249977   | Atypical_Bm_ITGAX | Underweight | Normal     | chrY  |  21249476 |  21249977 | down |\n",
       "| 19 | 5.140688e-11 |  0.6653898 | 1.737738e-05 | Underweight_vs_Normal     | chrX:73851023-73851524   | Atypical_Bm_ITGAX | Underweight | Normal     | chrX  |  73851023 |  73851524 | up   |\n",
       "| 20 | 3.149590e-10 | -2.7421408 | 1.064675e-04 | Underweight_vs_Normal     | chrY:13703557-13704058   | Atypical_Bm_ITGAX | Underweight | Normal     | chrY  |  13703557 |  13704058 | down |\n",
       "| 21 | 5.443223e-10 | -2.1728118 | 1.840005e-04 | Underweight_vs_Normal     | chrY:21256797-21257298   | Atypical_Bm_ITGAX | Underweight | Normal     | chrY  |  21256797 |  21257298 | down |\n",
       "| 22 | 6.260554e-10 |  1.0074009 | 2.116292e-04 | Underweight_vs_Normal     | chr17:83083463-83083964  | Atypical_Bm_ITGAX | Underweight | Normal     | chr17 |  83083463 |  83083964 | up   |\n",
       "| 23 | 1.527570e-09 | -2.2332651 | 5.163738e-04 | Underweight_vs_Normal     | chrY:20798332-20798833   | Atypical_Bm_ITGAX | Underweight | Normal     | chrY  |  20798332 |  20798833 | down |\n",
       "| 24 | 5.358021e-09 |  0.5625515 | 1.811204e-03 | Underweight_vs_Normal     | chr17:1607284-1607785    | Atypical_Bm_ITGAX | Underweight | Normal     | chr17 |   1607284 |   1607785 | up   |\n",
       "| 25 | 8.781393e-09 |  0.9031823 | 2.968427e-03 | Underweight_vs_Normal     | chr10:22505988-22506489  | Atypical_Bm_ITGAX | Underweight | Normal     | chr10 |  22505988 |  22506489 | up   |\n",
       "| 26 | 1.625563e-08 |  0.8910524 | 5.494988e-03 | Underweight_vs_Normal     | chr15:69881734-69882235  | Atypical_Bm_ITGAX | Underweight | Normal     | chr15 |  69881734 |  69882235 | up   |\n",
       "| 27 | 1.841356e-08 |  1.2060734 | 6.224447e-03 | Underweight_vs_Normal     | chr3:69984627-69985128   | Atypical_Bm_ITGAX | Underweight | Normal     | chr3  |  69984627 |  69985128 | up   |\n",
       "| 28 | 1.908690e-08 | -2.4654723 | 6.452060e-03 | Underweight_vs_Normal     | chr15:27865591-27866092  | Atypical_Bm_ITGAX | Underweight | Normal     | chr15 |  27865591 |  27866092 | down |\n",
       "| 29 | 1.988223e-08 |  1.0965137 | 6.720908e-03 | Underweight_vs_Normal     | chr9:130352049-130352550 | Atypical_Bm_ITGAX | Underweight | Normal     | chr9  | 130352049 | 130352550 | up   |\n",
       "| ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ |\n",
       "| 43508 | 1.919081e-07 | 10.285568 | 0.06487184 | Underweight_vs_Overweight | chr4:57529425-57529926    | Unswitched_Bm_CD1C | Underweight | Overweight | chr4  |  57529425 |  57529926 | up   |\n",
       "| 43509 | 1.920635e-07 | 10.300419 | 0.06492436 | Underweight_vs_Overweight | chr1:184427558-184428059  | Unswitched_Bm_CD1C | Underweight | Overweight | chr1  | 184427558 | 184428059 | up   |\n",
       "| 43510 | 1.925364e-07 |  2.854993 | 0.06508422 | Underweight_vs_Overweight | chr2:164739926-164740427  | Unswitched_Bm_CD1C | Underweight | Overweight | chr2  | 164739926 | 164740427 | up   |\n",
       "| 43511 | 1.930047e-07 | 10.312927 | 0.06524254 | Underweight_vs_Overweight | chr1:76868036-76868537    | Unswitched_Bm_CD1C | Underweight | Overweight | chr1  |  76868036 |  76868537 | up   |\n",
       "| 43512 | 1.939351e-07 | 10.346625 | 0.06555704 | Underweight_vs_Overweight | chr2:63125560-63126061    | Unswitched_Bm_CD1C | Underweight | Overweight | chr2  |  63125560 |  63126061 | up   |\n",
       "| 43513 | 1.943840e-07 | 10.340085 | 0.06570880 | Underweight_vs_Overweight | chr6:89244161-89244662    | Unswitched_Bm_CD1C | Underweight | Overweight | chr6  |  89244161 |  89244662 | up   |\n",
       "| 43514 | 1.944907e-07 |  4.163310 | 0.06574487 | Underweight_vs_Overweight | chr6:90402163-90402664    | Unswitched_Bm_CD1C | Underweight | Overweight | chr6  |  90402163 |  90402664 | up   |\n",
       "| 43515 | 1.965275e-07 | 10.333976 | 0.06643335 | Underweight_vs_Overweight | chr15:55777044-55777545   | Unswitched_Bm_CD1C | Underweight | Overweight | chr15 |  55777044 |  55777545 | up   |\n",
       "| 43516 | 1.967347e-07 | 10.299023 | 0.06650342 | Underweight_vs_Overweight | chr1:60792940-60793441    | Unswitched_Bm_CD1C | Underweight | Overweight | chr1  |  60792940 |  60793441 | up   |\n",
       "| 43517 | 1.970261e-07 | 10.287704 | 0.06660192 | Underweight_vs_Overweight | chr4:11610564-11611065    | Unswitched_Bm_CD1C | Underweight | Overweight | chr4  |  11610564 |  11611065 | up   |\n",
       "| 43518 | 1.977493e-07 | 10.307856 | 0.06684640 | Underweight_vs_Overweight | chr4:120575210-120575711  | Unswitched_Bm_CD1C | Underweight | Overweight | chr4  | 120575210 | 120575711 | up   |\n",
       "| 43519 | 1.977906e-07 | 10.356051 | 0.06686034 | Underweight_vs_Overweight | chr8:125660798-125661299  | Unswitched_Bm_CD1C | Underweight | Overweight | chr8  | 125660798 | 125661299 | up   |\n",
       "| 43520 | 1.989527e-07 | 10.366777 | 0.06725319 | Underweight_vs_Overweight | chr11:94928559-94929060   | Unswitched_Bm_CD1C | Underweight | Overweight | chr11 |  94928559 |  94929060 | up   |\n",
       "| 43521 | 1.991894e-07 | 10.329678 | 0.06733317 | Underweight_vs_Overweight | chr6:76917140-76917641    | Unswitched_Bm_CD1C | Underweight | Overweight | chr6  |  76917140 |  76917641 | up   |\n",
       "| 43522 | 1.997892e-07 | 10.343486 | 0.06753595 | Underweight_vs_Overweight | chr12:92137156-92137657   | Unswitched_Bm_CD1C | Underweight | Overweight | chr12 |  92137156 |  92137657 | up   |\n",
       "| 43523 | 2.007245e-07 | 10.336654 | 0.06785211 | Underweight_vs_Overweight | chr3:111841047-111841548  | Unswitched_Bm_CD1C | Underweight | Overweight | chr3  | 111841047 | 111841548 | up   |\n",
       "| 43524 | 2.013605e-07 | 10.371312 | 0.06806711 | Underweight_vs_Overweight | chr2:136422487-136422988  | Unswitched_Bm_CD1C | Underweight | Overweight | chr2  | 136422487 | 136422988 | up   |\n",
       "| 43525 | 2.015351e-07 | 10.379426 | 0.06812612 | Underweight_vs_Overweight | chr4:146155292-146155793  | Unswitched_Bm_CD1C | Underweight | Overweight | chr4  | 146155292 | 146155793 | up   |\n",
       "| 43526 | 2.026111e-07 | 10.361621 | 0.06848984 | Underweight_vs_Overweight | chr14:97921242-97921743   | Unswitched_Bm_CD1C | Underweight | Overweight | chr14 |  97921242 |  97921743 | up   |\n",
       "| 43527 | 2.026805e-07 | 10.323577 | 0.06851329 | Underweight_vs_Overweight | chr10:45456745-45457246   | Unswitched_Bm_CD1C | Underweight | Overweight | chr10 |  45456745 |  45457246 | up   |\n",
       "| 43528 | 2.032085e-07 | 10.337345 | 0.06869179 | Underweight_vs_Overweight | chr4:40111989-40112490    | Unswitched_Bm_CD1C | Underweight | Overweight | chr4  |  40111989 |  40112490 | up   |\n",
       "| 43529 | 2.034039e-07 | 10.379938 | 0.06875785 | Underweight_vs_Overweight | chr13:109984802-109985303 | Unswitched_Bm_CD1C | Underweight | Overweight | chr13 | 109984802 | 109985303 | up   |\n",
       "| 43530 | 2.041574e-07 | 10.342093 | 0.06901255 | Underweight_vs_Overweight | chrX:43350827-43351328    | Unswitched_Bm_CD1C | Underweight | Overweight | chrX  |  43350827 |  43351328 | up   |\n",
       "| 43531 | 2.054669e-07 | 10.381156 | 0.06945522 | Underweight_vs_Overweight | chr13:97413574-97414075   | Unswitched_Bm_CD1C | Underweight | Overweight | chr13 |  97413574 |  97414075 | up   |\n",
       "| 43532 | 2.058333e-07 | 10.380163 | 0.06957907 | Underweight_vs_Overweight | chr9:33271029-33271530    | Unswitched_Bm_CD1C | Underweight | Overweight | chr9  |  33271029 |  33271530 | up   |\n",
       "| 43533 | 2.092217e-07 | 10.388479 | 0.07072446 | Underweight_vs_Overweight | chr16:81578042-81578543   | Unswitched_Bm_CD1C | Underweight | Overweight | chr16 |  81578042 |  81578543 | up   |\n",
       "| 43534 | 2.447439e-07 |  4.547295 | 0.08273224 | Underweight_vs_Overweight | chr7:44631362-44631863    | Unswitched_Bm_CD1C | Underweight | Overweight | chr7  |  44631362 |  44631863 | up   |\n",
       "| 43535 | 2.463935e-07 | -2.429886 | 0.08328988 | Underweight_vs_Overweight | chr5:74839170-74839671    | Unswitched_Bm_CD1C | Underweight | Overweight | chr5  |  74839170 |  74839671 | down |\n",
       "| 43536 | 2.574729e-07 |  3.842864 | 0.08703511 | Underweight_vs_Overweight | chr20:45405444-45405945   | Unswitched_Bm_CD1C | Underweight | Overweight | chr20 |  45405444 |  45405945 | up   |\n",
       "| 43537 | 2.666070e-07 |  4.368078 | 0.09012276 | Underweight_vs_Overweight | chr20:49750515-49751016   | Unswitched_Bm_CD1C | Underweight | Overweight | chr20 |  49750515 |  49751016 | up   |\n",
       "\n"
      ],
      "text/plain": [
       "      V1    V2           V3         V4           V5                       \n",
       "1      0    1.799742e-15 -4.3480747 6.083777e-10 Underweight_vs_Normal    \n",
       "2      1    4.509594e-11 -2.6346098 1.524405e-05 Underweight_vs_Normal    \n",
       "3      2    4.035983e-09 -2.3952563 1.364308e-03 Underweight_vs_Normal    \n",
       "4      3    4.173088e-09 -3.3596235 1.410654e-03 Underweight_vs_Overweight\n",
       "5      4    9.946644e-08 -4.4064829 3.362324e-02 Underweight_vs_Overweight\n",
       "6      5    1.316137e-26 -1.8496298 4.449016e-21 Underweight_vs_Normal    \n",
       "7      6    5.311663e-25 -2.7756344 1.795533e-19 Underweight_vs_Normal    \n",
       "8      7    8.764596e-20 -1.7067677 2.962749e-14 Underweight_vs_Normal    \n",
       "9      8    3.747125e-19 -2.7628132 1.266663e-13 Underweight_vs_Normal    \n",
       "10     9    8.780786e-17 -2.3446464 2.968222e-11 Underweight_vs_Normal    \n",
       "11    10    4.452406e-16 -2.2751276 1.505073e-10 Underweight_vs_Normal    \n",
       "12    11    2.697797e-15 -2.8051588 9.119526e-10 Underweight_vs_Normal    \n",
       "13    12    2.884100e-14 -2.3366449 9.749297e-09 Underweight_vs_Normal    \n",
       "14    13    1.085742e-13 -2.1524231 3.670199e-08 Underweight_vs_Normal    \n",
       "15    14    1.294381e-13 -2.2614920 4.375474e-08 Underweight_vs_Normal    \n",
       "16    15    1.425224e-12  0.9795291 4.817772e-07 Underweight_vs_Normal    \n",
       "17    16    4.264866e-12 -2.3683660 1.441678e-06 Underweight_vs_Normal    \n",
       "18    17    1.101998e-11 -2.6900289 3.725149e-06 Underweight_vs_Normal    \n",
       "19    18    1.455245e-11 -3.2319445 4.919254e-06 Underweight_vs_Normal    \n",
       "20    19    5.140688e-11  0.6653898 1.737738e-05 Underweight_vs_Normal    \n",
       "21    20    3.149590e-10 -2.7421408 1.064675e-04 Underweight_vs_Normal    \n",
       "22    21    5.443223e-10 -2.1728118 1.840005e-04 Underweight_vs_Normal    \n",
       "23    22    6.260554e-10  1.0074009 2.116292e-04 Underweight_vs_Normal    \n",
       "24    23    1.527570e-09 -2.2332651 5.163738e-04 Underweight_vs_Normal    \n",
       "25    24    5.358021e-09  0.5625515 1.811204e-03 Underweight_vs_Normal    \n",
       "26    25    8.781393e-09  0.9031823 2.968427e-03 Underweight_vs_Normal    \n",
       "27    26    1.625563e-08  0.8910524 5.494988e-03 Underweight_vs_Normal    \n",
       "28    27    1.841356e-08  1.2060734 6.224447e-03 Underweight_vs_Normal    \n",
       "29    28    1.908690e-08 -2.4654723 6.452060e-03 Underweight_vs_Normal    \n",
       "30    29    1.988223e-08  1.0965137 6.720908e-03 Underweight_vs_Normal    \n",
       "⋮     ⋮     ⋮            ⋮          ⋮            ⋮                        \n",
       "33500 43508 1.919081e-07 10.285568  0.06487184   Underweight_vs_Overweight\n",
       "33501 43509 1.920635e-07 10.300419  0.06492436   Underweight_vs_Overweight\n",
       "33502 43510 1.925364e-07  2.854993  0.06508422   Underweight_vs_Overweight\n",
       "33503 43511 1.930047e-07 10.312927  0.06524254   Underweight_vs_Overweight\n",
       "33504 43512 1.939351e-07 10.346625  0.06555704   Underweight_vs_Overweight\n",
       "33505 43513 1.943840e-07 10.340085  0.06570880   Underweight_vs_Overweight\n",
       "33506 43514 1.944907e-07  4.163310  0.06574487   Underweight_vs_Overweight\n",
       "33507 43515 1.965275e-07 10.333976  0.06643335   Underweight_vs_Overweight\n",
       "33508 43516 1.967347e-07 10.299023  0.06650342   Underweight_vs_Overweight\n",
       "33509 43517 1.970261e-07 10.287704  0.06660192   Underweight_vs_Overweight\n",
       "33510 43518 1.977493e-07 10.307856  0.06684640   Underweight_vs_Overweight\n",
       "33511 43519 1.977906e-07 10.356051  0.06686034   Underweight_vs_Overweight\n",
       "33512 43520 1.989527e-07 10.366777  0.06725319   Underweight_vs_Overweight\n",
       "33513 43521 1.991894e-07 10.329678  0.06733317   Underweight_vs_Overweight\n",
       "33514 43522 1.997892e-07 10.343486  0.06753595   Underweight_vs_Overweight\n",
       "33515 43523 2.007245e-07 10.336654  0.06785211   Underweight_vs_Overweight\n",
       "33516 43524 2.013605e-07 10.371312  0.06806711   Underweight_vs_Overweight\n",
       "33517 43525 2.015351e-07 10.379426  0.06812612   Underweight_vs_Overweight\n",
       "33518 43526 2.026111e-07 10.361621  0.06848984   Underweight_vs_Overweight\n",
       "33519 43527 2.026805e-07 10.323577  0.06851329   Underweight_vs_Overweight\n",
       "33520 43528 2.032085e-07 10.337345  0.06869179   Underweight_vs_Overweight\n",
       "33521 43529 2.034039e-07 10.379938  0.06875785   Underweight_vs_Overweight\n",
       "33522 43530 2.041574e-07 10.342093  0.06901255   Underweight_vs_Overweight\n",
       "33523 43531 2.054669e-07 10.381156  0.06945522   Underweight_vs_Overweight\n",
       "33524 43532 2.058333e-07 10.380163  0.06957907   Underweight_vs_Overweight\n",
       "33525 43533 2.092217e-07 10.388479  0.07072446   Underweight_vs_Overweight\n",
       "33526 43534 2.447439e-07  4.547295  0.08273224   Underweight_vs_Overweight\n",
       "33527 43535 2.463935e-07 -2.429886  0.08328988   Underweight_vs_Overweight\n",
       "33528 43536 2.574729e-07  3.842864  0.08703511   Underweight_vs_Overweight\n",
       "33529 43537 2.666070e-07  4.368078  0.09012276   Underweight_vs_Overweight\n",
       "      V6                        V7                 V8          V9         V10  \n",
       "1     chrY:12904575-12905076    CD4_Treg_FCRL3     Underweight Normal     chrY \n",
       "2     chrY:20575413-20575914    CD4_Treg_FCRL3     Underweight Normal     chrY \n",
       "3     chrY:7273539-7274040      CD4_Treg_FCRL3     Underweight Normal     chrY \n",
       "4     chrY:20575413-20575914    CD4_Treg_FCRL3     Underweight Overweight chrY \n",
       "5     chrY:12904575-12905076    CD4_Treg_FCRL3     Underweight Overweight chrY \n",
       "6     chrY:12904575-12905076    Atypical_Bm_ITGAX  Underweight Normal     chrY \n",
       "7     chrY:19567010-19567511    Atypical_Bm_ITGAX  Underweight Normal     chrY \n",
       "8     chrY:20575413-20575914    Atypical_Bm_ITGAX  Underweight Normal     chrY \n",
       "9     chrY:12662009-12662510    Atypical_Bm_ITGAX  Underweight Normal     chrY \n",
       "10    chrY:7273539-7274040      Atypical_Bm_ITGAX  Underweight Normal     chrY \n",
       "11    chrY:21240194-21240695    Atypical_Bm_ITGAX  Underweight Normal     chrY \n",
       "12    chrY:8535695-8536196      Atypical_Bm_ITGAX  Underweight Normal     chrY \n",
       "13    chrY:13479855-13480356    Atypical_Bm_ITGAX  Underweight Normal     chrY \n",
       "14    chrY:19567551-19568052    Atypical_Bm_ITGAX  Underweight Normal     chrY \n",
       "15    chrY:2935801-2936302      Atypical_Bm_ITGAX  Underweight Normal     chrY \n",
       "16    chr8:2133762-2134263      Atypical_Bm_ITGAX  Underweight Normal     chr8 \n",
       "17    chrY:12905426-12905927    Atypical_Bm_ITGAX  Underweight Normal     chrY \n",
       "18    chrY:2934881-2935382      Atypical_Bm_ITGAX  Underweight Normal     chrY \n",
       "19    chrY:21249476-21249977    Atypical_Bm_ITGAX  Underweight Normal     chrY \n",
       "20    chrX:73851023-73851524    Atypical_Bm_ITGAX  Underweight Normal     chrX \n",
       "21    chrY:13703557-13704058    Atypical_Bm_ITGAX  Underweight Normal     chrY \n",
       "22    chrY:21256797-21257298    Atypical_Bm_ITGAX  Underweight Normal     chrY \n",
       "23    chr17:83083463-83083964   Atypical_Bm_ITGAX  Underweight Normal     chr17\n",
       "24    chrY:20798332-20798833    Atypical_Bm_ITGAX  Underweight Normal     chrY \n",
       "25    chr17:1607284-1607785     Atypical_Bm_ITGAX  Underweight Normal     chr17\n",
       "26    chr10:22505988-22506489   Atypical_Bm_ITGAX  Underweight Normal     chr10\n",
       "27    chr15:69881734-69882235   Atypical_Bm_ITGAX  Underweight Normal     chr15\n",
       "28    chr3:69984627-69985128    Atypical_Bm_ITGAX  Underweight Normal     chr3 \n",
       "29    chr15:27865591-27866092   Atypical_Bm_ITGAX  Underweight Normal     chr15\n",
       "30    chr9:130352049-130352550  Atypical_Bm_ITGAX  Underweight Normal     chr9 \n",
       "⋮     ⋮                         ⋮                  ⋮           ⋮          ⋮    \n",
       "33500 chr4:57529425-57529926    Unswitched_Bm_CD1C Underweight Overweight chr4 \n",
       "33501 chr1:184427558-184428059  Unswitched_Bm_CD1C Underweight Overweight chr1 \n",
       "33502 chr2:164739926-164740427  Unswitched_Bm_CD1C Underweight Overweight chr2 \n",
       "33503 chr1:76868036-76868537    Unswitched_Bm_CD1C Underweight Overweight chr1 \n",
       "33504 chr2:63125560-63126061    Unswitched_Bm_CD1C Underweight Overweight chr2 \n",
       "33505 chr6:89244161-89244662    Unswitched_Bm_CD1C Underweight Overweight chr6 \n",
       "33506 chr6:90402163-90402664    Unswitched_Bm_CD1C Underweight Overweight chr6 \n",
       "33507 chr15:55777044-55777545   Unswitched_Bm_CD1C Underweight Overweight chr15\n",
       "33508 chr1:60792940-60793441    Unswitched_Bm_CD1C Underweight Overweight chr1 \n",
       "33509 chr4:11610564-11611065    Unswitched_Bm_CD1C Underweight Overweight chr4 \n",
       "33510 chr4:120575210-120575711  Unswitched_Bm_CD1C Underweight Overweight chr4 \n",
       "33511 chr8:125660798-125661299  Unswitched_Bm_CD1C Underweight Overweight chr8 \n",
       "33512 chr11:94928559-94929060   Unswitched_Bm_CD1C Underweight Overweight chr11\n",
       "33513 chr6:76917140-76917641    Unswitched_Bm_CD1C Underweight Overweight chr6 \n",
       "33514 chr12:92137156-92137657   Unswitched_Bm_CD1C Underweight Overweight chr12\n",
       "33515 chr3:111841047-111841548  Unswitched_Bm_CD1C Underweight Overweight chr3 \n",
       "33516 chr2:136422487-136422988  Unswitched_Bm_CD1C Underweight Overweight chr2 \n",
       "33517 chr4:146155292-146155793  Unswitched_Bm_CD1C Underweight Overweight chr4 \n",
       "33518 chr14:97921242-97921743   Unswitched_Bm_CD1C Underweight Overweight chr14\n",
       "33519 chr10:45456745-45457246   Unswitched_Bm_CD1C Underweight Overweight chr10\n",
       "33520 chr4:40111989-40112490    Unswitched_Bm_CD1C Underweight Overweight chr4 \n",
       "33521 chr13:109984802-109985303 Unswitched_Bm_CD1C Underweight Overweight chr13\n",
       "33522 chrX:43350827-43351328    Unswitched_Bm_CD1C Underweight Overweight chrX \n",
       "33523 chr13:97413574-97414075   Unswitched_Bm_CD1C Underweight Overweight chr13\n",
       "33524 chr9:33271029-33271530    Unswitched_Bm_CD1C Underweight Overweight chr9 \n",
       "33525 chr16:81578042-81578543   Unswitched_Bm_CD1C Underweight Overweight chr16\n",
       "33526 chr7:44631362-44631863    Unswitched_Bm_CD1C Underweight Overweight chr7 \n",
       "33527 chr5:74839170-74839671    Unswitched_Bm_CD1C Underweight Overweight chr5 \n",
       "33528 chr20:45405444-45405945   Unswitched_Bm_CD1C Underweight Overweight chr20\n",
       "33529 chr20:49750515-49751016   Unswitched_Bm_CD1C Underweight Overweight chr20\n",
       "      V11       V12       V13 \n",
       "1      12904575  12905076 down\n",
       "2      20575413  20575914 down\n",
       "3       7273539   7274040 down\n",
       "4      20575413  20575914 down\n",
       "5      12904575  12905076 down\n",
       "6      12904575  12905076 down\n",
       "7      19567010  19567511 down\n",
       "8      20575413  20575914 down\n",
       "9      12662009  12662510 down\n",
       "10      7273539   7274040 down\n",
       "11     21240194  21240695 down\n",
       "12      8535695   8536196 down\n",
       "13     13479855  13480356 down\n",
       "14     19567551  19568052 down\n",
       "15      2935801   2936302 down\n",
       "16      2133762   2134263 up  \n",
       "17     12905426  12905927 down\n",
       "18      2934881   2935382 down\n",
       "19     21249476  21249977 down\n",
       "20     73851023  73851524 up  \n",
       "21     13703557  13704058 down\n",
       "22     21256797  21257298 down\n",
       "23     83083463  83083964 up  \n",
       "24     20798332  20798833 down\n",
       "25      1607284   1607785 up  \n",
       "26     22505988  22506489 up  \n",
       "27     69881734  69882235 up  \n",
       "28     69984627  69985128 up  \n",
       "29     27865591  27866092 down\n",
       "30    130352049 130352550 up  \n",
       "⋮     ⋮         ⋮         ⋮   \n",
       "33500  57529425  57529926 up  \n",
       "33501 184427558 184428059 up  \n",
       "33502 164739926 164740427 up  \n",
       "33503  76868036  76868537 up  \n",
       "33504  63125560  63126061 up  \n",
       "33505  89244161  89244662 up  \n",
       "33506  90402163  90402664 up  \n",
       "33507  55777044  55777545 up  \n",
       "33508  60792940  60793441 up  \n",
       "33509  11610564  11611065 up  \n",
       "33510 120575210 120575711 up  \n",
       "33511 125660798 125661299 up  \n",
       "33512  94928559  94929060 up  \n",
       "33513  76917140  76917641 up  \n",
       "33514  92137156  92137657 up  \n",
       "33515 111841047 111841548 up  \n",
       "33516 136422487 136422988 up  \n",
       "33517 146155292 146155793 up  \n",
       "33518  97921242  97921743 up  \n",
       "33519  45456745  45457246 up  \n",
       "33520  40111989  40112490 up  \n",
       "33521 109984802 109985303 up  \n",
       "33522  43350827  43351328 up  \n",
       "33523  97413574  97414075 up  \n",
       "33524  33271029  33271530 up  \n",
       "33525  81578042  81578543 up  \n",
       "33526  44631362  44631863 up  \n",
       "33527  74839170  74839671 down\n",
       "33528  45405444  45405945 up  \n",
       "33529  49750515  49751016 up  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "peaks "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "0e417f6d-bbd3-406d-87f6-66e12ad296a9",
   "metadata": {},
   "outputs": [],
   "source": [
    "run_great_go_by_group <- function(peaks, \n",
    "                                  cell_col = \"V7\",      # 细胞类型列\n",
    "                                  contrast_col = \"V5\",  # 比较组列（如 \"Underweight_vs_Normal\"）\n",
    "                                  dir_col = \"V13\",      # 方向列（\"up\" / \"down\"）\n",
    "                                  pval_col = \"V10\",     # chr\n",
    "                                  start_col = \"V11\",    # start\n",
    "                                  end_col = \"V12\",      # end\n",
    "                                  genome = \"hg38\",\n",
    "                                  fdr_cutoff = 0.05) {\n",
    "  \n",
    "  # 提取唯一组合\n",
    "  groups <- peaks %>%\n",
    "    distinct(!!sym(cell_col), !!sym(contrast_col), !!sym(dir_col)) %>%\n",
    "    filter(!is.na(!!sym(dir_col)))  # 排除 NA\n",
    "  \n",
    "  all_results <- list()\n",
    "  \n",
    "  for (i in seq_len(nrow(groups))) {\n",
    "    ct   <- groups[[cell_col]][i]\n",
    "    comp <- groups[[contrast_col]][i]\n",
    "    dir  <- groups[[dir_col]][i]\n",
    "    \n",
    "    cat(\"Processing:\", ct, \"|\", comp, \"|\", dir, \"\\n\")\n",
    "    \n",
    "    # 筛选子集\n",
    "    sub_peaks <- peaks %>%\n",
    "      filter(!!sym(cell_col) == ct,\n",
    "             !!sym(contrast_col) == comp,\n",
    "             !!sym(dir_col) == dir) %>%\n",
    "      select(chr = !!sym(pval_col),  # 注意：V11 是 chr 列？请确认！\n",
    "             start = !!sym(start_col),\n",
    "             end = !!sym(end_col))\n",
    "    \n",
    "    # 检查是否有 peak\n",
    "    if (nrow(sub_peaks) == 0) {\n",
    "      cat(\"  -> No peaks, skipping.\\n\")\n",
    "      next\n",
    "    }\n",
    "    \n",
    "    # 转 GRanges（确保列名为 seqnames/start/end）\n",
    "    names(sub_peaks) <- c(\"seqnames\", \"start\", \"end\")\n",
    "    gr <- makeGRangesFromDataFrame(sub_peaks, ignore.strand = TRUE)\n",
    "    \n",
    "    # 提交 GREAT 任务\n",
    "    set.seed(123)\n",
    "    job <- tryCatch({\n",
    "      submitGreatJob(gr, genome = genome)\n",
    "    }, error = function(e) {\n",
    "      cat(\"  -> GREAT submission failed:\", e$message, \"\\n\")\n",
    "      return(NULL)\n",
    "    })\n",
    "    \n",
    "    if (is.null(job)) next\n",
    "    \n",
    "    # 获取 GO Molecular Function 表\n",
    "    tbl <- tryCatch({\n",
    "      getEnrichmentTables(job)\n",
    "    }, error = function(e) {\n",
    "      cat(\"  -> Failed to retrieve results:\", e$message, \"\\n\")\n",
    "      return(NULL)\n",
    "    })\n",
    "    \n",
    "    if (is.null(tbl) || is.null(tbl$`GO Molecular Function`)) {\n",
    "      cat(\"  -> No GO MF results.\\n\")\n",
    "      next\n",
    "    }\n",
    "    \n",
    "    go_df <- tbl$`GO Molecular Function`\n",
    "    \n",
    "    # 筛选显著结果\n",
    "    sig_go <- go_df %>%\n",
    "      filter(Binom_Adjp_BH < fdr_cutoff) %>%\n",
    "      mutate(\n",
    "        cell_type = ct,\n",
    "        contrast = comp,\n",
    "        direction = dir,\n",
    "        .before = everything()\n",
    "      )\n",
    "    \n",
    "    if (nrow(sig_go) > 0) {\n",
    "      all_results[[length(all_results) + 1]] <- sig_go\n",
    "    } else {\n",
    "      cat(\"  -> No significant terms.\\n\")\n",
    "    }\n",
    "  }\n",
    "  \n",
    "  # 合并所有结果\n",
    "  if (length(all_results) == 0) {\n",
    "    stop(\"No significant GO terms found for any group.\")\n",
    "  }\n",
    "  \n",
    "  final_df <- bind_rows(all_results)\n",
    "  return(final_df)\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "f038c73d-9677-484b-95c1-3fce2f578eea",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Treg_FCRL3 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Treg_FCRL3 | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Atypical_Bm_ITGAX | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |====================================================================  |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Atypical_Bm_ITGAX | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Atypical_Bm_ITGAX | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Atypical_Bm_ITGAX | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Atypical_Bm_ITGAX | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 46s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Bn_TCL1A | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Bn_TCL1A | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Bn_TCL1A | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |====================================================================  |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Bn_TCL1A | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Bn_TCL1A | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Bn_TCL1A | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_CTL_GZMH | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_CTL_GZMH | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_CTL_GZMH | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n",
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n",
      "\n",
      "failed with the request, try after 1 min (1st try)\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n",
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n",
      "\n",
      "failed with the request, try after 1 min (2nd try)\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_CTL_GZMH | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_CTL_GZMH | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_CTL_GZMH | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Tcm_CXCR5 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Tcm_CXCR5 | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Tcm_CXCR5 | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Tcm_IFI44L | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |====================================================================  |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Tcm_IFI44L | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Tcm_IFI44L | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Tcm_IFI44L | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Tcm_IFI44L | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Tcm_IFI44L | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Tem_CCR7neg | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Tem_CCR7neg | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Tem_CCR7neg | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n",
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n",
      "\n",
      "failed with the request, try after 1 min (1st try)\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n",
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n",
      "\n",
      "failed with the request, try after 1 min (2nd try)\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Tem_CCR7neg | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Tem_CCR7neg | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Tem_CCR7neg | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Tfh-like_CXCR5 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Tfh-like_CXCR5 | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n",
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n",
      "\n",
      "failed with the request, try after 1 min (1st try)\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n",
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n",
      "\n",
      "failed with the request, try after 1 min (2nd try)\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Tfh-like_CXCR5 | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Tfh-like_CXCR5 | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Tfh-like_CXCR5 | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Tfh-like_CXCR5 | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Th1-like_GZMK | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Th1-like_GZMK | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Th1-like_GZMK | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Th1-like_GZMK | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Th1-like_GZMK | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Th1-like_GZMK | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Th17-like_RORC | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |====================================================================  |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Th17-like_RORC | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Th17-like_RORC | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n",
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n",
      "\n",
      "failed with the request, try after 1 min (1st try)\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n",
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n",
      "\n",
      "failed with the request, try after 1 min (2nd try)\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Th17-like_RORC | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Th17-like_RORC | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Th17-like_RORC | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Th_LMNA | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Tn_CCR7 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 46s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Tn_CCR7 | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |====================================================================  |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Tn_CCR7 | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Tn_CCR7 | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Tn_CCR7 | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Tn_CCR7 | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Tr1_IL10 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Treg_FOXP3 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Treg_FOXP3 | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Treg_FOXP3 | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 34s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |====================================================================  |  97%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD4_Treg_FOXP3 | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 34s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD4_Treg_FOXP3 | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 41s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD8_CTL_GZMB | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD8_CTL_GZMB | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD8_CTL_GZMB | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD8_CTL_GZMB | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 41s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD8_CTL_GZMB | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD8_CTL_GZMB | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD8_CTL_GZMK | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD8_Tcm_IFI44L | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD8_Tcm_IFI44L | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD8_Tem_CCR7neg | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD8_Tem_CCR7neg | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD8_Tem_CCR7neg | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD8_Tem_CCR7neg | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |====================================================================  |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD8_Tem_CCR7neg | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD8_Tem_CCR7neg | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD8_Tn_CCR7 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD8_Tn_CCR7 | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD8_Tn_CCR7 | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 40s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD8_Tn_CCR7 | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 41s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: CD8_Tn_CCR7 | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: CD8_Tn_CCR7 | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: cMono_CD14 | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 40s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: cMono_CD14 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 39s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: cMono_CD14 | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 40s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: cMono_CD14 | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 36s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: cMono_CD14 | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 39s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: cMono_CD14 | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 39s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: cMono_IFI44L | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 41s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: cMono_IFI44L | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 38s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |====================================================================  |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: cMono_IFI44L | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 40s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: cMono_IFI44L | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: cMono_IFI44L | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 41s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: cMono_IFI44L | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: cMono_IL1B | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 41s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: DC2_CD1C | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: DC2_CD1C | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: DC2_CD1C | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: DC2_CD1C | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: DC2_CD1C | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 41s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: DC2_CD1C | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 40s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: DC_CSF2RA | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: DC_CSF2RA | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: DC_CSF2RA | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: DC_CSF2RA | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: DC_CSF2RA | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: DC_CSF2RA | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |====================================================================  |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: gdT2_GZMH | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: gdT2_GZMH | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: gdT2_GZMH | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: gdT2_GZMH | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: gdT2_GZMH | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: gdT2_GZMH | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: gdT2_IL12RB2 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: gdT2_IL12RB2 | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: gdT2_IL12RB2 | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: gdT2_IL12RB2 | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: gdT2_IL12RB2 | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: gdT2_IL12RB2 | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: intMono_GFRA2 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: intMono_GFRA2 | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: intMono_GFRA2 | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: intMono_GFRA2 | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: intMono_GFRA2 | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: intMono_GFRA2 | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: MAIT_SLC4A10 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: MAIT_SLC4A10 | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 37s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: MAIT_SLC4A10 | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |====================================================================  |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: MAIT_SLC4A10 | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: MAIT_SLC4A10 | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: MAIT_SLC4A10 | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Mature_NK_dim_FCGR3A | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Mature_NK_dim_FCGR3A | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Mature_NK_dim_FCGR3A | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Mature_NK_dim_FCGR3A | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 41s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Mature_NK_dim_FCGR3A | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Mature_NK_dim_FCGR3A | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: MK_GP9 | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: MK_GP9 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: MK_GP9 | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: MK_GP9 | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: MK_GP9 | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n",
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n",
      "\n",
      "failed with the request, try after 1 min (1st try)\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n",
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n",
      "\n",
      "failed with the request, try after 1 min (2nd try)\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: MK_GP9 | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: ncMono_C1QA | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: ncMono_C1QA | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: ncMono_C1QA | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 46s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n",
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Error in function (type, msg, asError = TRUE)  : Empty reply from server\n",
      "\n",
      "failed with the request, try after 1 min (1st try)\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: ncMono_C1QA | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: ncMono_C1QA | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: ncMono_C1QA | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: ncMono_FCGR3A | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 41s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: ncMono_FCGR3A | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: ncMono_FCGR3A | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: ncMono_FCGR3A | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: ncMono_FCGR3A | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: ncMono_FCGR3A | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: NKT_NCR1 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: NKT_NCR1 | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: NKT_NCR1 | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 41s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: NKT_NCR1 | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: NKT_NCR1 | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: NKT_NCR1 | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 41s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |====================================================================  |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: NK_bright_XCL1 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: NK_bright_XCL1 | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 41s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: NK_bright_XCL1 | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: pDC_IRF4 | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: pDC_IRF4 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: pDC_IRF4 | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: pDC_IRF4 | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: pDC_IRF4 | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: pDC_IRF4 | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 41s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Plasma_IGHA1 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Plasma_IGHA1 | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Plasma_IGHA1 | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Plasma_IGHA1 | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: pre-Switched_Bm_JAM3 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: pre-Switched_Bm_JAM3 | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |====================================================================  |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: pre-Switched_Bm_JAM3 | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: pre-Switched_Bm_JAM3 | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Switched_activated_Bm_CD86 | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Switched_activated_Bm_CD86 | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Switched_activated_Bm_CD86 | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 46s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Switched_activated_Bm_CD86 | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 46s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Switched_activated_Bm_CD86 | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 47s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Switched_activated_Bm_CD86 | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Switched_Bm_IGHDneg | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 46s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Switched_Bm_IGHDneg | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 46s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Switched_Bm_IGHDneg | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 46s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Switched_Bm_IGHDneg | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 47s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Switched_Bm_IGHDneg | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 47s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Switched_Bm_IGHDneg | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 47s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Switched_Bm_IGHE | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Switched_Bm_IGHE | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Switched_Bm_IGHE | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Switched_Bm_IGHE | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Switched_Bm_IGHE | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Terminal_NK_dim_CD160neg | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Terminal_NK_dim_CD160neg | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Terminal_NK_dim_CD160neg | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Terminal_NK_dim_CD160neg | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Transitional_B_MME | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Transitional_B_MME | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 42s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Transitional_B_MME | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Transitional_B_MME | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Transitional_B_MME | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Transitional_B_MME | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Transitional_NK_GZMK | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Transitional_NK_GZMK | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Transitional_NK_GZMK | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Transitional_NK_GZMK | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Transitional_NK_GZMK | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 43s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================|  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Unswitched_Bm_CD1C | Underweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Unswitched_Bm_CD1C | Underweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Unswitched_Bm_CD1C | Overweight_vs_Normal | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |======================================================================| 100%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Unswitched_Bm_CD1C | Overweight_vs_Normal | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 44s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  -> No significant terms.\n",
      "Processing: Unswitched_Bm_CD1C | Underweight_vs_Overweight | down \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  99%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: Unswitched_Bm_CD1C | Underweight_vs_Overweight | up \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Don't make too frequent requests. The time break is 60s.\n",
      "Please wait for 45s for the next request.\n",
      "The time break can be set by `request_interval` argument.\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  |===================================================================== |  98%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The default enrichment table does not contain informatin of associated\n",
      "genes for each input region. You can set `download_by = 'tsv'` to\n",
      "download the complete table, but note only the top 500 regions can be\n",
      "retreived. See the following link:\n",
      "\n",
      "https://great-help.atlassian.net/wiki/spaces/GREAT/pages/655401/Export#Export-GlobalExport\n",
      "\n",
      "Except the additional gene-region association column if taking 'tsv' as\n",
      "the source of result, all other columns are the same if you choose\n",
      "'json' (the default) as the source. Or you can try the local GREAT\n",
      "analysis with the function `great()`.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "result_all <- run_great_go_by_group(peaks)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "3c88befc-780b-4e5e-b08f-4e98b001d5ab",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A data.frame: 2633 × 20</caption>\n",
       "<thead>\n",
       "\t<tr><th scope=col>cell_type</th><th scope=col>contrast</th><th scope=col>direction</th><th scope=col>ID</th><th scope=col>name</th><th scope=col>Binom_Genome_Fraction</th><th scope=col>Binom_Expected</th><th scope=col>Binom_Observed_Region_Hits</th><th scope=col>Binom_Fold_Enrichment</th><th scope=col>Binom_Region_Set_Coverage</th><th scope=col>Binom_Raw_PValue</th><th scope=col>Binom_Adjp_BH</th><th scope=col>Hyper_Total_Genes</th><th scope=col>Hyper_Expected</th><th scope=col>Hyper_Observed_Gene_Hits</th><th scope=col>Hyper_Fold_Enrichment</th><th scope=col>Hyper_Gene_Set_Coverage</th><th scope=col>Hyper_Term_Gene_Coverage</th><th scope=col>Hyper_Raw_PValue</th><th scope=col>Hyper_Adjp_BH</th></tr>\n",
       "\t<tr><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><td>Atypical_Bm_ITGAX</td><td>Underweight_vs_Normal</td><td>down</td><td>GO:0019843</td><td>rRNA binding                                                                             </td><td>1.981897e-03</td><td>0.035674150</td><td> 4</td><td>112.126000</td><td>0.22222220</td><td>4.617453e-08</td><td>1.949489e-04</td><td>  60</td><td>0.047930980</td><td> 1</td><td> 20.8633300</td><td>0.066666670</td><td>0.016666670</td><td>0.0468906700</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Atypical_Bm_ITGAX</td><td>Underweight_vs_Normal</td><td>down</td><td>GO:0032452</td><td>histone demethylase activity                                                             </td><td>3.594768e-03</td><td>0.064705820</td><td> 4</td><td> 61.818240</td><td>0.22222220</td><td>4.908039e-07</td><td>1.036087e-03</td><td>  28</td><td>0.022367790</td><td> 2</td><td> 89.4142900</td><td>0.133333300</td><td>0.071428570</td><td>0.0002224690</td><td>0.8693951</td></tr>\n",
       "\t<tr><td>Atypical_Bm_ITGAX</td><td>Underweight_vs_Normal</td><td>down</td><td>GO:0032451</td><td>demethylase activity                                                                     </td><td>4.128775e-03</td><td>0.074317940</td><td> 4</td><td> 53.822800</td><td>0.22222220</td><td>8.490012e-07</td><td>1.194828e-03</td><td>  38</td><td>0.030356290</td><td> 2</td><td> 65.8842100</td><td>0.133333300</td><td>0.052631580</td><td>0.0004118404</td><td>0.8693951</td></tr>\n",
       "\t<tr><td>Atypical_Bm_ITGAX</td><td>Underweight_vs_Normal</td><td>down</td><td>GO:0003735</td><td>structural constituent of ribosome                                                       </td><td>8.019204e-03</td><td>0.144345700</td><td> 4</td><td> 27.711260</td><td>0.22222220</td><td>1.156601e-05</td><td>1.044238e-02</td><td> 162</td><td>0.129413600</td><td> 1</td><td>  7.7271600</td><td>0.066666670</td><td>0.006172840</td><td>0.1219255000</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Atypical_Bm_ITGAX</td><td>Underweight_vs_Normal</td><td>down</td><td>GO:0051213</td><td>dioxygenase activity                                                                     </td><td>8.157687e-03</td><td>0.146838400</td><td> 4</td><td> 27.240840</td><td>0.22222220</td><td>1.236663e-05</td><td>1.044238e-02</td><td>  89</td><td>0.071097620</td><td> 2</td><td> 28.1303400</td><td>0.133333300</td><td>0.022471910</td><td>0.0022407900</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>down</td><td>GO:0032452</td><td>histone demethylase activity                                                             </td><td>3.594768e-03</td><td>0.291176200</td><td>10</td><td> 34.343470</td><td>0.12345680</td><td>5.366114e-13</td><td>2.265573e-09</td><td>  28</td><td>0.146136200</td><td> 3</td><td> 20.5287900</td><td>0.030612240</td><td>0.107142900</td><td>0.0004107958</td><td>0.5781266</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>down</td><td>GO:0032451</td><td>demethylase activity                                                                     </td><td>4.128775e-03</td><td>0.334430800</td><td>10</td><td> 29.901560</td><td>0.12345680</td><td>2.070966e-12</td><td>4.371809e-09</td><td>  38</td><td>0.198327700</td><td> 3</td><td> 15.1264800</td><td>0.030612240</td><td>0.078947370</td><td>0.0010186160</td><td>0.7514860</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>down</td><td>GO:0051213</td><td>dioxygenase activity                                                                     </td><td>8.157687e-03</td><td>0.660772700</td><td>12</td><td> 18.160560</td><td>0.14814810</td><td>3.649426e-12</td><td>5.135959e-09</td><td>  89</td><td>0.464504400</td><td> 5</td><td> 10.7641600</td><td>0.051020410</td><td>0.056179780</td><td>0.0001025428</td><td>0.4329357</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>down</td><td>GO:0019843</td><td>rRNA binding                                                                             </td><td>1.981897e-03</td><td>0.160533700</td><td> 8</td><td> 49.833770</td><td>0.09876543</td><td>6.732352e-12</td><td>7.105998e-09</td><td>  60</td><td>0.313149100</td><td> 3</td><td>  9.5801020</td><td>0.030612240</td><td>0.050000000</td><td>0.0038035080</td><td>0.7514860</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>down</td><td>GO:0032453</td><td>histone demethylase activity (H3-K4 specific)                                            </td><td>1.087137e-03</td><td>0.088058070</td><td> 5</td><td> 56.780710</td><td>0.06172840</td><td>3.631956e-08</td><td>3.066824e-05</td><td>   8</td><td>0.041753210</td><td> 1</td><td> 23.9502600</td><td>0.010204080</td><td>0.125000000</td><td>0.0410059200</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>down</td><td>GO:0071558</td><td>histone demethylase activity (H3-K27 specific)                                           </td><td>7.155255e-04</td><td>0.057957570</td><td> 4</td><td> 69.016010</td><td>0.04938272</td><td>4.173072e-07</td><td>2.936452e-04</td><td>   5</td><td>0.026095760</td><td> 1</td><td> 38.3204100</td><td>0.010204080</td><td>0.200000000</td><td>0.0258275000</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>down</td><td>GO:0050681</td><td>androgen receptor binding                                                                </td><td>5.738002e-03</td><td>0.464778100</td><td> 7</td><td> 15.060950</td><td>0.08641975</td><td>4.912225e-07</td><td>2.962773e-04</td><td>  43</td><td>0.224423500</td><td> 3</td><td> 13.3675800</td><td>0.030612240</td><td>0.069767440</td><td>0.0014622900</td><td>0.7514860</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>down</td><td>GO:0003735</td><td>structural constituent of ribosome                                                       </td><td>8.019204e-03</td><td>0.649555500</td><td> 7</td><td> 10.776600</td><td>0.08641975</td><td>4.413757e-06</td><td>2.329360e-03</td><td> 162</td><td>0.845502500</td><td> 2</td><td>  2.3654570</td><td>0.020408160</td><td>0.012345680</td><td>0.2072462000</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>down</td><td>GO:0035258</td><td>steroid hormone receptor binding                                                         </td><td>1.030543e-02</td><td>0.834739700</td><td> 7</td><td>  8.385848</td><td>0.08641975</td><td>2.203871e-05</td><td>1.033860e-02</td><td>  88</td><td>0.459285300</td><td> 3</td><td>  6.5318880</td><td>0.030612240</td><td>0.034090910</td><td>0.0109834000</td><td>0.9866365</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>down</td><td>GO:0035257</td><td>nuclear hormone receptor binding                                                         </td><td>1.462713e-02</td><td>1.184797000</td><td> 8</td><td>  6.752209</td><td>0.09876543</td><td>2.608199e-05</td><td>1.101182e-02</td><td> 137</td><td>0.715023700</td><td> 4</td><td>  5.5942200</td><td>0.040816330</td><td>0.029197080</td><td>0.0057714330</td><td>0.7860319</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>down</td><td>GO:0051427</td><td>hormone receptor binding                                                                 </td><td>1.666248e-02</td><td>1.349661000</td><td> 8</td><td>  5.927416</td><td>0.09876543</td><td>6.480817e-05</td><td>2.487455e-02</td><td> 161</td><td>0.840283300</td><td> 4</td><td>  4.7602990</td><td>0.040816330</td><td>0.024844720</td><td>0.0100826900</td><td>0.9768151</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>down</td><td>GO:0042799</td><td>histone methyltransferase activity (H4-K20 specific)                                     </td><td>2.066923e-04</td><td>0.016742080</td><td> 2</td><td>119.459500</td><td>0.02469136</td><td>1.369206e-04</td><td>4.817323e-02</td><td>   5</td><td>0.026095760</td><td> 2</td><td> 76.6408200</td><td>0.020408160</td><td>0.400000000</td><td>0.0002668833</td><td>0.5633906</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>up  </td><td>GO:0008349</td><td>MAP kinase kinase kinase kinase activity                                                 </td><td>1.474917e-03</td><td>0.131267600</td><td>11</td><td> 83.798270</td><td>0.12359550</td><td>2.361332e-18</td><td>9.969544e-15</td><td>  11</td><td>0.064440540</td><td> 1</td><td> 15.5181800</td><td>0.009090909</td><td>0.090909090</td><td>0.0626019700</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>up  </td><td>GO:0004702</td><td>signal transducer, downstream of receptor, with serine/threonine kinase activity         </td><td>7.837579e-03</td><td>0.697544600</td><td>11</td><td> 15.769600</td><td>0.12359550</td><td>1.428125e-10</td><td>3.014772e-07</td><td>  70</td><td>0.410076200</td><td> 1</td><td>  2.4385710</td><td>0.009090909</td><td>0.014285710</td><td>0.3377034000</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>up  </td><td>GO:0005057</td><td>signal transducer activity, downstream of receptor                                       </td><td>1.441310e-02</td><td>1.282766000</td><td>13</td><td> 10.134350</td><td>0.14606740</td><td>5.865720e-10</td><td>8.255023e-07</td><td> 122</td><td>0.714704200</td><td> 2</td><td>  2.7983610</td><td>0.018181820</td><td>0.016393440</td><td>0.1603025000</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>up  </td><td>GO:0004984</td><td>olfactory receptor activity                                                              </td><td>1.019862e-02</td><td>0.907677400</td><td>11</td><td> 12.118840</td><td>0.12359550</td><td>2.183453e-09</td><td>2.304635e-06</td><td> 357</td><td>2.091388000</td><td> 1</td><td>  0.4781513</td><td>0.009090909</td><td>0.002801120</td><td>0.8796965000</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>up  </td><td>GO:0000287</td><td>magnesium ion binding                                                                    </td><td>1.826515e-02</td><td>1.625598000</td><td>11</td><td>  6.766739</td><td>0.12359550</td><td>7.433790e-07</td><td>6.277092e-04</td><td> 199</td><td>1.165788000</td><td> 1</td><td>  0.8577889</td><td>0.009090909</td><td>0.005025126</td><td>0.6913177000</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>up  </td><td>GO:0004000</td><td>adenosine deaminase activity                                                             </td><td>1.260969e-03</td><td>0.112226200</td><td> 4</td><td> 35.642290</td><td>0.04494382</td><td>5.666375e-06</td><td>3.987239e-03</td><td>  11</td><td>0.064440540</td><td> 1</td><td> 15.5181800</td><td>0.009090909</td><td>0.090909090</td><td>0.0626019700</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>up  </td><td>GO:0003727</td><td>single-stranded RNA binding                                                              </td><td>7.803229e-03</td><td>0.694487400</td><td> 7</td><td> 10.079380</td><td>0.07865169</td><td>6.938828e-06</td><td>4.185105e-03</td><td>  81</td><td>0.474516700</td><td> 3</td><td>  6.3222220</td><td>0.027272730</td><td>0.037037040</td><td>0.0119890800</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>up  </td><td>GO:0004930</td><td>G-protein coupled receptor activity                                                      </td><td>5.726050e-02</td><td>5.096184000</td><td>17</td><td>  3.335829</td><td>0.19101120</td><td>1.090262e-05</td><td>5.753858e-03</td><td> 798</td><td>4.674868000</td><td> 8</td><td>  1.7112780</td><td>0.072727270</td><td>0.010025060</td><td>0.0964867900</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>up  </td><td>GO:0030616</td><td>transforming growth factor beta receptor, common-partner cytoplasmic mediator activity   </td><td>7.737478e-05</td><td>0.006886355</td><td> 2</td><td>290.429400</td><td>0.02247191</td><td>2.333958e-05</td><td>1.094886e-02</td><td>   1</td><td>0.005858231</td><td> 1</td><td>170.7000000</td><td>0.009090909</td><td>1.000000000</td><td>0.0058582310</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>up  </td><td>GO:0004674</td><td>protein serine/threonine kinase activity                                                 </td><td>4.525708e-02</td><td>4.027880000</td><td>14</td><td>  3.475773</td><td>0.15730340</td><td>4.659862e-05</td><td>1.967394e-02</td><td> 449</td><td>2.630346000</td><td> 4</td><td>  1.5207130</td><td>0.036363640</td><td>0.008908686</td><td>0.2693638000</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>up  </td><td>GO:0004888</td><td>transmembrane signaling receptor activity                                                </td><td>1.020089e-01</td><td>9.078791000</td><td>22</td><td>  2.423230</td><td>0.24719100</td><td>6.814491e-05</td><td>2.615526e-02</td><td>1207</td><td>7.070885000</td><td>13</td><td>  1.8385250</td><td>0.118181800</td><td>0.010770510</td><td>0.0241704600</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>up  </td><td>GO:0019239</td><td>deaminase activity                                                                       </td><td>2.504054e-03</td><td>0.222860800</td><td> 4</td><td> 17.948420</td><td>0.04494382</td><td>8.100153e-05</td><td>2.849904e-02</td><td>  34</td><td>0.199179800</td><td> 1</td><td>  5.0205880</td><td>0.009090909</td><td>0.029411760</td><td>0.1812216000</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>Bn_TCL1A         </td><td>Underweight_vs_Normal</td><td>up  </td><td>GO:0016814</td><td>hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines</td><td>2.726487e-03</td><td>0.242657400</td><td> 4</td><td> 16.484150</td><td>0.04494382</td><td>1.121500e-04</td><td>3.642287e-02</td><td>  35</td><td>0.205038100</td><td> 1</td><td>  4.8771430</td><td>0.009090909</td><td>0.028571430</td><td>0.1860269000</td><td>1.0000000</td></tr>\n",
       "\t<tr><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Normal    </td><td>up  </td><td>GO:0048037</td><td>cofactor binding                                                              </td><td>0.0208731400</td><td> 17.72130000</td><td> 35</td><td>  1.975025</td><td>0.041224970</td><td>1.562711e-04</td><td>2.999790e-02</td><td>  265</td><td>1.867151e+01</td><td> 26</td><td>  1.392496</td><td>0.0196523100</td><td>0.09811321</td><td>5.449844e-02</td><td>0.937493115</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Normal    </td><td>up  </td><td>GO:0050662</td><td>coenzyme binding                                                              </td><td>0.0136182200</td><td> 11.56187000</td><td> 26</td><td>  2.248772</td><td>0.030624260</td><td>1.563131e-04</td><td>2.999790e-02</td><td>  184</td><td>1.296437e+01</td><td> 18</td><td>  1.388421</td><td>0.0136054400</td><td>0.09782609</td><td>9.849451e-02</td><td>1.000000000</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Normal    </td><td>up  </td><td>GO:0005070</td><td>SH3/SH2 adaptor activity                                                      </td><td>0.0064376140</td><td>  5.46553400</td><td> 16</td><td>  2.927436</td><td>0.018845700</td><td>1.755588e-04</td><td>3.222649e-02</td><td>   57</td><td>4.016137e+00</td><td> 12</td><td>  2.987946</td><td>0.0090702950</td><td>0.21052630</td><td>5.178047e-04</td><td>0.095050932</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Normal    </td><td>up  </td><td>GO:0003690</td><td>double-stranded DNA binding                                                   </td><td>0.0949064300</td><td> 80.57556000</td><td>113</td><td>  1.402410</td><td>0.133097800</td><td>1.831933e-04</td><td>3.222675e-02</td><td>  809</td><td>5.700096e+01</td><td> 85</td><td>  1.491203</td><td>0.0642479200</td><td>0.10506800</td><td>1.311516e-04</td><td>0.048230861</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Normal    </td><td>up  </td><td>GO:0003677</td><td>DNA binding                                                                   </td><td>0.2140410000</td><td>181.72080000</td><td>225</td><td>  1.238163</td><td>0.265017700</td><td>2.367535e-04</td><td>3.986706e-02</td><td> 2476</td><td>1.744553e+02</td><td>191</td><td>  1.094836</td><td>0.1443689000</td><td>0.07714055</td><td>8.925708e-02</td><td>1.000000000</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Normal    </td><td>up  </td><td>GO:0008889</td><td>glycerophosphodiester phosphodiesterase activity                              </td><td>0.0006400373</td><td>  0.54339170</td><td>  5</td><td>  9.201466</td><td>0.005889282</td><td>2.494599e-04</td><td>3.986706e-02</td><td>    6</td><td>4.227512e-01</td><td>  3</td><td>  7.096372</td><td>0.0022675740</td><td>0.50000000</td><td>5.937450e-03</td><td>0.358113056</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Normal    </td><td>up  </td><td>GO:0030159</td><td>receptor signaling complex scaffold activity                                  </td><td>0.0023874270</td><td>  2.02692600</td><td>  9</td><td>  4.440222</td><td>0.010600710</td><td>2.549528e-04</td><td>3.986706e-02</td><td>   23</td><td>1.620546e+00</td><td>  7</td><td>  4.319531</td><td>0.0052910050</td><td>0.30434780</td><td>7.624330e-04</td><td>0.114964004</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Normal    </td><td>up  </td><td>GO:0070403</td><td>NAD+ binding                                                                  </td><td>0.0010140410</td><td>  0.86092090</td><td>  6</td><td>  6.969281</td><td>0.007067138</td><td>2.685020e-04</td><td>4.048627e-02</td><td>   14</td><td>9.864196e-01</td><td>  3</td><td>  3.041302</td><td>0.0022675740</td><td>0.21428570</td><td>7.083380e-02</td><td>0.937493115</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Normal    </td><td>up  </td><td>GO:0001517</td><td>N-acetylglucosamine 6-O-sulfotransferase activity                             </td><td>0.0006633551</td><td>  0.56318840</td><td>  5</td><td>  8.878023</td><td>0.005889282</td><td>2.935458e-04</td><td>4.273622e-02</td><td>    7</td><td>4.932098e-01</td><td>  4</td><td>  8.110139</td><td>0.0030234320</td><td>0.57142860</td><td>7.224176e-04</td><td>0.112964708</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Normal    </td><td>up  </td><td>GO:0001784</td><td>phosphotyrosine binding                                                       </td><td>0.0041880240</td><td>  3.55563200</td><td> 12</td><td>  3.374927</td><td>0.014134280</td><td>3.194725e-04</td><td>4.363090e-02</td><td>   35</td><td>2.466049e+00</td><td> 10</td><td>  4.055070</td><td>0.0075585790</td><td>0.28571430</td><td>1.049218e-04</td><td>0.047261237</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Normal    </td><td>up  </td><td>GO:1990837</td><td>sequence-specific double-stranded DNA binding                                 </td><td>0.0893120800</td><td> 75.82596000</td><td>106</td><td>  1.397938</td><td>0.124852800</td><td>3.302948e-04</td><td>4.363090e-02</td><td>  731</td><td>5.150519e+01</td><td> 80</td><td>  1.553241</td><td>0.0604686300</td><td>0.10943910</td><td>5.369848e-05</td><td>0.032387855</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Normal    </td><td>up  </td><td>GO:0000976</td><td>transcription regulatory region sequence-specific DNA binding                 </td><td>0.0862857200</td><td> 73.25658000</td><td>103</td><td>  1.406017</td><td>0.121319200</td><td>3.306937e-04</td><td>4.363090e-02</td><td>  692</td><td>4.875731e+01</td><td> 77</td><td>  1.579250</td><td>0.0582010600</td><td>0.11127170</td><td>4.253793e-05</td><td>0.029932523</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Normal    </td><td>up  </td><td>GO:0000981</td><td>RNA polymerase II transcription factor activity, sequence-specific DNA binding</td><td>0.1006345000</td><td> 85.43870000</td><td>117</td><td>  1.369403</td><td>0.137809200</td><td>3.468743e-04</td><td>4.428693e-02</td><td>  766</td><td>5.397124e+01</td><td> 88</td><td>  1.630498</td><td>0.0665155000</td><td>0.11488250</td><td>3.455085e-06</td><td>0.004862456</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Normal    </td><td>up  </td><td>GO:0001096</td><td>TFIIF-class transcription factor binding                                      </td><td>0.0001570776</td><td>  0.13335890</td><td>  3</td><td> 22.495690</td><td>0.003533569</td><td>3.566451e-04</td><td>4.428693e-02</td><td>    1</td><td>7.045854e-02</td><td>  1</td><td> 14.192740</td><td>0.0007558579</td><td>1.00000000</td><td>7.045854e-02</td><td>0.937493115</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Normal    </td><td>up  </td><td>GO:0008446</td><td>GDP-mannose 4,6-dehydratase activity                                          </td><td>0.0003877198</td><td>  0.32917410</td><td>  4</td><td> 12.151620</td><td>0.004711425</td><td>3.742874e-04</td><td>4.514975e-02</td><td>    1</td><td>7.045854e-02</td><td>  1</td><td> 14.192740</td><td>0.0007558579</td><td>1.00000000</td><td>7.045854e-02</td><td>0.937493115</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Normal    </td><td>up  </td><td>GO:0070401</td><td>NADP+ binding                                                                 </td><td>0.0003945207</td><td>  0.33494810</td><td>  4</td><td> 11.942150</td><td>0.004711425</td><td>3.994276e-04</td><td>4.684398e-02</td><td>    2</td><td>1.409171e-01</td><td>  1</td><td>  7.096372</td><td>0.0007558579</td><td>0.50000000</td><td>1.359562e-01</td><td>1.000000000</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Overweight</td><td>down</td><td>GO:0019843</td><td>rRNA binding                                                                  </td><td>0.0019818970</td><td>  0.04360174</td><td>  5</td><td>114.674300</td><td>0.227272700</td><td>7.829313e-10</td><td>3.305536e-06</td><td>   60</td><td>5.751718e-02</td><td>  1</td><td> 17.386110</td><td>0.0555555600</td><td>0.01666667</td><td>5.600594e-02</td><td>1.000000000</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Overweight</td><td>down</td><td>GO:0032452</td><td>histone demethylase activity                                                  </td><td>0.0035947680</td><td>  0.07908489</td><td>  5</td><td> 63.223200</td><td>0.227272700</td><td>1.502230e-08</td><td>3.171208e-05</td><td>   28</td><td>2.684135e-02</td><td>  2</td><td> 74.511900</td><td>0.1111111000</td><td>0.07142857</td><td>3.232730e-04</td><td>1.000000000</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Overweight</td><td>down</td><td>GO:0032451</td><td>demethylase activity                                                          </td><td>0.0041287750</td><td>  0.09083304</td><td>  5</td><td> 55.046050</td><td>0.227272700</td><td>2.979867e-08</td><td>4.193666e-05</td><td>   38</td><td>3.642754e-02</td><td>  2</td><td> 54.903510</td><td>0.1111111000</td><td>0.05263158</td><td>5.978157e-04</td><td>1.000000000</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Overweight</td><td>down</td><td>GO:0003735</td><td>structural constituent of ribosome                                            </td><td>0.0080192040</td><td>  0.17642250</td><td>  5</td><td> 28.341060</td><td>0.227272700</td><td>7.793745e-07</td><td>7.155144e-04</td><td>  162</td><td>1.552964e-01</td><td>  1</td><td>  6.439300</td><td>0.0555555600</td><td>0.00617284</td><td>1.444760e-01</td><td>1.000000000</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Overweight</td><td>down</td><td>GO:0051213</td><td>dioxygenase activity                                                          </td><td>0.0081576870</td><td>  0.17946910</td><td>  5</td><td> 27.859950</td><td>0.227272700</td><td>8.473643e-07</td><td>7.155144e-04</td><td>   89</td><td>8.531714e-02</td><td>  2</td><td> 23.441950</td><td>0.1111111000</td><td>0.02247191</td><td>3.235131e-03</td><td>1.000000000</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Overweight</td><td>down</td><td>GO:0032453</td><td>histone demethylase activity (H3-K4 specific)                                 </td><td>0.0010871370</td><td>  0.02391701</td><td>  3</td><td>125.433800</td><td>0.136363600</td><td>1.948255e-06</td><td>1.370922e-03</td><td>    8</td><td>7.668957e-03</td><td>  1</td><td>130.395800</td><td>0.0555555600</td><td>0.12500000</td><td>7.644696e-03</td><td>1.000000000</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Overweight</td><td>up  </td><td>GO:0051378</td><td>serotonin binding                                                             </td><td>0.0029198980</td><td>  0.50222240</td><td>  8</td><td> 15.929200</td><td>0.046511630</td><td>5.562894e-08</td><td>2.252977e-04</td><td>   11</td><td>1.675454e-01</td><td>  1</td><td>  5.968531</td><td>0.0034965030</td><td>0.09090909</td><td>1.553896e-01</td><td>1.000000000</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Overweight</td><td>up  </td><td>GO:0004993</td><td>G-protein coupled serotonin receptor activity                                 </td><td>0.0033145340</td><td>  0.57009980</td><td>  8</td><td> 14.032630</td><td>0.046511630</td><td>1.448213e-07</td><td>2.252977e-04</td><td>   22</td><td>3.350908e-01</td><td>  1</td><td>  2.984266</td><td>0.0034965030</td><td>0.04545455</td><td>2.867044e-01</td><td>1.000000000</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Overweight</td><td>up  </td><td>GO:0099589</td><td>serotonin receptor activity                                                   </td><td>0.0034451000</td><td>  0.59255720</td><td>  8</td><td> 13.500810</td><td>0.046511630</td><td>1.935662e-07</td><td>2.252977e-04</td><td>   29</td><td>4.417106e-01</td><td>  1</td><td>  2.263926</td><td>0.0034965030</td><td>0.03448276</td><td>3.594601e-01</td><td>1.000000000</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Overweight</td><td>up  </td><td>GO:0005201</td><td>extracellular matrix structural constituent                                   </td><td>0.0081131180</td><td>  1.39545600</td><td> 11</td><td>  7.882726</td><td>0.063953490</td><td>2.134512e-07</td><td>2.252977e-04</td><td>   78</td><td>1.188049e+00</td><td>  4</td><td>  3.366864</td><td>0.0139860100</td><td>0.05128205</td><td>3.124303e-02</td><td>1.000000000</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Overweight</td><td>up  </td><td>GO:0005515</td><td>protein binding                                                               </td><td>0.7330151000</td><td>126.07860000</td><td>151</td><td>  1.197666</td><td>0.877907000</td><td>2.969338e-06</td><td>2.507309e-03</td><td>11170</td><td>1.701347e+02</td><td>197</td><td>  1.157906</td><td>0.6888112000</td><td>0.01763653</td><td>5.874243e-04</td><td>0.620026349</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Overweight</td><td>up  </td><td>GO:0070405</td><td>ammonium ion binding                                                          </td><td>0.0080907640</td><td>  1.39161100</td><td>  9</td><td>  6.467323</td><td>0.052325580</td><td>1.334797e-05</td><td>9.392522e-03</td><td>   71</td><td>1.081429e+00</td><td>  2</td><td>  1.849404</td><td>0.0069930070</td><td>0.02816901</td><td>2.944349e-01</td><td>1.000000000</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Overweight</td><td>up  </td><td>GO:0008227</td><td>G-protein coupled amine receptor activity                                     </td><td>0.0070103290</td><td>  1.20577700</td><td>  8</td><td>  6.634728</td><td>0.046511630</td><td>3.393613e-05</td><td>2.046833e-02</td><td>   48</td><td>7.311072e-01</td><td>  1</td><td>  1.367788</td><td>0.0034965030</td><td>0.02083333</td><td>5.217698e-01</td><td>1.000000000</td></tr>\n",
       "\t<tr><td>Unswitched_Bm_CD1C</td><td>Underweight_vs_Overweight</td><td>up  </td><td>GO:0030159</td><td>receptor signaling complex scaffold activity                                  </td><td>0.0023874270</td><td>  0.41063750</td><td>  5</td><td> 12.176190</td><td>0.029069770</td><td>6.590213e-05</td><td>3.477985e-02</td><td>   23</td><td>3.503222e-01</td><td>  4</td><td> 11.418060</td><td>0.0139860100</td><td>0.17391300</td><td>3.713652e-04</td><td>0.522634625</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 2633 × 20\n",
       "\\begin{tabular}{llllllllllllllllllll}\n",
       " cell\\_type & contrast & direction & ID & name & Binom\\_Genome\\_Fraction & Binom\\_Expected & Binom\\_Observed\\_Region\\_Hits & Binom\\_Fold\\_Enrichment & Binom\\_Region\\_Set\\_Coverage & Binom\\_Raw\\_PValue & Binom\\_Adjp\\_BH & Hyper\\_Total\\_Genes & Hyper\\_Expected & Hyper\\_Observed\\_Gene\\_Hits & Hyper\\_Fold\\_Enrichment & Hyper\\_Gene\\_Set\\_Coverage & Hyper\\_Term\\_Gene\\_Coverage & Hyper\\_Raw\\_PValue & Hyper\\_Adjp\\_BH\\\\\n",
       " <chr> & <chr> & <chr> & <chr> & <chr> & <dbl> & <dbl> & <dbl> & <dbl> & <dbl> & <dbl> & <dbl> & <dbl> & <dbl> & <dbl> & <dbl> & <dbl> & <dbl> & <dbl> & <dbl>\\\\\n",
       "\\hline\n",
       "\t Atypical\\_Bm\\_ITGAX & Underweight\\_vs\\_Normal & down & GO:0019843 & rRNA binding                                                                              & 1.981897e-03 & 0.035674150 &  4 & 112.126000 & 0.22222220 & 4.617453e-08 & 1.949489e-04 &   60 & 0.047930980 &  1 &  20.8633300 & 0.066666670 & 0.016666670 & 0.0468906700 & 1.0000000\\\\\n",
       "\t Atypical\\_Bm\\_ITGAX & Underweight\\_vs\\_Normal & down & GO:0032452 & histone demethylase activity                                                              & 3.594768e-03 & 0.064705820 &  4 &  61.818240 & 0.22222220 & 4.908039e-07 & 1.036087e-03 &   28 & 0.022367790 &  2 &  89.4142900 & 0.133333300 & 0.071428570 & 0.0002224690 & 0.8693951\\\\\n",
       "\t Atypical\\_Bm\\_ITGAX & Underweight\\_vs\\_Normal & down & GO:0032451 & demethylase activity                                                                      & 4.128775e-03 & 0.074317940 &  4 &  53.822800 & 0.22222220 & 8.490012e-07 & 1.194828e-03 &   38 & 0.030356290 &  2 &  65.8842100 & 0.133333300 & 0.052631580 & 0.0004118404 & 0.8693951\\\\\n",
       "\t Atypical\\_Bm\\_ITGAX & Underweight\\_vs\\_Normal & down & GO:0003735 & structural constituent of ribosome                                                        & 8.019204e-03 & 0.144345700 &  4 &  27.711260 & 0.22222220 & 1.156601e-05 & 1.044238e-02 &  162 & 0.129413600 &  1 &   7.7271600 & 0.066666670 & 0.006172840 & 0.1219255000 & 1.0000000\\\\\n",
       "\t Atypical\\_Bm\\_ITGAX & Underweight\\_vs\\_Normal & down & GO:0051213 & dioxygenase activity                                                                      & 8.157687e-03 & 0.146838400 &  4 &  27.240840 & 0.22222220 & 1.236663e-05 & 1.044238e-02 &   89 & 0.071097620 &  2 &  28.1303400 & 0.133333300 & 0.022471910 & 0.0022407900 & 1.0000000\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & down & GO:0032452 & histone demethylase activity                                                              & 3.594768e-03 & 0.291176200 & 10 &  34.343470 & 0.12345680 & 5.366114e-13 & 2.265573e-09 &   28 & 0.146136200 &  3 &  20.5287900 & 0.030612240 & 0.107142900 & 0.0004107958 & 0.5781266\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & down & GO:0032451 & demethylase activity                                                                      & 4.128775e-03 & 0.334430800 & 10 &  29.901560 & 0.12345680 & 2.070966e-12 & 4.371809e-09 &   38 & 0.198327700 &  3 &  15.1264800 & 0.030612240 & 0.078947370 & 0.0010186160 & 0.7514860\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & down & GO:0051213 & dioxygenase activity                                                                      & 8.157687e-03 & 0.660772700 & 12 &  18.160560 & 0.14814810 & 3.649426e-12 & 5.135959e-09 &   89 & 0.464504400 &  5 &  10.7641600 & 0.051020410 & 0.056179780 & 0.0001025428 & 0.4329357\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & down & GO:0019843 & rRNA binding                                                                              & 1.981897e-03 & 0.160533700 &  8 &  49.833770 & 0.09876543 & 6.732352e-12 & 7.105998e-09 &   60 & 0.313149100 &  3 &   9.5801020 & 0.030612240 & 0.050000000 & 0.0038035080 & 0.7514860\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & down & GO:0032453 & histone demethylase activity (H3-K4 specific)                                             & 1.087137e-03 & 0.088058070 &  5 &  56.780710 & 0.06172840 & 3.631956e-08 & 3.066824e-05 &    8 & 0.041753210 &  1 &  23.9502600 & 0.010204080 & 0.125000000 & 0.0410059200 & 1.0000000\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & down & GO:0071558 & histone demethylase activity (H3-K27 specific)                                            & 7.155255e-04 & 0.057957570 &  4 &  69.016010 & 0.04938272 & 4.173072e-07 & 2.936452e-04 &    5 & 0.026095760 &  1 &  38.3204100 & 0.010204080 & 0.200000000 & 0.0258275000 & 1.0000000\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & down & GO:0050681 & androgen receptor binding                                                                 & 5.738002e-03 & 0.464778100 &  7 &  15.060950 & 0.08641975 & 4.912225e-07 & 2.962773e-04 &   43 & 0.224423500 &  3 &  13.3675800 & 0.030612240 & 0.069767440 & 0.0014622900 & 0.7514860\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & down & GO:0003735 & structural constituent of ribosome                                                        & 8.019204e-03 & 0.649555500 &  7 &  10.776600 & 0.08641975 & 4.413757e-06 & 2.329360e-03 &  162 & 0.845502500 &  2 &   2.3654570 & 0.020408160 & 0.012345680 & 0.2072462000 & 1.0000000\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & down & GO:0035258 & steroid hormone receptor binding                                                          & 1.030543e-02 & 0.834739700 &  7 &   8.385848 & 0.08641975 & 2.203871e-05 & 1.033860e-02 &   88 & 0.459285300 &  3 &   6.5318880 & 0.030612240 & 0.034090910 & 0.0109834000 & 0.9866365\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & down & GO:0035257 & nuclear hormone receptor binding                                                          & 1.462713e-02 & 1.184797000 &  8 &   6.752209 & 0.09876543 & 2.608199e-05 & 1.101182e-02 &  137 & 0.715023700 &  4 &   5.5942200 & 0.040816330 & 0.029197080 & 0.0057714330 & 0.7860319\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & down & GO:0051427 & hormone receptor binding                                                                  & 1.666248e-02 & 1.349661000 &  8 &   5.927416 & 0.09876543 & 6.480817e-05 & 2.487455e-02 &  161 & 0.840283300 &  4 &   4.7602990 & 0.040816330 & 0.024844720 & 0.0100826900 & 0.9768151\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & down & GO:0042799 & histone methyltransferase activity (H4-K20 specific)                                      & 2.066923e-04 & 0.016742080 &  2 & 119.459500 & 0.02469136 & 1.369206e-04 & 4.817323e-02 &    5 & 0.026095760 &  2 &  76.6408200 & 0.020408160 & 0.400000000 & 0.0002668833 & 0.5633906\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & up   & GO:0008349 & MAP kinase kinase kinase kinase activity                                                  & 1.474917e-03 & 0.131267600 & 11 &  83.798270 & 0.12359550 & 2.361332e-18 & 9.969544e-15 &   11 & 0.064440540 &  1 &  15.5181800 & 0.009090909 & 0.090909090 & 0.0626019700 & 1.0000000\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & up   & GO:0004702 & signal transducer, downstream of receptor, with serine/threonine kinase activity          & 7.837579e-03 & 0.697544600 & 11 &  15.769600 & 0.12359550 & 1.428125e-10 & 3.014772e-07 &   70 & 0.410076200 &  1 &   2.4385710 & 0.009090909 & 0.014285710 & 0.3377034000 & 1.0000000\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & up   & GO:0005057 & signal transducer activity, downstream of receptor                                        & 1.441310e-02 & 1.282766000 & 13 &  10.134350 & 0.14606740 & 5.865720e-10 & 8.255023e-07 &  122 & 0.714704200 &  2 &   2.7983610 & 0.018181820 & 0.016393440 & 0.1603025000 & 1.0000000\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & up   & GO:0004984 & olfactory receptor activity                                                               & 1.019862e-02 & 0.907677400 & 11 &  12.118840 & 0.12359550 & 2.183453e-09 & 2.304635e-06 &  357 & 2.091388000 &  1 &   0.4781513 & 0.009090909 & 0.002801120 & 0.8796965000 & 1.0000000\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & up   & GO:0000287 & magnesium ion binding                                                                     & 1.826515e-02 & 1.625598000 & 11 &   6.766739 & 0.12359550 & 7.433790e-07 & 6.277092e-04 &  199 & 1.165788000 &  1 &   0.8577889 & 0.009090909 & 0.005025126 & 0.6913177000 & 1.0000000\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & up   & GO:0004000 & adenosine deaminase activity                                                              & 1.260969e-03 & 0.112226200 &  4 &  35.642290 & 0.04494382 & 5.666375e-06 & 3.987239e-03 &   11 & 0.064440540 &  1 &  15.5181800 & 0.009090909 & 0.090909090 & 0.0626019700 & 1.0000000\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & up   & GO:0003727 & single-stranded RNA binding                                                               & 7.803229e-03 & 0.694487400 &  7 &  10.079380 & 0.07865169 & 6.938828e-06 & 4.185105e-03 &   81 & 0.474516700 &  3 &   6.3222220 & 0.027272730 & 0.037037040 & 0.0119890800 & 1.0000000\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & up   & GO:0004930 & G-protein coupled receptor activity                                                       & 5.726050e-02 & 5.096184000 & 17 &   3.335829 & 0.19101120 & 1.090262e-05 & 5.753858e-03 &  798 & 4.674868000 &  8 &   1.7112780 & 0.072727270 & 0.010025060 & 0.0964867900 & 1.0000000\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & up   & GO:0030616 & transforming growth factor beta receptor, common-partner cytoplasmic mediator activity    & 7.737478e-05 & 0.006886355 &  2 & 290.429400 & 0.02247191 & 2.333958e-05 & 1.094886e-02 &    1 & 0.005858231 &  1 & 170.7000000 & 0.009090909 & 1.000000000 & 0.0058582310 & 1.0000000\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & up   & GO:0004674 & protein serine/threonine kinase activity                                                  & 4.525708e-02 & 4.027880000 & 14 &   3.475773 & 0.15730340 & 4.659862e-05 & 1.967394e-02 &  449 & 2.630346000 &  4 &   1.5207130 & 0.036363640 & 0.008908686 & 0.2693638000 & 1.0000000\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & up   & GO:0004888 & transmembrane signaling receptor activity                                                 & 1.020089e-01 & 9.078791000 & 22 &   2.423230 & 0.24719100 & 6.814491e-05 & 2.615526e-02 & 1207 & 7.070885000 & 13 &   1.8385250 & 0.118181800 & 0.010770510 & 0.0241704600 & 1.0000000\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & up   & GO:0019239 & deaminase activity                                                                        & 2.504054e-03 & 0.222860800 &  4 &  17.948420 & 0.04494382 & 8.100153e-05 & 2.849904e-02 &   34 & 0.199179800 &  1 &   5.0205880 & 0.009090909 & 0.029411760 & 0.1812216000 & 1.0000000\\\\\n",
       "\t Bn\\_TCL1A          & Underweight\\_vs\\_Normal & up   & GO:0016814 & hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines & 2.726487e-03 & 0.242657400 &  4 &  16.484150 & 0.04494382 & 1.121500e-04 & 3.642287e-02 &   35 & 0.205038100 &  1 &   4.8771430 & 0.009090909 & 0.028571430 & 0.1860269000 & 1.0000000\\\\\n",
       "\t ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Normal     & up   & GO:0048037 & cofactor binding                                                               & 0.0208731400 &  17.72130000 &  35 &   1.975025 & 0.041224970 & 1.562711e-04 & 2.999790e-02 &   265 & 1.867151e+01 &  26 &   1.392496 & 0.0196523100 & 0.09811321 & 5.449844e-02 & 0.937493115\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Normal     & up   & GO:0050662 & coenzyme binding                                                               & 0.0136182200 &  11.56187000 &  26 &   2.248772 & 0.030624260 & 1.563131e-04 & 2.999790e-02 &   184 & 1.296437e+01 &  18 &   1.388421 & 0.0136054400 & 0.09782609 & 9.849451e-02 & 1.000000000\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Normal     & up   & GO:0005070 & SH3/SH2 adaptor activity                                                       & 0.0064376140 &   5.46553400 &  16 &   2.927436 & 0.018845700 & 1.755588e-04 & 3.222649e-02 &    57 & 4.016137e+00 &  12 &   2.987946 & 0.0090702950 & 0.21052630 & 5.178047e-04 & 0.095050932\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Normal     & up   & GO:0003690 & double-stranded DNA binding                                                    & 0.0949064300 &  80.57556000 & 113 &   1.402410 & 0.133097800 & 1.831933e-04 & 3.222675e-02 &   809 & 5.700096e+01 &  85 &   1.491203 & 0.0642479200 & 0.10506800 & 1.311516e-04 & 0.048230861\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Normal     & up   & GO:0003677 & DNA binding                                                                    & 0.2140410000 & 181.72080000 & 225 &   1.238163 & 0.265017700 & 2.367535e-04 & 3.986706e-02 &  2476 & 1.744553e+02 & 191 &   1.094836 & 0.1443689000 & 0.07714055 & 8.925708e-02 & 1.000000000\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Normal     & up   & GO:0008889 & glycerophosphodiester phosphodiesterase activity                               & 0.0006400373 &   0.54339170 &   5 &   9.201466 & 0.005889282 & 2.494599e-04 & 3.986706e-02 &     6 & 4.227512e-01 &   3 &   7.096372 & 0.0022675740 & 0.50000000 & 5.937450e-03 & 0.358113056\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Normal     & up   & GO:0030159 & receptor signaling complex scaffold activity                                   & 0.0023874270 &   2.02692600 &   9 &   4.440222 & 0.010600710 & 2.549528e-04 & 3.986706e-02 &    23 & 1.620546e+00 &   7 &   4.319531 & 0.0052910050 & 0.30434780 & 7.624330e-04 & 0.114964004\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Normal     & up   & GO:0070403 & NAD+ binding                                                                   & 0.0010140410 &   0.86092090 &   6 &   6.969281 & 0.007067138 & 2.685020e-04 & 4.048627e-02 &    14 & 9.864196e-01 &   3 &   3.041302 & 0.0022675740 & 0.21428570 & 7.083380e-02 & 0.937493115\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Normal     & up   & GO:0001517 & N-acetylglucosamine 6-O-sulfotransferase activity                              & 0.0006633551 &   0.56318840 &   5 &   8.878023 & 0.005889282 & 2.935458e-04 & 4.273622e-02 &     7 & 4.932098e-01 &   4 &   8.110139 & 0.0030234320 & 0.57142860 & 7.224176e-04 & 0.112964708\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Normal     & up   & GO:0001784 & phosphotyrosine binding                                                        & 0.0041880240 &   3.55563200 &  12 &   3.374927 & 0.014134280 & 3.194725e-04 & 4.363090e-02 &    35 & 2.466049e+00 &  10 &   4.055070 & 0.0075585790 & 0.28571430 & 1.049218e-04 & 0.047261237\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Normal     & up   & GO:1990837 & sequence-specific double-stranded DNA binding                                  & 0.0893120800 &  75.82596000 & 106 &   1.397938 & 0.124852800 & 3.302948e-04 & 4.363090e-02 &   731 & 5.150519e+01 &  80 &   1.553241 & 0.0604686300 & 0.10943910 & 5.369848e-05 & 0.032387855\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Normal     & up   & GO:0000976 & transcription regulatory region sequence-specific DNA binding                  & 0.0862857200 &  73.25658000 & 103 &   1.406017 & 0.121319200 & 3.306937e-04 & 4.363090e-02 &   692 & 4.875731e+01 &  77 &   1.579250 & 0.0582010600 & 0.11127170 & 4.253793e-05 & 0.029932523\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Normal     & up   & GO:0000981 & RNA polymerase II transcription factor activity, sequence-specific DNA binding & 0.1006345000 &  85.43870000 & 117 &   1.369403 & 0.137809200 & 3.468743e-04 & 4.428693e-02 &   766 & 5.397124e+01 &  88 &   1.630498 & 0.0665155000 & 0.11488250 & 3.455085e-06 & 0.004862456\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Normal     & up   & GO:0001096 & TFIIF-class transcription factor binding                                       & 0.0001570776 &   0.13335890 &   3 &  22.495690 & 0.003533569 & 3.566451e-04 & 4.428693e-02 &     1 & 7.045854e-02 &   1 &  14.192740 & 0.0007558579 & 1.00000000 & 7.045854e-02 & 0.937493115\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Normal     & up   & GO:0008446 & GDP-mannose 4,6-dehydratase activity                                           & 0.0003877198 &   0.32917410 &   4 &  12.151620 & 0.004711425 & 3.742874e-04 & 4.514975e-02 &     1 & 7.045854e-02 &   1 &  14.192740 & 0.0007558579 & 1.00000000 & 7.045854e-02 & 0.937493115\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Normal     & up   & GO:0070401 & NADP+ binding                                                                  & 0.0003945207 &   0.33494810 &   4 &  11.942150 & 0.004711425 & 3.994276e-04 & 4.684398e-02 &     2 & 1.409171e-01 &   1 &   7.096372 & 0.0007558579 & 0.50000000 & 1.359562e-01 & 1.000000000\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Overweight & down & GO:0019843 & rRNA binding                                                                   & 0.0019818970 &   0.04360174 &   5 & 114.674300 & 0.227272700 & 7.829313e-10 & 3.305536e-06 &    60 & 5.751718e-02 &   1 &  17.386110 & 0.0555555600 & 0.01666667 & 5.600594e-02 & 1.000000000\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Overweight & down & GO:0032452 & histone demethylase activity                                                   & 0.0035947680 &   0.07908489 &   5 &  63.223200 & 0.227272700 & 1.502230e-08 & 3.171208e-05 &    28 & 2.684135e-02 &   2 &  74.511900 & 0.1111111000 & 0.07142857 & 3.232730e-04 & 1.000000000\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Overweight & down & GO:0032451 & demethylase activity                                                           & 0.0041287750 &   0.09083304 &   5 &  55.046050 & 0.227272700 & 2.979867e-08 & 4.193666e-05 &    38 & 3.642754e-02 &   2 &  54.903510 & 0.1111111000 & 0.05263158 & 5.978157e-04 & 1.000000000\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Overweight & down & GO:0003735 & structural constituent of ribosome                                             & 0.0080192040 &   0.17642250 &   5 &  28.341060 & 0.227272700 & 7.793745e-07 & 7.155144e-04 &   162 & 1.552964e-01 &   1 &   6.439300 & 0.0555555600 & 0.00617284 & 1.444760e-01 & 1.000000000\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Overweight & down & GO:0051213 & dioxygenase activity                                                           & 0.0081576870 &   0.17946910 &   5 &  27.859950 & 0.227272700 & 8.473643e-07 & 7.155144e-04 &    89 & 8.531714e-02 &   2 &  23.441950 & 0.1111111000 & 0.02247191 & 3.235131e-03 & 1.000000000\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Overweight & down & GO:0032453 & histone demethylase activity (H3-K4 specific)                                  & 0.0010871370 &   0.02391701 &   3 & 125.433800 & 0.136363600 & 1.948255e-06 & 1.370922e-03 &     8 & 7.668957e-03 &   1 & 130.395800 & 0.0555555600 & 0.12500000 & 7.644696e-03 & 1.000000000\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Overweight & up   & GO:0051378 & serotonin binding                                                              & 0.0029198980 &   0.50222240 &   8 &  15.929200 & 0.046511630 & 5.562894e-08 & 2.252977e-04 &    11 & 1.675454e-01 &   1 &   5.968531 & 0.0034965030 & 0.09090909 & 1.553896e-01 & 1.000000000\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Overweight & up   & GO:0004993 & G-protein coupled serotonin receptor activity                                  & 0.0033145340 &   0.57009980 &   8 &  14.032630 & 0.046511630 & 1.448213e-07 & 2.252977e-04 &    22 & 3.350908e-01 &   1 &   2.984266 & 0.0034965030 & 0.04545455 & 2.867044e-01 & 1.000000000\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Overweight & up   & GO:0099589 & serotonin receptor activity                                                    & 0.0034451000 &   0.59255720 &   8 &  13.500810 & 0.046511630 & 1.935662e-07 & 2.252977e-04 &    29 & 4.417106e-01 &   1 &   2.263926 & 0.0034965030 & 0.03448276 & 3.594601e-01 & 1.000000000\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Overweight & up   & GO:0005201 & extracellular matrix structural constituent                                    & 0.0081131180 &   1.39545600 &  11 &   7.882726 & 0.063953490 & 2.134512e-07 & 2.252977e-04 &    78 & 1.188049e+00 &   4 &   3.366864 & 0.0139860100 & 0.05128205 & 3.124303e-02 & 1.000000000\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Overweight & up   & GO:0005515 & protein binding                                                                & 0.7330151000 & 126.07860000 & 151 &   1.197666 & 0.877907000 & 2.969338e-06 & 2.507309e-03 & 11170 & 1.701347e+02 & 197 &   1.157906 & 0.6888112000 & 0.01763653 & 5.874243e-04 & 0.620026349\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Overweight & up   & GO:0070405 & ammonium ion binding                                                           & 0.0080907640 &   1.39161100 &   9 &   6.467323 & 0.052325580 & 1.334797e-05 & 9.392522e-03 &    71 & 1.081429e+00 &   2 &   1.849404 & 0.0069930070 & 0.02816901 & 2.944349e-01 & 1.000000000\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Overweight & up   & GO:0008227 & G-protein coupled amine receptor activity                                      & 0.0070103290 &   1.20577700 &   8 &   6.634728 & 0.046511630 & 3.393613e-05 & 2.046833e-02 &    48 & 7.311072e-01 &   1 &   1.367788 & 0.0034965030 & 0.02083333 & 5.217698e-01 & 1.000000000\\\\\n",
       "\t Unswitched\\_Bm\\_CD1C & Underweight\\_vs\\_Overweight & up   & GO:0030159 & receptor signaling complex scaffold activity                                   & 0.0023874270 &   0.41063750 &   5 &  12.176190 & 0.029069770 & 6.590213e-05 & 3.477985e-02 &    23 & 3.503222e-01 &   4 &  11.418060 & 0.0139860100 & 0.17391300 & 3.713652e-04 & 0.522634625\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 2633 × 20\n",
       "\n",
       "| cell_type &lt;chr&gt; | contrast &lt;chr&gt; | direction &lt;chr&gt; | ID &lt;chr&gt; | name &lt;chr&gt; | Binom_Genome_Fraction &lt;dbl&gt; | Binom_Expected &lt;dbl&gt; | Binom_Observed_Region_Hits &lt;dbl&gt; | Binom_Fold_Enrichment &lt;dbl&gt; | Binom_Region_Set_Coverage &lt;dbl&gt; | Binom_Raw_PValue &lt;dbl&gt; | Binom_Adjp_BH &lt;dbl&gt; | Hyper_Total_Genes &lt;dbl&gt; | Hyper_Expected &lt;dbl&gt; | Hyper_Observed_Gene_Hits &lt;dbl&gt; | Hyper_Fold_Enrichment &lt;dbl&gt; | Hyper_Gene_Set_Coverage &lt;dbl&gt; | Hyper_Term_Gene_Coverage &lt;dbl&gt; | Hyper_Raw_PValue &lt;dbl&gt; | Hyper_Adjp_BH &lt;dbl&gt; |\n",
       "|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\n",
       "| Atypical_Bm_ITGAX | Underweight_vs_Normal | down | GO:0019843 | rRNA binding                                                                              | 1.981897e-03 | 0.035674150 |  4 | 112.126000 | 0.22222220 | 4.617453e-08 | 1.949489e-04 |   60 | 0.047930980 |  1 |  20.8633300 | 0.066666670 | 0.016666670 | 0.0468906700 | 1.0000000 |\n",
       "| Atypical_Bm_ITGAX | Underweight_vs_Normal | down | GO:0032452 | histone demethylase activity                                                              | 3.594768e-03 | 0.064705820 |  4 |  61.818240 | 0.22222220 | 4.908039e-07 | 1.036087e-03 |   28 | 0.022367790 |  2 |  89.4142900 | 0.133333300 | 0.071428570 | 0.0002224690 | 0.8693951 |\n",
       "| Atypical_Bm_ITGAX | Underweight_vs_Normal | down | GO:0032451 | demethylase activity                                                                      | 4.128775e-03 | 0.074317940 |  4 |  53.822800 | 0.22222220 | 8.490012e-07 | 1.194828e-03 |   38 | 0.030356290 |  2 |  65.8842100 | 0.133333300 | 0.052631580 | 0.0004118404 | 0.8693951 |\n",
       "| Atypical_Bm_ITGAX | Underweight_vs_Normal | down | GO:0003735 | structural constituent of ribosome                                                        | 8.019204e-03 | 0.144345700 |  4 |  27.711260 | 0.22222220 | 1.156601e-05 | 1.044238e-02 |  162 | 0.129413600 |  1 |   7.7271600 | 0.066666670 | 0.006172840 | 0.1219255000 | 1.0000000 |\n",
       "| Atypical_Bm_ITGAX | Underweight_vs_Normal | down | GO:0051213 | dioxygenase activity                                                                      | 8.157687e-03 | 0.146838400 |  4 |  27.240840 | 0.22222220 | 1.236663e-05 | 1.044238e-02 |   89 | 0.071097620 |  2 |  28.1303400 | 0.133333300 | 0.022471910 | 0.0022407900 | 1.0000000 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | down | GO:0032452 | histone demethylase activity                                                              | 3.594768e-03 | 0.291176200 | 10 |  34.343470 | 0.12345680 | 5.366114e-13 | 2.265573e-09 |   28 | 0.146136200 |  3 |  20.5287900 | 0.030612240 | 0.107142900 | 0.0004107958 | 0.5781266 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | down | GO:0032451 | demethylase activity                                                                      | 4.128775e-03 | 0.334430800 | 10 |  29.901560 | 0.12345680 | 2.070966e-12 | 4.371809e-09 |   38 | 0.198327700 |  3 |  15.1264800 | 0.030612240 | 0.078947370 | 0.0010186160 | 0.7514860 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | down | GO:0051213 | dioxygenase activity                                                                      | 8.157687e-03 | 0.660772700 | 12 |  18.160560 | 0.14814810 | 3.649426e-12 | 5.135959e-09 |   89 | 0.464504400 |  5 |  10.7641600 | 0.051020410 | 0.056179780 | 0.0001025428 | 0.4329357 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | down | GO:0019843 | rRNA binding                                                                              | 1.981897e-03 | 0.160533700 |  8 |  49.833770 | 0.09876543 | 6.732352e-12 | 7.105998e-09 |   60 | 0.313149100 |  3 |   9.5801020 | 0.030612240 | 0.050000000 | 0.0038035080 | 0.7514860 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | down | GO:0032453 | histone demethylase activity (H3-K4 specific)                                             | 1.087137e-03 | 0.088058070 |  5 |  56.780710 | 0.06172840 | 3.631956e-08 | 3.066824e-05 |    8 | 0.041753210 |  1 |  23.9502600 | 0.010204080 | 0.125000000 | 0.0410059200 | 1.0000000 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | down | GO:0071558 | histone demethylase activity (H3-K27 specific)                                            | 7.155255e-04 | 0.057957570 |  4 |  69.016010 | 0.04938272 | 4.173072e-07 | 2.936452e-04 |    5 | 0.026095760 |  1 |  38.3204100 | 0.010204080 | 0.200000000 | 0.0258275000 | 1.0000000 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | down | GO:0050681 | androgen receptor binding                                                                 | 5.738002e-03 | 0.464778100 |  7 |  15.060950 | 0.08641975 | 4.912225e-07 | 2.962773e-04 |   43 | 0.224423500 |  3 |  13.3675800 | 0.030612240 | 0.069767440 | 0.0014622900 | 0.7514860 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | down | GO:0003735 | structural constituent of ribosome                                                        | 8.019204e-03 | 0.649555500 |  7 |  10.776600 | 0.08641975 | 4.413757e-06 | 2.329360e-03 |  162 | 0.845502500 |  2 |   2.3654570 | 0.020408160 | 0.012345680 | 0.2072462000 | 1.0000000 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | down | GO:0035258 | steroid hormone receptor binding                                                          | 1.030543e-02 | 0.834739700 |  7 |   8.385848 | 0.08641975 | 2.203871e-05 | 1.033860e-02 |   88 | 0.459285300 |  3 |   6.5318880 | 0.030612240 | 0.034090910 | 0.0109834000 | 0.9866365 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | down | GO:0035257 | nuclear hormone receptor binding                                                          | 1.462713e-02 | 1.184797000 |  8 |   6.752209 | 0.09876543 | 2.608199e-05 | 1.101182e-02 |  137 | 0.715023700 |  4 |   5.5942200 | 0.040816330 | 0.029197080 | 0.0057714330 | 0.7860319 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | down | GO:0051427 | hormone receptor binding                                                                  | 1.666248e-02 | 1.349661000 |  8 |   5.927416 | 0.09876543 | 6.480817e-05 | 2.487455e-02 |  161 | 0.840283300 |  4 |   4.7602990 | 0.040816330 | 0.024844720 | 0.0100826900 | 0.9768151 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | down | GO:0042799 | histone methyltransferase activity (H4-K20 specific)                                      | 2.066923e-04 | 0.016742080 |  2 | 119.459500 | 0.02469136 | 1.369206e-04 | 4.817323e-02 |    5 | 0.026095760 |  2 |  76.6408200 | 0.020408160 | 0.400000000 | 0.0002668833 | 0.5633906 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | up   | GO:0008349 | MAP kinase kinase kinase kinase activity                                                  | 1.474917e-03 | 0.131267600 | 11 |  83.798270 | 0.12359550 | 2.361332e-18 | 9.969544e-15 |   11 | 0.064440540 |  1 |  15.5181800 | 0.009090909 | 0.090909090 | 0.0626019700 | 1.0000000 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | up   | GO:0004702 | signal transducer, downstream of receptor, with serine/threonine kinase activity          | 7.837579e-03 | 0.697544600 | 11 |  15.769600 | 0.12359550 | 1.428125e-10 | 3.014772e-07 |   70 | 0.410076200 |  1 |   2.4385710 | 0.009090909 | 0.014285710 | 0.3377034000 | 1.0000000 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | up   | GO:0005057 | signal transducer activity, downstream of receptor                                        | 1.441310e-02 | 1.282766000 | 13 |  10.134350 | 0.14606740 | 5.865720e-10 | 8.255023e-07 |  122 | 0.714704200 |  2 |   2.7983610 | 0.018181820 | 0.016393440 | 0.1603025000 | 1.0000000 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | up   | GO:0004984 | olfactory receptor activity                                                               | 1.019862e-02 | 0.907677400 | 11 |  12.118840 | 0.12359550 | 2.183453e-09 | 2.304635e-06 |  357 | 2.091388000 |  1 |   0.4781513 | 0.009090909 | 0.002801120 | 0.8796965000 | 1.0000000 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | up   | GO:0000287 | magnesium ion binding                                                                     | 1.826515e-02 | 1.625598000 | 11 |   6.766739 | 0.12359550 | 7.433790e-07 | 6.277092e-04 |  199 | 1.165788000 |  1 |   0.8577889 | 0.009090909 | 0.005025126 | 0.6913177000 | 1.0000000 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | up   | GO:0004000 | adenosine deaminase activity                                                              | 1.260969e-03 | 0.112226200 |  4 |  35.642290 | 0.04494382 | 5.666375e-06 | 3.987239e-03 |   11 | 0.064440540 |  1 |  15.5181800 | 0.009090909 | 0.090909090 | 0.0626019700 | 1.0000000 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | up   | GO:0003727 | single-stranded RNA binding                                                               | 7.803229e-03 | 0.694487400 |  7 |  10.079380 | 0.07865169 | 6.938828e-06 | 4.185105e-03 |   81 | 0.474516700 |  3 |   6.3222220 | 0.027272730 | 0.037037040 | 0.0119890800 | 1.0000000 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | up   | GO:0004930 | G-protein coupled receptor activity                                                       | 5.726050e-02 | 5.096184000 | 17 |   3.335829 | 0.19101120 | 1.090262e-05 | 5.753858e-03 |  798 | 4.674868000 |  8 |   1.7112780 | 0.072727270 | 0.010025060 | 0.0964867900 | 1.0000000 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | up   | GO:0030616 | transforming growth factor beta receptor, common-partner cytoplasmic mediator activity    | 7.737478e-05 | 0.006886355 |  2 | 290.429400 | 0.02247191 | 2.333958e-05 | 1.094886e-02 |    1 | 0.005858231 |  1 | 170.7000000 | 0.009090909 | 1.000000000 | 0.0058582310 | 1.0000000 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | up   | GO:0004674 | protein serine/threonine kinase activity                                                  | 4.525708e-02 | 4.027880000 | 14 |   3.475773 | 0.15730340 | 4.659862e-05 | 1.967394e-02 |  449 | 2.630346000 |  4 |   1.5207130 | 0.036363640 | 0.008908686 | 0.2693638000 | 1.0000000 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | up   | GO:0004888 | transmembrane signaling receptor activity                                                 | 1.020089e-01 | 9.078791000 | 22 |   2.423230 | 0.24719100 | 6.814491e-05 | 2.615526e-02 | 1207 | 7.070885000 | 13 |   1.8385250 | 0.118181800 | 0.010770510 | 0.0241704600 | 1.0000000 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | up   | GO:0019239 | deaminase activity                                                                        | 2.504054e-03 | 0.222860800 |  4 |  17.948420 | 0.04494382 | 8.100153e-05 | 2.849904e-02 |   34 | 0.199179800 |  1 |   5.0205880 | 0.009090909 | 0.029411760 | 0.1812216000 | 1.0000000 |\n",
       "| Bn_TCL1A          | Underweight_vs_Normal | up   | GO:0016814 | hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines | 2.726487e-03 | 0.242657400 |  4 |  16.484150 | 0.04494382 | 1.121500e-04 | 3.642287e-02 |   35 | 0.205038100 |  1 |   4.8771430 | 0.009090909 | 0.028571430 | 0.1860269000 | 1.0000000 |\n",
       "| ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Normal     | up   | GO:0048037 | cofactor binding                                                               | 0.0208731400 |  17.72130000 |  35 |   1.975025 | 0.041224970 | 1.562711e-04 | 2.999790e-02 |   265 | 1.867151e+01 |  26 |   1.392496 | 0.0196523100 | 0.09811321 | 5.449844e-02 | 0.937493115 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Normal     | up   | GO:0050662 | coenzyme binding                                                               | 0.0136182200 |  11.56187000 |  26 |   2.248772 | 0.030624260 | 1.563131e-04 | 2.999790e-02 |   184 | 1.296437e+01 |  18 |   1.388421 | 0.0136054400 | 0.09782609 | 9.849451e-02 | 1.000000000 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Normal     | up   | GO:0005070 | SH3/SH2 adaptor activity                                                       | 0.0064376140 |   5.46553400 |  16 |   2.927436 | 0.018845700 | 1.755588e-04 | 3.222649e-02 |    57 | 4.016137e+00 |  12 |   2.987946 | 0.0090702950 | 0.21052630 | 5.178047e-04 | 0.095050932 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Normal     | up   | GO:0003690 | double-stranded DNA binding                                                    | 0.0949064300 |  80.57556000 | 113 |   1.402410 | 0.133097800 | 1.831933e-04 | 3.222675e-02 |   809 | 5.700096e+01 |  85 |   1.491203 | 0.0642479200 | 0.10506800 | 1.311516e-04 | 0.048230861 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Normal     | up   | GO:0003677 | DNA binding                                                                    | 0.2140410000 | 181.72080000 | 225 |   1.238163 | 0.265017700 | 2.367535e-04 | 3.986706e-02 |  2476 | 1.744553e+02 | 191 |   1.094836 | 0.1443689000 | 0.07714055 | 8.925708e-02 | 1.000000000 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Normal     | up   | GO:0008889 | glycerophosphodiester phosphodiesterase activity                               | 0.0006400373 |   0.54339170 |   5 |   9.201466 | 0.005889282 | 2.494599e-04 | 3.986706e-02 |     6 | 4.227512e-01 |   3 |   7.096372 | 0.0022675740 | 0.50000000 | 5.937450e-03 | 0.358113056 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Normal     | up   | GO:0030159 | receptor signaling complex scaffold activity                                   | 0.0023874270 |   2.02692600 |   9 |   4.440222 | 0.010600710 | 2.549528e-04 | 3.986706e-02 |    23 | 1.620546e+00 |   7 |   4.319531 | 0.0052910050 | 0.30434780 | 7.624330e-04 | 0.114964004 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Normal     | up   | GO:0070403 | NAD+ binding                                                                   | 0.0010140410 |   0.86092090 |   6 |   6.969281 | 0.007067138 | 2.685020e-04 | 4.048627e-02 |    14 | 9.864196e-01 |   3 |   3.041302 | 0.0022675740 | 0.21428570 | 7.083380e-02 | 0.937493115 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Normal     | up   | GO:0001517 | N-acetylglucosamine 6-O-sulfotransferase activity                              | 0.0006633551 |   0.56318840 |   5 |   8.878023 | 0.005889282 | 2.935458e-04 | 4.273622e-02 |     7 | 4.932098e-01 |   4 |   8.110139 | 0.0030234320 | 0.57142860 | 7.224176e-04 | 0.112964708 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Normal     | up   | GO:0001784 | phosphotyrosine binding                                                        | 0.0041880240 |   3.55563200 |  12 |   3.374927 | 0.014134280 | 3.194725e-04 | 4.363090e-02 |    35 | 2.466049e+00 |  10 |   4.055070 | 0.0075585790 | 0.28571430 | 1.049218e-04 | 0.047261237 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Normal     | up   | GO:1990837 | sequence-specific double-stranded DNA binding                                  | 0.0893120800 |  75.82596000 | 106 |   1.397938 | 0.124852800 | 3.302948e-04 | 4.363090e-02 |   731 | 5.150519e+01 |  80 |   1.553241 | 0.0604686300 | 0.10943910 | 5.369848e-05 | 0.032387855 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Normal     | up   | GO:0000976 | transcription regulatory region sequence-specific DNA binding                  | 0.0862857200 |  73.25658000 | 103 |   1.406017 | 0.121319200 | 3.306937e-04 | 4.363090e-02 |   692 | 4.875731e+01 |  77 |   1.579250 | 0.0582010600 | 0.11127170 | 4.253793e-05 | 0.029932523 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Normal     | up   | GO:0000981 | RNA polymerase II transcription factor activity, sequence-specific DNA binding | 0.1006345000 |  85.43870000 | 117 |   1.369403 | 0.137809200 | 3.468743e-04 | 4.428693e-02 |   766 | 5.397124e+01 |  88 |   1.630498 | 0.0665155000 | 0.11488250 | 3.455085e-06 | 0.004862456 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Normal     | up   | GO:0001096 | TFIIF-class transcription factor binding                                       | 0.0001570776 |   0.13335890 |   3 |  22.495690 | 0.003533569 | 3.566451e-04 | 4.428693e-02 |     1 | 7.045854e-02 |   1 |  14.192740 | 0.0007558579 | 1.00000000 | 7.045854e-02 | 0.937493115 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Normal     | up   | GO:0008446 | GDP-mannose 4,6-dehydratase activity                                           | 0.0003877198 |   0.32917410 |   4 |  12.151620 | 0.004711425 | 3.742874e-04 | 4.514975e-02 |     1 | 7.045854e-02 |   1 |  14.192740 | 0.0007558579 | 1.00000000 | 7.045854e-02 | 0.937493115 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Normal     | up   | GO:0070401 | NADP+ binding                                                                  | 0.0003945207 |   0.33494810 |   4 |  11.942150 | 0.004711425 | 3.994276e-04 | 4.684398e-02 |     2 | 1.409171e-01 |   1 |   7.096372 | 0.0007558579 | 0.50000000 | 1.359562e-01 | 1.000000000 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Overweight | down | GO:0019843 | rRNA binding                                                                   | 0.0019818970 |   0.04360174 |   5 | 114.674300 | 0.227272700 | 7.829313e-10 | 3.305536e-06 |    60 | 5.751718e-02 |   1 |  17.386110 | 0.0555555600 | 0.01666667 | 5.600594e-02 | 1.000000000 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Overweight | down | GO:0032452 | histone demethylase activity                                                   | 0.0035947680 |   0.07908489 |   5 |  63.223200 | 0.227272700 | 1.502230e-08 | 3.171208e-05 |    28 | 2.684135e-02 |   2 |  74.511900 | 0.1111111000 | 0.07142857 | 3.232730e-04 | 1.000000000 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Overweight | down | GO:0032451 | demethylase activity                                                           | 0.0041287750 |   0.09083304 |   5 |  55.046050 | 0.227272700 | 2.979867e-08 | 4.193666e-05 |    38 | 3.642754e-02 |   2 |  54.903510 | 0.1111111000 | 0.05263158 | 5.978157e-04 | 1.000000000 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Overweight | down | GO:0003735 | structural constituent of ribosome                                             | 0.0080192040 |   0.17642250 |   5 |  28.341060 | 0.227272700 | 7.793745e-07 | 7.155144e-04 |   162 | 1.552964e-01 |   1 |   6.439300 | 0.0555555600 | 0.00617284 | 1.444760e-01 | 1.000000000 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Overweight | down | GO:0051213 | dioxygenase activity                                                           | 0.0081576870 |   0.17946910 |   5 |  27.859950 | 0.227272700 | 8.473643e-07 | 7.155144e-04 |    89 | 8.531714e-02 |   2 |  23.441950 | 0.1111111000 | 0.02247191 | 3.235131e-03 | 1.000000000 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Overweight | down | GO:0032453 | histone demethylase activity (H3-K4 specific)                                  | 0.0010871370 |   0.02391701 |   3 | 125.433800 | 0.136363600 | 1.948255e-06 | 1.370922e-03 |     8 | 7.668957e-03 |   1 | 130.395800 | 0.0555555600 | 0.12500000 | 7.644696e-03 | 1.000000000 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Overweight | up   | GO:0051378 | serotonin binding                                                              | 0.0029198980 |   0.50222240 |   8 |  15.929200 | 0.046511630 | 5.562894e-08 | 2.252977e-04 |    11 | 1.675454e-01 |   1 |   5.968531 | 0.0034965030 | 0.09090909 | 1.553896e-01 | 1.000000000 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Overweight | up   | GO:0004993 | G-protein coupled serotonin receptor activity                                  | 0.0033145340 |   0.57009980 |   8 |  14.032630 | 0.046511630 | 1.448213e-07 | 2.252977e-04 |    22 | 3.350908e-01 |   1 |   2.984266 | 0.0034965030 | 0.04545455 | 2.867044e-01 | 1.000000000 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Overweight | up   | GO:0099589 | serotonin receptor activity                                                    | 0.0034451000 |   0.59255720 |   8 |  13.500810 | 0.046511630 | 1.935662e-07 | 2.252977e-04 |    29 | 4.417106e-01 |   1 |   2.263926 | 0.0034965030 | 0.03448276 | 3.594601e-01 | 1.000000000 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Overweight | up   | GO:0005201 | extracellular matrix structural constituent                                    | 0.0081131180 |   1.39545600 |  11 |   7.882726 | 0.063953490 | 2.134512e-07 | 2.252977e-04 |    78 | 1.188049e+00 |   4 |   3.366864 | 0.0139860100 | 0.05128205 | 3.124303e-02 | 1.000000000 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Overweight | up   | GO:0005515 | protein binding                                                                | 0.7330151000 | 126.07860000 | 151 |   1.197666 | 0.877907000 | 2.969338e-06 | 2.507309e-03 | 11170 | 1.701347e+02 | 197 |   1.157906 | 0.6888112000 | 0.01763653 | 5.874243e-04 | 0.620026349 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Overweight | up   | GO:0070405 | ammonium ion binding                                                           | 0.0080907640 |   1.39161100 |   9 |   6.467323 | 0.052325580 | 1.334797e-05 | 9.392522e-03 |    71 | 1.081429e+00 |   2 |   1.849404 | 0.0069930070 | 0.02816901 | 2.944349e-01 | 1.000000000 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Overweight | up   | GO:0008227 | G-protein coupled amine receptor activity                                      | 0.0070103290 |   1.20577700 |   8 |   6.634728 | 0.046511630 | 3.393613e-05 | 2.046833e-02 |    48 | 7.311072e-01 |   1 |   1.367788 | 0.0034965030 | 0.02083333 | 5.217698e-01 | 1.000000000 |\n",
       "| Unswitched_Bm_CD1C | Underweight_vs_Overweight | up   | GO:0030159 | receptor signaling complex scaffold activity                                   | 0.0023874270 |   0.41063750 |   5 |  12.176190 | 0.029069770 | 6.590213e-05 | 3.477985e-02 |    23 | 3.503222e-01 |   4 |  11.418060 | 0.0139860100 | 0.17391300 | 3.713652e-04 | 0.522634625 |\n",
       "\n"
      ],
      "text/plain": [
       "     cell_type          contrast                  direction ID        \n",
       "1    Atypical_Bm_ITGAX  Underweight_vs_Normal     down      GO:0019843\n",
       "2    Atypical_Bm_ITGAX  Underweight_vs_Normal     down      GO:0032452\n",
       "3    Atypical_Bm_ITGAX  Underweight_vs_Normal     down      GO:0032451\n",
       "4    Atypical_Bm_ITGAX  Underweight_vs_Normal     down      GO:0003735\n",
       "5    Atypical_Bm_ITGAX  Underweight_vs_Normal     down      GO:0051213\n",
       "6    Bn_TCL1A           Underweight_vs_Normal     down      GO:0032452\n",
       "7    Bn_TCL1A           Underweight_vs_Normal     down      GO:0032451\n",
       "8    Bn_TCL1A           Underweight_vs_Normal     down      GO:0051213\n",
       "9    Bn_TCL1A           Underweight_vs_Normal     down      GO:0019843\n",
       "10   Bn_TCL1A           Underweight_vs_Normal     down      GO:0032453\n",
       "11   Bn_TCL1A           Underweight_vs_Normal     down      GO:0071558\n",
       "12   Bn_TCL1A           Underweight_vs_Normal     down      GO:0050681\n",
       "13   Bn_TCL1A           Underweight_vs_Normal     down      GO:0003735\n",
       "14   Bn_TCL1A           Underweight_vs_Normal     down      GO:0035258\n",
       "15   Bn_TCL1A           Underweight_vs_Normal     down      GO:0035257\n",
       "16   Bn_TCL1A           Underweight_vs_Normal     down      GO:0051427\n",
       "17   Bn_TCL1A           Underweight_vs_Normal     down      GO:0042799\n",
       "18   Bn_TCL1A           Underweight_vs_Normal     up        GO:0008349\n",
       "19   Bn_TCL1A           Underweight_vs_Normal     up        GO:0004702\n",
       "20   Bn_TCL1A           Underweight_vs_Normal     up        GO:0005057\n",
       "21   Bn_TCL1A           Underweight_vs_Normal     up        GO:0004984\n",
       "22   Bn_TCL1A           Underweight_vs_Normal     up        GO:0000287\n",
       "23   Bn_TCL1A           Underweight_vs_Normal     up        GO:0004000\n",
       "24   Bn_TCL1A           Underweight_vs_Normal     up        GO:0003727\n",
       "25   Bn_TCL1A           Underweight_vs_Normal     up        GO:0004930\n",
       "26   Bn_TCL1A           Underweight_vs_Normal     up        GO:0030616\n",
       "27   Bn_TCL1A           Underweight_vs_Normal     up        GO:0004674\n",
       "28   Bn_TCL1A           Underweight_vs_Normal     up        GO:0004888\n",
       "29   Bn_TCL1A           Underweight_vs_Normal     up        GO:0019239\n",
       "30   Bn_TCL1A           Underweight_vs_Normal     up        GO:0016814\n",
       "⋮    ⋮                  ⋮                         ⋮         ⋮         \n",
       "2604 Unswitched_Bm_CD1C Underweight_vs_Normal     up        GO:0048037\n",
       "2605 Unswitched_Bm_CD1C Underweight_vs_Normal     up        GO:0050662\n",
       "2606 Unswitched_Bm_CD1C Underweight_vs_Normal     up        GO:0005070\n",
       "2607 Unswitched_Bm_CD1C Underweight_vs_Normal     up        GO:0003690\n",
       "2608 Unswitched_Bm_CD1C Underweight_vs_Normal     up        GO:0003677\n",
       "2609 Unswitched_Bm_CD1C Underweight_vs_Normal     up        GO:0008889\n",
       "2610 Unswitched_Bm_CD1C Underweight_vs_Normal     up        GO:0030159\n",
       "2611 Unswitched_Bm_CD1C Underweight_vs_Normal     up        GO:0070403\n",
       "2612 Unswitched_Bm_CD1C Underweight_vs_Normal     up        GO:0001517\n",
       "2613 Unswitched_Bm_CD1C Underweight_vs_Normal     up        GO:0001784\n",
       "2614 Unswitched_Bm_CD1C Underweight_vs_Normal     up        GO:1990837\n",
       "2615 Unswitched_Bm_CD1C Underweight_vs_Normal     up        GO:0000976\n",
       "2616 Unswitched_Bm_CD1C Underweight_vs_Normal     up        GO:0000981\n",
       "2617 Unswitched_Bm_CD1C Underweight_vs_Normal     up        GO:0001096\n",
       "2618 Unswitched_Bm_CD1C Underweight_vs_Normal     up        GO:0008446\n",
       "2619 Unswitched_Bm_CD1C Underweight_vs_Normal     up        GO:0070401\n",
       "2620 Unswitched_Bm_CD1C Underweight_vs_Overweight down      GO:0019843\n",
       "2621 Unswitched_Bm_CD1C Underweight_vs_Overweight down      GO:0032452\n",
       "2622 Unswitched_Bm_CD1C Underweight_vs_Overweight down      GO:0032451\n",
       "2623 Unswitched_Bm_CD1C Underweight_vs_Overweight down      GO:0003735\n",
       "2624 Unswitched_Bm_CD1C Underweight_vs_Overweight down      GO:0051213\n",
       "2625 Unswitched_Bm_CD1C Underweight_vs_Overweight down      GO:0032453\n",
       "2626 Unswitched_Bm_CD1C Underweight_vs_Overweight up        GO:0051378\n",
       "2627 Unswitched_Bm_CD1C Underweight_vs_Overweight up        GO:0004993\n",
       "2628 Unswitched_Bm_CD1C Underweight_vs_Overweight up        GO:0099589\n",
       "2629 Unswitched_Bm_CD1C Underweight_vs_Overweight up        GO:0005201\n",
       "2630 Unswitched_Bm_CD1C Underweight_vs_Overweight up        GO:0005515\n",
       "2631 Unswitched_Bm_CD1C Underweight_vs_Overweight up        GO:0070405\n",
       "2632 Unswitched_Bm_CD1C Underweight_vs_Overweight up        GO:0008227\n",
       "2633 Unswitched_Bm_CD1C Underweight_vs_Overweight up        GO:0030159\n",
       "     name                                                                                     \n",
       "1    rRNA binding                                                                             \n",
       "2    histone demethylase activity                                                             \n",
       "3    demethylase activity                                                                     \n",
       "4    structural constituent of ribosome                                                       \n",
       "5    dioxygenase activity                                                                     \n",
       "6    histone demethylase activity                                                             \n",
       "7    demethylase activity                                                                     \n",
       "8    dioxygenase activity                                                                     \n",
       "9    rRNA binding                                                                             \n",
       "10   histone demethylase activity (H3-K4 specific)                                            \n",
       "11   histone demethylase activity (H3-K27 specific)                                           \n",
       "12   androgen receptor binding                                                                \n",
       "13   structural constituent of ribosome                                                       \n",
       "14   steroid hormone receptor binding                                                         \n",
       "15   nuclear hormone receptor binding                                                         \n",
       "16   hormone receptor binding                                                                 \n",
       "17   histone methyltransferase activity (H4-K20 specific)                                     \n",
       "18   MAP kinase kinase kinase kinase activity                                                 \n",
       "19   signal transducer, downstream of receptor, with serine/threonine kinase activity         \n",
       "20   signal transducer activity, downstream of receptor                                       \n",
       "21   olfactory receptor activity                                                              \n",
       "22   magnesium ion binding                                                                    \n",
       "23   adenosine deaminase activity                                                             \n",
       "24   single-stranded RNA binding                                                              \n",
       "25   G-protein coupled receptor activity                                                      \n",
       "26   transforming growth factor beta receptor, common-partner cytoplasmic mediator activity   \n",
       "27   protein serine/threonine kinase activity                                                 \n",
       "28   transmembrane signaling receptor activity                                                \n",
       "29   deaminase activity                                                                       \n",
       "30   hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines\n",
       "⋮    ⋮                                                                                        \n",
       "2604 cofactor binding                                                                         \n",
       "2605 coenzyme binding                                                                         \n",
       "2606 SH3/SH2 adaptor activity                                                                 \n",
       "2607 double-stranded DNA binding                                                              \n",
       "2608 DNA binding                                                                              \n",
       "2609 glycerophosphodiester phosphodiesterase activity                                         \n",
       "2610 receptor signaling complex scaffold activity                                             \n",
       "2611 NAD+ binding                                                                             \n",
       "2612 N-acetylglucosamine 6-O-sulfotransferase activity                                        \n",
       "2613 phosphotyrosine binding                                                                  \n",
       "2614 sequence-specific double-stranded DNA binding                                            \n",
       "2615 transcription regulatory region sequence-specific DNA binding                            \n",
       "2616 RNA polymerase II transcription factor activity, sequence-specific DNA binding           \n",
       "2617 TFIIF-class transcription factor binding                                                 \n",
       "2618 GDP-mannose 4,6-dehydratase activity                                                     \n",
       "2619 NADP+ binding                                                                            \n",
       "2620 rRNA binding                                                                             \n",
       "2621 histone demethylase activity                                                             \n",
       "2622 demethylase activity                                                                     \n",
       "2623 structural constituent of ribosome                                                       \n",
       "2624 dioxygenase activity                                                                     \n",
       "2625 histone demethylase activity (H3-K4 specific)                                            \n",
       "2626 serotonin binding                                                                        \n",
       "2627 G-protein coupled serotonin receptor activity                                            \n",
       "2628 serotonin receptor activity                                                              \n",
       "2629 extracellular matrix structural constituent                                              \n",
       "2630 protein binding                                                                          \n",
       "2631 ammonium ion binding                                                                     \n",
       "2632 G-protein coupled amine receptor activity                                                \n",
       "2633 receptor signaling complex scaffold activity                                             \n",
       "     Binom_Genome_Fraction Binom_Expected Binom_Observed_Region_Hits\n",
       "1    1.981897e-03          0.035674150     4                        \n",
       "2    3.594768e-03          0.064705820     4                        \n",
       "3    4.128775e-03          0.074317940     4                        \n",
       "4    8.019204e-03          0.144345700     4                        \n",
       "5    8.157687e-03          0.146838400     4                        \n",
       "6    3.594768e-03          0.291176200    10                        \n",
       "7    4.128775e-03          0.334430800    10                        \n",
       "8    8.157687e-03          0.660772700    12                        \n",
       "9    1.981897e-03          0.160533700     8                        \n",
       "10   1.087137e-03          0.088058070     5                        \n",
       "11   7.155255e-04          0.057957570     4                        \n",
       "12   5.738002e-03          0.464778100     7                        \n",
       "13   8.019204e-03          0.649555500     7                        \n",
       "14   1.030543e-02          0.834739700     7                        \n",
       "15   1.462713e-02          1.184797000     8                        \n",
       "16   1.666248e-02          1.349661000     8                        \n",
       "17   2.066923e-04          0.016742080     2                        \n",
       "18   1.474917e-03          0.131267600    11                        \n",
       "19   7.837579e-03          0.697544600    11                        \n",
       "20   1.441310e-02          1.282766000    13                        \n",
       "21   1.019862e-02          0.907677400    11                        \n",
       "22   1.826515e-02          1.625598000    11                        \n",
       "23   1.260969e-03          0.112226200     4                        \n",
       "24   7.803229e-03          0.694487400     7                        \n",
       "25   5.726050e-02          5.096184000    17                        \n",
       "26   7.737478e-05          0.006886355     2                        \n",
       "27   4.525708e-02          4.027880000    14                        \n",
       "28   1.020089e-01          9.078791000    22                        \n",
       "29   2.504054e-03          0.222860800     4                        \n",
       "30   2.726487e-03          0.242657400     4                        \n",
       "⋮    ⋮                     ⋮              ⋮                         \n",
       "2604 0.0208731400           17.72130000    35                       \n",
       "2605 0.0136182200           11.56187000    26                       \n",
       "2606 0.0064376140            5.46553400    16                       \n",
       "2607 0.0949064300           80.57556000   113                       \n",
       "2608 0.2140410000          181.72080000   225                       \n",
       "2609 0.0006400373            0.54339170     5                       \n",
       "2610 0.0023874270            2.02692600     9                       \n",
       "2611 0.0010140410            0.86092090     6                       \n",
       "2612 0.0006633551            0.56318840     5                       \n",
       "2613 0.0041880240            3.55563200    12                       \n",
       "2614 0.0893120800           75.82596000   106                       \n",
       "2615 0.0862857200           73.25658000   103                       \n",
       "2616 0.1006345000           85.43870000   117                       \n",
       "2617 0.0001570776            0.13335890     3                       \n",
       "2618 0.0003877198            0.32917410     4                       \n",
       "2619 0.0003945207            0.33494810     4                       \n",
       "2620 0.0019818970            0.04360174     5                       \n",
       "2621 0.0035947680            0.07908489     5                       \n",
       "2622 0.0041287750            0.09083304     5                       \n",
       "2623 0.0080192040            0.17642250     5                       \n",
       "2624 0.0081576870            0.17946910     5                       \n",
       "2625 0.0010871370            0.02391701     3                       \n",
       "2626 0.0029198980            0.50222240     8                       \n",
       "2627 0.0033145340            0.57009980     8                       \n",
       "2628 0.0034451000            0.59255720     8                       \n",
       "2629 0.0081131180            1.39545600    11                       \n",
       "2630 0.7330151000          126.07860000   151                       \n",
       "2631 0.0080907640            1.39161100     9                       \n",
       "2632 0.0070103290            1.20577700     8                       \n",
       "2633 0.0023874270            0.41063750     5                       \n",
       "     Binom_Fold_Enrichment Binom_Region_Set_Coverage Binom_Raw_PValue\n",
       "1    112.126000            0.22222220                4.617453e-08    \n",
       "2     61.818240            0.22222220                4.908039e-07    \n",
       "3     53.822800            0.22222220                8.490012e-07    \n",
       "4     27.711260            0.22222220                1.156601e-05    \n",
       "5     27.240840            0.22222220                1.236663e-05    \n",
       "6     34.343470            0.12345680                5.366114e-13    \n",
       "7     29.901560            0.12345680                2.070966e-12    \n",
       "8     18.160560            0.14814810                3.649426e-12    \n",
       "9     49.833770            0.09876543                6.732352e-12    \n",
       "10    56.780710            0.06172840                3.631956e-08    \n",
       "11    69.016010            0.04938272                4.173072e-07    \n",
       "12    15.060950            0.08641975                4.912225e-07    \n",
       "13    10.776600            0.08641975                4.413757e-06    \n",
       "14     8.385848            0.08641975                2.203871e-05    \n",
       "15     6.752209            0.09876543                2.608199e-05    \n",
       "16     5.927416            0.09876543                6.480817e-05    \n",
       "17   119.459500            0.02469136                1.369206e-04    \n",
       "18    83.798270            0.12359550                2.361332e-18    \n",
       "19    15.769600            0.12359550                1.428125e-10    \n",
       "20    10.134350            0.14606740                5.865720e-10    \n",
       "21    12.118840            0.12359550                2.183453e-09    \n",
       "22     6.766739            0.12359550                7.433790e-07    \n",
       "23    35.642290            0.04494382                5.666375e-06    \n",
       "24    10.079380            0.07865169                6.938828e-06    \n",
       "25     3.335829            0.19101120                1.090262e-05    \n",
       "26   290.429400            0.02247191                2.333958e-05    \n",
       "27     3.475773            0.15730340                4.659862e-05    \n",
       "28     2.423230            0.24719100                6.814491e-05    \n",
       "29    17.948420            0.04494382                8.100153e-05    \n",
       "30    16.484150            0.04494382                1.121500e-04    \n",
       "⋮    ⋮                     ⋮                         ⋮               \n",
       "2604   1.975025            0.041224970               1.562711e-04    \n",
       "2605   2.248772            0.030624260               1.563131e-04    \n",
       "2606   2.927436            0.018845700               1.755588e-04    \n",
       "2607   1.402410            0.133097800               1.831933e-04    \n",
       "2608   1.238163            0.265017700               2.367535e-04    \n",
       "2609   9.201466            0.005889282               2.494599e-04    \n",
       "2610   4.440222            0.010600710               2.549528e-04    \n",
       "2611   6.969281            0.007067138               2.685020e-04    \n",
       "2612   8.878023            0.005889282               2.935458e-04    \n",
       "2613   3.374927            0.014134280               3.194725e-04    \n",
       "2614   1.397938            0.124852800               3.302948e-04    \n",
       "2615   1.406017            0.121319200               3.306937e-04    \n",
       "2616   1.369403            0.137809200               3.468743e-04    \n",
       "2617  22.495690            0.003533569               3.566451e-04    \n",
       "2618  12.151620            0.004711425               3.742874e-04    \n",
       "2619  11.942150            0.004711425               3.994276e-04    \n",
       "2620 114.674300            0.227272700               7.829313e-10    \n",
       "2621  63.223200            0.227272700               1.502230e-08    \n",
       "2622  55.046050            0.227272700               2.979867e-08    \n",
       "2623  28.341060            0.227272700               7.793745e-07    \n",
       "2624  27.859950            0.227272700               8.473643e-07    \n",
       "2625 125.433800            0.136363600               1.948255e-06    \n",
       "2626  15.929200            0.046511630               5.562894e-08    \n",
       "2627  14.032630            0.046511630               1.448213e-07    \n",
       "2628  13.500810            0.046511630               1.935662e-07    \n",
       "2629   7.882726            0.063953490               2.134512e-07    \n",
       "2630   1.197666            0.877907000               2.969338e-06    \n",
       "2631   6.467323            0.052325580               1.334797e-05    \n",
       "2632   6.634728            0.046511630               3.393613e-05    \n",
       "2633  12.176190            0.029069770               6.590213e-05    \n",
       "     Binom_Adjp_BH Hyper_Total_Genes Hyper_Expected Hyper_Observed_Gene_Hits\n",
       "1    1.949489e-04    60              0.047930980     1                      \n",
       "2    1.036087e-03    28              0.022367790     2                      \n",
       "3    1.194828e-03    38              0.030356290     2                      \n",
       "4    1.044238e-02   162              0.129413600     1                      \n",
       "5    1.044238e-02    89              0.071097620     2                      \n",
       "6    2.265573e-09    28              0.146136200     3                      \n",
       "7    4.371809e-09    38              0.198327700     3                      \n",
       "8    5.135959e-09    89              0.464504400     5                      \n",
       "9    7.105998e-09    60              0.313149100     3                      \n",
       "10   3.066824e-05     8              0.041753210     1                      \n",
       "11   2.936452e-04     5              0.026095760     1                      \n",
       "12   2.962773e-04    43              0.224423500     3                      \n",
       "13   2.329360e-03   162              0.845502500     2                      \n",
       "14   1.033860e-02    88              0.459285300     3                      \n",
       "15   1.101182e-02   137              0.715023700     4                      \n",
       "16   2.487455e-02   161              0.840283300     4                      \n",
       "17   4.817323e-02     5              0.026095760     2                      \n",
       "18   9.969544e-15    11              0.064440540     1                      \n",
       "19   3.014772e-07    70              0.410076200     1                      \n",
       "20   8.255023e-07   122              0.714704200     2                      \n",
       "21   2.304635e-06   357              2.091388000     1                      \n",
       "22   6.277092e-04   199              1.165788000     1                      \n",
       "23   3.987239e-03    11              0.064440540     1                      \n",
       "24   4.185105e-03    81              0.474516700     3                      \n",
       "25   5.753858e-03   798              4.674868000     8                      \n",
       "26   1.094886e-02     1              0.005858231     1                      \n",
       "27   1.967394e-02   449              2.630346000     4                      \n",
       "28   2.615526e-02  1207              7.070885000    13                      \n",
       "29   2.849904e-02    34              0.199179800     1                      \n",
       "30   3.642287e-02    35              0.205038100     1                      \n",
       "⋮    ⋮             ⋮                 ⋮              ⋮                       \n",
       "2604 2.999790e-02    265             1.867151e+01    26                     \n",
       "2605 2.999790e-02    184             1.296437e+01    18                     \n",
       "2606 3.222649e-02     57             4.016137e+00    12                     \n",
       "2607 3.222675e-02    809             5.700096e+01    85                     \n",
       "2608 3.986706e-02   2476             1.744553e+02   191                     \n",
       "2609 3.986706e-02      6             4.227512e-01     3                     \n",
       "2610 3.986706e-02     23             1.620546e+00     7                     \n",
       "2611 4.048627e-02     14             9.864196e-01     3                     \n",
       "2612 4.273622e-02      7             4.932098e-01     4                     \n",
       "2613 4.363090e-02     35             2.466049e+00    10                     \n",
       "2614 4.363090e-02    731             5.150519e+01    80                     \n",
       "2615 4.363090e-02    692             4.875731e+01    77                     \n",
       "2616 4.428693e-02    766             5.397124e+01    88                     \n",
       "2617 4.428693e-02      1             7.045854e-02     1                     \n",
       "2618 4.514975e-02      1             7.045854e-02     1                     \n",
       "2619 4.684398e-02      2             1.409171e-01     1                     \n",
       "2620 3.305536e-06     60             5.751718e-02     1                     \n",
       "2621 3.171208e-05     28             2.684135e-02     2                     \n",
       "2622 4.193666e-05     38             3.642754e-02     2                     \n",
       "2623 7.155144e-04    162             1.552964e-01     1                     \n",
       "2624 7.155144e-04     89             8.531714e-02     2                     \n",
       "2625 1.370922e-03      8             7.668957e-03     1                     \n",
       "2626 2.252977e-04     11             1.675454e-01     1                     \n",
       "2627 2.252977e-04     22             3.350908e-01     1                     \n",
       "2628 2.252977e-04     29             4.417106e-01     1                     \n",
       "2629 2.252977e-04     78             1.188049e+00     4                     \n",
       "2630 2.507309e-03  11170             1.701347e+02   197                     \n",
       "2631 9.392522e-03     71             1.081429e+00     2                     \n",
       "2632 2.046833e-02     48             7.311072e-01     1                     \n",
       "2633 3.477985e-02     23             3.503222e-01     4                     \n",
       "     Hyper_Fold_Enrichment Hyper_Gene_Set_Coverage Hyper_Term_Gene_Coverage\n",
       "1     20.8633300           0.066666670             0.016666670             \n",
       "2     89.4142900           0.133333300             0.071428570             \n",
       "3     65.8842100           0.133333300             0.052631580             \n",
       "4      7.7271600           0.066666670             0.006172840             \n",
       "5     28.1303400           0.133333300             0.022471910             \n",
       "6     20.5287900           0.030612240             0.107142900             \n",
       "7     15.1264800           0.030612240             0.078947370             \n",
       "8     10.7641600           0.051020410             0.056179780             \n",
       "9      9.5801020           0.030612240             0.050000000             \n",
       "10    23.9502600           0.010204080             0.125000000             \n",
       "11    38.3204100           0.010204080             0.200000000             \n",
       "12    13.3675800           0.030612240             0.069767440             \n",
       "13     2.3654570           0.020408160             0.012345680             \n",
       "14     6.5318880           0.030612240             0.034090910             \n",
       "15     5.5942200           0.040816330             0.029197080             \n",
       "16     4.7602990           0.040816330             0.024844720             \n",
       "17    76.6408200           0.020408160             0.400000000             \n",
       "18    15.5181800           0.009090909             0.090909090             \n",
       "19     2.4385710           0.009090909             0.014285710             \n",
       "20     2.7983610           0.018181820             0.016393440             \n",
       "21     0.4781513           0.009090909             0.002801120             \n",
       "22     0.8577889           0.009090909             0.005025126             \n",
       "23    15.5181800           0.009090909             0.090909090             \n",
       "24     6.3222220           0.027272730             0.037037040             \n",
       "25     1.7112780           0.072727270             0.010025060             \n",
       "26   170.7000000           0.009090909             1.000000000             \n",
       "27     1.5207130           0.036363640             0.008908686             \n",
       "28     1.8385250           0.118181800             0.010770510             \n",
       "29     5.0205880           0.009090909             0.029411760             \n",
       "30     4.8771430           0.009090909             0.028571430             \n",
       "⋮    ⋮                     ⋮                       ⋮                       \n",
       "2604   1.392496            0.0196523100            0.09811321              \n",
       "2605   1.388421            0.0136054400            0.09782609              \n",
       "2606   2.987946            0.0090702950            0.21052630              \n",
       "2607   1.491203            0.0642479200            0.10506800              \n",
       "2608   1.094836            0.1443689000            0.07714055              \n",
       "2609   7.096372            0.0022675740            0.50000000              \n",
       "2610   4.319531            0.0052910050            0.30434780              \n",
       "2611   3.041302            0.0022675740            0.21428570              \n",
       "2612   8.110139            0.0030234320            0.57142860              \n",
       "2613   4.055070            0.0075585790            0.28571430              \n",
       "2614   1.553241            0.0604686300            0.10943910              \n",
       "2615   1.579250            0.0582010600            0.11127170              \n",
       "2616   1.630498            0.0665155000            0.11488250              \n",
       "2617  14.192740            0.0007558579            1.00000000              \n",
       "2618  14.192740            0.0007558579            1.00000000              \n",
       "2619   7.096372            0.0007558579            0.50000000              \n",
       "2620  17.386110            0.0555555600            0.01666667              \n",
       "2621  74.511900            0.1111111000            0.07142857              \n",
       "2622  54.903510            0.1111111000            0.05263158              \n",
       "2623   6.439300            0.0555555600            0.00617284              \n",
       "2624  23.441950            0.1111111000            0.02247191              \n",
       "2625 130.395800            0.0555555600            0.12500000              \n",
       "2626   5.968531            0.0034965030            0.09090909              \n",
       "2627   2.984266            0.0034965030            0.04545455              \n",
       "2628   2.263926            0.0034965030            0.03448276              \n",
       "2629   3.366864            0.0139860100            0.05128205              \n",
       "2630   1.157906            0.6888112000            0.01763653              \n",
       "2631   1.849404            0.0069930070            0.02816901              \n",
       "2632   1.367788            0.0034965030            0.02083333              \n",
       "2633  11.418060            0.0139860100            0.17391300              \n",
       "     Hyper_Raw_PValue Hyper_Adjp_BH\n",
       "1    0.0468906700     1.0000000    \n",
       "2    0.0002224690     0.8693951    \n",
       "3    0.0004118404     0.8693951    \n",
       "4    0.1219255000     1.0000000    \n",
       "5    0.0022407900     1.0000000    \n",
       "6    0.0004107958     0.5781266    \n",
       "7    0.0010186160     0.7514860    \n",
       "8    0.0001025428     0.4329357    \n",
       "9    0.0038035080     0.7514860    \n",
       "10   0.0410059200     1.0000000    \n",
       "11   0.0258275000     1.0000000    \n",
       "12   0.0014622900     0.7514860    \n",
       "13   0.2072462000     1.0000000    \n",
       "14   0.0109834000     0.9866365    \n",
       "15   0.0057714330     0.7860319    \n",
       "16   0.0100826900     0.9768151    \n",
       "17   0.0002668833     0.5633906    \n",
       "18   0.0626019700     1.0000000    \n",
       "19   0.3377034000     1.0000000    \n",
       "20   0.1603025000     1.0000000    \n",
       "21   0.8796965000     1.0000000    \n",
       "22   0.6913177000     1.0000000    \n",
       "23   0.0626019700     1.0000000    \n",
       "24   0.0119890800     1.0000000    \n",
       "25   0.0964867900     1.0000000    \n",
       "26   0.0058582310     1.0000000    \n",
       "27   0.2693638000     1.0000000    \n",
       "28   0.0241704600     1.0000000    \n",
       "29   0.1812216000     1.0000000    \n",
       "30   0.1860269000     1.0000000    \n",
       "⋮    ⋮                ⋮            \n",
       "2604 5.449844e-02     0.937493115  \n",
       "2605 9.849451e-02     1.000000000  \n",
       "2606 5.178047e-04     0.095050932  \n",
       "2607 1.311516e-04     0.048230861  \n",
       "2608 8.925708e-02     1.000000000  \n",
       "2609 5.937450e-03     0.358113056  \n",
       "2610 7.624330e-04     0.114964004  \n",
       "2611 7.083380e-02     0.937493115  \n",
       "2612 7.224176e-04     0.112964708  \n",
       "2613 1.049218e-04     0.047261237  \n",
       "2614 5.369848e-05     0.032387855  \n",
       "2615 4.253793e-05     0.029932523  \n",
       "2616 3.455085e-06     0.004862456  \n",
       "2617 7.045854e-02     0.937493115  \n",
       "2618 7.045854e-02     0.937493115  \n",
       "2619 1.359562e-01     1.000000000  \n",
       "2620 5.600594e-02     1.000000000  \n",
       "2621 3.232730e-04     1.000000000  \n",
       "2622 5.978157e-04     1.000000000  \n",
       "2623 1.444760e-01     1.000000000  \n",
       "2624 3.235131e-03     1.000000000  \n",
       "2625 7.644696e-03     1.000000000  \n",
       "2626 1.553896e-01     1.000000000  \n",
       "2627 2.867044e-01     1.000000000  \n",
       "2628 3.594601e-01     1.000000000  \n",
       "2629 3.124303e-02     1.000000000  \n",
       "2630 5.874243e-04     0.620026349  \n",
       "2631 2.944349e-01     1.000000000  \n",
       "2632 5.217698e-01     1.000000000  \n",
       "2633 3.713652e-04     0.522634625  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "result_all"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "1f69c29f-a439-45a7-ba9e-ac8394243655",
   "metadata": {},
   "outputs": [],
   "source": [
    "write.csv(result_all,'DAR_GO_enrichment.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "ba5dfcc9-da5f-46cd-88bd-97f6f2d314f4",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "R (r-signac)",
   "language": "R",
   "name": "r-signac"
  },
  "language_info": {
   "codemirror_mode": "r",
   "file_extension": ".r",
   "mimetype": "text/x-r-source",
   "name": "R",
   "pygments_lexer": "r",
   "version": "4.4.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
